<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
    <meta charset="utf-8">
    <meta name="generator" content="quarto-1.6.3">

    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

    <meta name="author" content="Ovie Edafe">
    <meta name="author" content="Neil Shephard">
    <meta name="author" content="Karen Sisley">
    <meta name="author" content="Sabapathy P Balasubramanian">
    <meta name="dcterms.date" content="2024-04-26">
    <meta name="keywords" content="Thyroid nodules, Thyroid cancer">

    <title>An investigation of the predictors of thyroid cancer in patients with thyroid nodules</title>
    <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      div.columns{display: flex; gap: min(4vw, 1.5em);}
      div.column{flex: auto; overflow-x: auto;}
      div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
      ul.task-list{list-style: none;}
      ul.task-list li input[type="checkbox"] {
        width: 0.8em;
        margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
        vertical-align: middle;
      }
      /* CSS for syntax highlighting */
      pre > code.sourceCode { white-space: pre; position: relative; }
      pre > code.sourceCode > span { line-height: 1.25; }
      pre > code.sourceCode > span:empty { height: 1.2em; }
      .sourceCode { overflow: visible; }
      code.sourceCode > span { color: inherit; text-decoration: inherit; }
      div.sourceCode { margin: 1em 0; }
      pre.sourceCode { margin: 0; }
      @media screen {
      div.sourceCode { overflow: auto; }
      }
      @media print {
      pre > code.sourceCode { white-space: pre-wrap; }
      pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
      }
      pre.numberSource code
        { counter-reset: source-line 0; }
      pre.numberSource code > span
        { position: relative; left: -4em; counter-increment: source-line; }
      pre.numberSource code > span > a:first-child::before
        { content: counter(source-line);
          position: relative; left: -1em; text-align: right; vertical-align: baseline;
          border: none; display: inline-block;
          -webkit-touch-callout: none; -webkit-user-select: none;
          -khtml-user-select: none; -moz-user-select: none;
          -ms-user-select: none; user-select: none;
          padding: 0 4px; width: 4em;
        }
      pre.numberSource { margin-left: 3em;  padding-left: 4px; }
      div.sourceCode
        {   }
      @media screen {
      pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
      }
      /* CSS for citations */
      div.csl-bib-body { }
      div.csl-entry {
        clear: both;
        margin-bottom: 0em;
      }
      .hanging-indent div.csl-entry {
        margin-left:2em;
        text-indent:-2em;
      }
      div.csl-left-margin {
        min-width:2em;
        float:left;
      }
      div.csl-right-inline {
        margin-left:2em;
        padding-left:1em;
      }
      div.csl-indent {
        margin-left: 2em;
      }    </style>

    <style>
      body.hypothesis-enabled #quarto-embed-header {
        padding-right: 36px;
      }

      #quarto-embed-header {
        height: 3em;
        width: 100%;
        display: flex;
        justify-content: space-between;
        align-items: center;
        border-bottom: solid 1px;
      }

      #quarto-embed-header h6 {
        font-size: 1.1em;
        padding-top: 0.6em;
        margin-left: 1em;
        margin-right: 1em;
        font-weight: 400;
      }

      #quarto-embed-header a.quarto-back-link,
      #quarto-embed-header a.quarto-download-embed {
        font-size: 0.8em;
        margin-top: 1em;
        margin-bottom: 1em;
        margin-left: 1em;
        margin-right: 1em;
      }

      .quarto-back-container {
        padding-left: 0.5em;
        display: flex;
      }

      .headroom {
          will-change: transform;
          transition: transform 200ms linear;
      }

      .headroom--pinned {
          transform: translateY(0%);
      }

      .headroom--unpinned {
          transform: translateY(-100%);
      }      
    </style>

    <script>
    window.document.addEventListener("DOMContentLoaded", function () {

      var header = window.document.querySelector("#quarto-embed-header");
      const titleBannerEl = window.document.querySelector("body > #title-block-header");
      if (titleBannerEl) {
        titleBannerEl.style.paddingTop = header.clientHeight + "px";
      }
      const contentEl = window.document.getElementById('quarto-content');
      for (const child of contentEl.children) {
        child.style.paddingTop = header.clientHeight + "px";
        child.style.marginTop = "1em";
      }

      // Use the article root if the `back` call doesn't work. This isn't perfect
      // but should typically work
      window.quartoBackToArticle = () => {
        var currentUrl = window.location.href;
        window.history.back();
        setTimeout(() => {
            // if location was not changed in 100 ms, then there is no history back
            if(currentUrl === window.location.href){              
                // redirect to site root
                window.location.href = "index.html";
            }
        }, 100);
      }

      const headroom = new window.Headroom(header, {
        tolerance: 5,
        onPin: function () {
        },
        onUnpin: function () {
        },
      });
      headroom.init();
    });
    </script>

    
<script src="site_libs/manuscript-notebook/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
     <script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>  
      <meta name="citation_title" content="An investigation of the predictors of thyroid cancer in patients with thyroid nodules">
<meta name="citation_abstract" content="An abstract summarising the work undertaken and the overall conclusions can be placed here. Sub-headings are currently
removed because they conflict with those in the body of the text and mess up the links in the Table of Contents.
">
<meta name="citation_keywords" content="Thyroid nodules,Thyroid cancer">
<meta name="citation_author" content="Ovie Edafe">
<meta name="citation_author" content="Neil Shephard">
<meta name="citation_author" content="Karen Sisley">
<meta name="citation_author" content="Sabapathy P Balasubramanian">
<meta name="citation_publication_date" content="2024-04-26">
<meta name="citation_cover_date" content="2024-04-26">
<meta name="citation_year" content="2024">
<meta name="citation_online_date" content="2024-04-26">
<meta name="citation_language" content="en">
<meta name="citation_journal_title" content="Sheffield study on thyroid nodules">
<meta name="citation_reference" content="citation_title=Computer Age Statistical Inference: Algorithms, Evidence, and Data Science;,citation_abstract=The twenty-first century has seen a breathtaking expansion of statistical methodology, both in scope and in influence. ’Big data’, ’data science’, and ’machine learning’ have become familiar terms in the news, as statistical methods are brought to bear upon the enormous data sets of modern science and commerce. How did we get here? And where are we going? This book takes us on an exhilarating journey through the revolution in data analysis following the introduction of electronic computation in the 1950s. Beginning with classical inferential theories - Bayesian, frequentist, Fisherian - individual chapters take up a series of influential topics: survival analysis, logistic regression, empirical Bayes, the jackknife and bootstrap, random forests, neural networks, Markov chain Monte Carlo, inference after model selection, and dozens more. The distinctly modern approach integrates methodology and algorithms with statistical inference. The book ends with speculation on the future direction of statistics and data science.;,citation_author=Bradley Efron;,citation_author=Trevor Hastie;,citation_publication_date=2016-07;,citation_cover_date=2016-07;,citation_year=2016;,citation_doi=10.1017/CBO9781316576533;,citation_isbn=978-1-10714989-2;,citation_book_title=Cambridge Core;">
<meta name="citation_reference" content="citation_title=Ggmice: Visualizations for ’mice’ with ’ggplot2’;,citation_author=Hanne Oberman;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://github.com/amices/ggmice;">
<meta name="citation_reference" content="citation_title=mice: Multivariate imputation by chained equations in r;,citation_author=Stef Buuren;,citation_author=Karin Groothuis-Oudshoorn;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_issue=3;,citation_doi=10.18637/jss.v045.i03;,citation_volume=45;,citation_journal_title=Journal of Statistical Software;">
<meta name="citation_reference" content="citation_title=mice: Multivariate Imputation by Chained Equations in R;,citation_abstract=The R package mice imputes incomplete multivariate data by chained equations. The software mice 1.0 appeared in the year 2000 as an S-PLUS library, and in 2001 as an R package. mice 1.0 introduced predictor selection, passive imputation and automatic pooling. This article documents mice, which extends the functionality of mice 1.0 in several ways. In mice, the analysis of imputed data is made completely general, whereas the range of models under which pooling works is substantially extended. mice adds new functionality for imputing multilevel data, automatic predictor selection, data handling, post-processing imputed values, specialized pooling routines, model selection tools, and diagnostic graphs. Imputation of categorical data is improved in order to bypass problems caused by perfect prediction. Special attention is paid to transformations, sum scores, indices and interactions using passive imputation, and to the proper setup of the predictor matrix. mice can be downloaded from the Comprehensive R Archive Network. This article provides a hands-on, stepwise approach to solve applied incomplete data problems.;,citation_author=Stef Buuren;,citation_author=Karin Groothuis-Oudshoorn;,citation_publication_date=2011-12;,citation_cover_date=2011-12;,citation_year=2011;,citation_doi=10.18637/jss.v045.i03;,citation_issn=1548-7660;,citation_volume=45;,citation_journal_title=J. Stat. Soft.;">
<meta name="citation_reference" content="citation_title=The Concordance Test, an Alternative to Kruskal-Wallis Based on the Kendall-$\tau$ Distance: An R Package;,citation_abstract=The Kendall rank correlation coefficient, based on the Kendall-$\tau$ distance, is used to measure the ordinal association between two measurements. In this paper, we introduce a new coefficient also based on the Kendall-$\tau$ distance, the Concordance coefficient, and a test to measure whether different samples come from the same distribution. This work also presents a new R package, ConcordanceTest, with the implementation of the proposed coefficient. We illustrate the use of the Concordance coefficient to measure the ordinal association between quantity and quality measures when two or more samples are considered. In this sense, the Concordance coefficient can be seen as a generalization of the Kendall rank correlation coefficient and an alternative to the non-parametric mean rank-based methods for comparing two or more samples. A comparison of the proposed Concordance coefficient and the classical Kruskal-Wallis statistic is presented through a comparison of the exact distributions of both statistics.;,citation_author=Javier Alcaraz;,citation_author=Laura Anton-Sanchez;,citation_author=Juan Francisco Monge;,citation_publication_date=2022-10;,citation_cover_date=2022-10;,citation_year=2022;,citation_issue=2;,citation_doi=10.32614/RJ-2022-039;,citation_issn=2073-4859;,citation_volume=14;,citation_journal_title=R Journal;">
<meta name="citation_reference" content="citation_title=A unified approach to interpreting model predictions;,citation_author=Scott M Lundberg;,citation_author=Su-In Lee;,citation_editor=I. Guyon;,citation_editor=U. V. Luxburg;,citation_editor=S. Bengio;,citation_editor=H. Wallach;,citation_editor=R. Fergus;,citation_editor=S. Vishwanathan;,citation_editor=R. Garnett;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_fulltext_html_url=http://papers.nips.cc/paper/7062-a-unified-approach-to-interpreting-model-predictions.pdf;,citation_publisher=Curran Associates, Inc.;">
<meta name="citation_reference" content="citation_title=&amp;amp;amp;quot;Why Should I Trust You?&amp;quot;: Explaining the Predictions of Any Classifier;,citation_abstract=Despite widespread adoption, machine learning models remain mostly black boxes. Understanding the reasons behind predictions is, however, quite important in assessing trust, which is fundamental if one plans to take action based on a prediction, or when choosing whether to deploy a new model. Such understanding also provides insights into the model, which can be used to transform an untrustworthy model or prediction into a trustworthy one. In this work, we propose LIME, a novel explanation technique that explains the predictions of any classifier in an interpretable and faithful manner, by learning an interpretable model locally around the prediction. We also propose a method to explain models by presenting representative individual predictions and their explanations in a non-redundant way, framing the task as a submodular optimization problem. We demonstrate the flexibility of these methods by explaining different models for text (e.g. random forests) and image classification (e.g. neural networks). We show the utility of explanations via novel experiments, both simulated and with human subjects, on various scenarios that require trust: deciding if one should trust a prediction, choosing between models, improving an untrustworthy classifier, and identifying why a classifier should not be trusted.;,citation_author=Marco Tulio Ribeiro;,citation_author=Sameer Singh;,citation_author=Carlos Guestrin;,citation_publication_date=2016-02;,citation_cover_date=2016-02;,citation_year=2016;,citation_doi=10.48550/arXiv.1602.04938;,citation_journal_title=arXiv;">
<meta name="citation_reference" content="citation_title=Step away from stepwise;,citation_abstract=Stepwise regression is a popular data-mining tool that uses statistical significance to select the explanatory variables to be used in a multiple-regression model. A fundamental problem with stepwise regression is that some real explanatory variables that have causal effects on the dependent variable may happen to not be statistically significant, while nuisance variables may be coincidentally significant. As a result, the model may fit the data well in-sample, but do poorly out-of-sample. Many Big-Data researchers believe that, the larger the number of possible explanatory variables, the more useful is stepwise regression for selecting explanatory variables. The reality is that stepwise regression is less effective the larger the number of potential explanatory variables. Stepwise regression does not solve the Big-Data problem of too many explanatory variables. Big Data exacerbates the failings of stepwise regression.;,citation_author=Gary Smith;,citation_publication_date=2018-12;,citation_cover_date=2018-12;,citation_year=2018;,citation_issue=1;,citation_doi=10.1186/s40537-018-0143-6;,citation_issn=2196-1115;,citation_volume=5;,citation_journal_title=J. Big Data;,citation_publisher=SpringerOpen;">
<meta name="citation_reference" content="citation_title=Prognostic modeling with logistic regression analysis;,citation_abstract=Clinical decision making often requires estimates of the likelihood of a dichotomous outcome in individual patients. When empirical data are available, these estimates may well be obtained from a logistic regression model. Several strategies may be followed in the development of such a model. In this study, the authors compare alternative strategies in 23 small subsamples from a large data set of patients with an acute myocardial infarction, where they developed predictive models for 30-day mortality. Evaluations were performed in an independent part of the data set. Specifically, the authors studied the effect of coding of covariables and stepwise selection on discriminative ability of the resulting model, and the effect of statistical ” shrinkage” techniques on calibration. As expected, dichotomization of continuous covariables implied a loss of information. Remarkably, stepwise selection resulted in less discriminating models compared to full models including all available covariables, even when more than half of these were randomly associated with the outcome. Using qualitative information on the sign of the effect of predictors slightly improved the predictive ability. Calibration improved when shrinkage was applied on the standard maximum likelihood estimates of the regression coefficients. In conclusion, a sensible strategy in small data sets is to apply shrinkage methods in full models that include well-coded predictors that are selected based on external information.;,citation_author=Ewout W. Steyerberg;,citation_author=Marinus J. C. Eijkemans;,citation_author=Frank E. Harrell;,citation_author=undefined Dik;,citation_publication_date=2001-02-01;,citation_cover_date=2001-02-01;,citation_year=2001;,citation_fulltext_html_url=http://dx.doi.org/10.1177/0272989x0102100106;,citation_issue=1;,citation_doi=10.1177/0272989x0102100106;,citation_issn=1552-681X;,citation_pmid=11206946;,citation_volume=21;,citation_journal_title=Medical Decision Making;,citation_publisher=SAGE Publications;">
<meta name="citation_reference" content="citation_title=Stepwise regression and stepwise discriminant analysis need not apply here: A guidelines editorial;,citation_abstract=Stepwise methods are frequently employed in educational and psychological research, both to select useful subsets of variables and to evaluate the order of importance of variables. Three problems with stepwise applications are explored in some detail. First, computer packages use incorrect degrees of freedom in their stepwise computations, resulting in artifactually greater likelihood of obtaining spurious statistical significance. Second, stepwise methods do not correctly identify the best variable set of a given size, as illustrated by a concrete heuristic example. Third, stepwise methods tend to capitalize on sampling error and thus tend to yield results that are not replicable.;,citation_author=Bruce Thompson;,citation_publication_date=1995-08-01;,citation_cover_date=1995-08-01;,citation_year=1995;,citation_fulltext_html_url=http://dx.doi.org/10.1177/0013164495055004001;,citation_issue=4;,citation_doi=10.1177/0013164495055004001;,citation_issn=1552-3888;,citation_volume=55;,citation_journal_title=Educational and Psychological Measurement;,citation_publisher=SAGE Publications;">
<meta name="citation_reference" content="citation_title=Regularization and Variable Selection Via the Elastic Net;,citation_abstract=We propose the elastic net, a new regularization and variable selection method. Real world data and a simulation study show that the elastic net often outperforms the lasso, while enjoying a similar sparsity of representation. In addition, the elastic net encourages a grouping effect, where strongly correlated predictors tend to be in or out of the model together. The elastic net is particularly useful when the number of predictors (p) is much bigger than the number of observations (n). By contrast, the lasso is not a very satisfactory variable selection method in the p$\gg$n case. An algorithm called LARS-EN is proposed for computing elastic net regularization paths efficiently, much like algorithm LARS does for the lasso.;,citation_author=Hui Zou;,citation_author=Trevor Hastie;,citation_publication_date=2005-04;,citation_cover_date=2005-04;,citation_year=2005;,citation_issue=2;,citation_doi=10.1111/j.1467-9868.2005.00503.x;,citation_issn=1369-7412;,citation_volume=67;,citation_journal_title=J. R. Stat. Soc. Ser. B. Stat. Methodol.;,citation_publisher=Oxford Academic;">
<meta name="citation_reference" content="citation_title=Regression Shrinkage and Selection via the Lasso;,citation_abstract=Robert Tibshirani, Regression Shrinkage and Selection via the Lasso, Journal of the Royal Statistical Society. Series B (Methodological), Vol. 58, No. 1 (1996), pp. 267-288;,citation_publication_date=1996;,citation_cover_date=1996;,citation_year=1996;,citation_fulltext_html_url=https://www.jstor.org/stable/2346178;,citation_volume=58;,citation_journal_title=Journal of the Royal Statistical Society. Series B (Methodological);">
<meta name="citation_reference" content="citation_title=Tidymodels: A collection of packages for modeling and machine learning using tidyverse principles.;,citation_author=Max Kuhn;,citation_author=Hadley Wickham;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://www.tidymodels.org;">
<meta name="citation_reference" content="citation_title=Genomic alterations in thyroid cancer: biological and clinical insights;,citation_abstract=Tumours can arise from thyroid follicular cells if they acquire driver mutations that constitutively activate the MAPK signalling pathway. In addition, a limited set of additional mutations in key genes drive tumour progression towards more aggressive and less differentiated disease. Unprecedented insights into thyroid tumour biology have come from the breadth of thyroid tumour sequencing data from patients and the wide range of mutation-specific mechanisms identified in experimental models, in combination with the genomic simplicity of thyroid cancers. This knowledge is gradually being translated into refined strategies to stratify, manage and treat patients with thyroid cancer. This Review summarizes the biological underpinnings of the genetic alterations involved in thyroid cancer initiation and progression. We also provide a rationale for and discuss specific examples of how to implement genomic information to inform both recommended and investigational approaches to improve thyroid cancer prognosis, redifferentiation strategies and targeted therapies.;,citation_author=Iñigo Landa;,citation_author=Maria E. Cabanillas;,citation_publication_date=2024-02;,citation_cover_date=2024-02;,citation_year=2024;,citation_issue=2;,citation_doi=10.1038/s41574-023-00920-6;,citation_issn=1759-5037;,citation_volume=20;,citation_journal_title=Nat. Rev. Endocrinol.;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=Welcome to the tidyverse;,citation_author=Hadley Wickham;,citation_author=Mara Averick;,citation_author=Jennifer Bryan;,citation_author=Winston Chang;,citation_author=Lucy D’Agostino McGowan;,citation_author=Romain François;,citation_author=Garrett Grolemund;,citation_author=Alex Hayes;,citation_author=Lionel Henry;,citation_author=Jim Hester;,citation_author=Max Kuhn;,citation_author=Thomas Lin Pedersen;,citation_author=Evan Miller;,citation_author=Stephan Milton Bache;,citation_author=Kirill Müller;,citation_author=Jeroen Ooms;,citation_author=David Robinson;,citation_author=Dana Paige Seidel;,citation_author=Vitalie Spinu;,citation_author=Kohske Takahashi;,citation_author=Davis Vaughan;,citation_author=Claus Wilke;,citation_author=Kara Woo;,citation_author=Hiroaki Yutani;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=43;,citation_doi=10.21105/joss.01686;,citation_volume=4;,citation_journal_title=Journal of Open Source Software;">
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;,citation_author=R Core Team;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.R-project.org/;">
<meta name="citation_reference" content="citation_title=Overview $\vert$ Thyroid cancer: assessment and management $\vert$ Guidance $\vert$ NICE;,citation_abstract=This guideline covers diagnosis and management of thyroid cancer in people aged 16 and over. It aims to reduce variation in practice and increase the quality of care and survival for people with thyroid cancer;,citation_publication_date=2024-03;,citation_cover_date=2024-03;,citation_year=2024;,citation_fulltext_html_url=https://www.nice.org.uk/guidance/ng230;,citation_publisher=NICE;">
<meta name="citation_reference" content="citation_title=The role of vitamin d, platelet-derived growth factor and insulin-like growth factor 1 in the progression of thyroid diseases;,citation_author=M. S. Abdellateif;,citation_author=S. Shaarawy;,citation_author=Y. F. Elesawy;,citation_author=M. Mansour;,citation_author=E. Tharwat;,citation_author=N. H. Ibrahim;,citation_author=M. S. Eissa;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=7;,citation_doi=10.31557/apjcp.2020.21.7.2083;,citation_issn=1513-7368 (Print)
1513-7368;,citation_volume=21;,citation_journal_title=Asian Pac J Cancer Prev;">
<meta name="citation_reference" content="citation_title=Point of care measurement of body mass index and thyroid nodule malignancy risk assessment;,citation_author=S. Ahmadi;,citation_author=T. Pappa;,citation_author=A. S. Kang;,citation_author=A. K. Coleman;,citation_author=I. Landa;,citation_author=E. Marqusee;,citation_author=M. Kim;,citation_author=T. E. Angell;,citation_author=E. K. Alexander;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_doi=10.3389/fendo.2022.824226;,citation_issn=1664-2392 (Print)
1664-2392;,citation_volume=13;,citation_journal_title=Front Endocrinol (Lausanne);">
<meta name="citation_reference" content="citation_title=Thyroid cancer screening in south korea increases detection of papillary cancers with no impact on other subtypes or thyroid cancer mortality;,citation_author=H. S. Ahn;,citation_author=H. J. Kim;,citation_author=K. H. Kim;,citation_author=Y. S. Lee;,citation_author=S. J. Han;,citation_author=Y. Kim;,citation_author=M. J. Ko;,citation_author=J. P. Brito;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=11;,citation_volume=26;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Thyroid nodule management: Thyroid-stimulating hormone, ultrasound, and cytological classification system for predicting malignancy;,citation_author=M. A. Al Dawish;,citation_author=A. Alwin Robert;,citation_author=M. A. Thabet;,citation_author=R. Braham;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_doi=10.1177/1176935118765132;,citation_issn=1176-9351 (Print)
1176-9351;,citation_volume=17;,citation_journal_title=Cancer Inform;">
<meta name="citation_reference" content="citation_title=TSH level as a risk factor of thyroid malignancy for nodules in euthyroid patients;,citation_author=A. K. Alaraifi;,citation_author=M. Alessa;,citation_author=L. O. Hijazi;,citation_author=A. M. Alayed;,citation_author=A. A. Alsalem;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=3;,citation_doi=10.14639/0392-100x-n2288;,citation_issn=0392-100X (Print)
0392-100x;,citation_volume=43;,citation_journal_title=Acta Otorhinolaryngol Ital;">
<meta name="citation_reference" content="citation_title=Nasal septal hematoma in children: Time to diagnosis and resulting complications;,citation_author=H. M. Ali;,citation_author=H. Zavala;,citation_author=S. Chinnadurai;,citation_author=B. Roby;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.elsevier.com/locate/ijporl
http://ovidsp.ovid.com/ovidweb.cgi?T=JS&amp;amp;amp;PAGE=reference&amp;D=emed22&amp;NEWS=N&amp;AN=2011859732;,citation_doi=https://dx.doi.org/10.1016/j.ijporl.2021.110734;,citation_issn=0165-5876
1872-8464;,citation_volume=145;,citation_journal_title=International Journal of Pediatric Otorhinolaryngology;">
<meta name="citation_reference" content="citation_title=The relationship between papillary thyroid cancer and triglyceride/glucose index, which is an indicator of insulin resistance;,citation_author=E. G. Alkurt;,citation_author=F. Şahin;,citation_author=B. Tutan;,citation_author=K. Canal;,citation_author=V. B. Turhan;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=17;,citation_doi=10.26355/eurrev_202209_29629;,citation_issn=1128-3602;,citation_volume=26;,citation_journal_title=Eur Rev Med Pharmacol Sci;">
<meta name="citation_reference" content="citation_title=Prevalence of pathogenic germline DICER1 variants in young individuals thyroidectomised due to goitre - a national danish cohort;,citation_author=M. Altaraihi;,citation_author=T. V. O. Hansen;,citation_author=E. Santoni-Rugiu;,citation_author=M. Rossing;,citation_author=K. Rasmussen Å;,citation_author=A. M. Gerdes;,citation_author=K. Wadt;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_doi=10.3389/fendo.2021.727970;,citation_issn=1664-2392 (Print)
1664-2392;,citation_volume=12;,citation_journal_title=Front Endocrinol (Lausanne);">
<meta name="citation_reference" content="citation_title=Controversy on the management of patients carrying RET p.V804M mutation;,citation_author=A. S. Alzahrani;,citation_author=M. Alswailem;,citation_author=B. Alghamdi;,citation_author=R. Rafiullah;,citation_author=M. Aldawish;,citation_author=H. Al-Hindi;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=2;,citation_doi=10.1007/s12020-021-02895-8;,citation_issn=1355-008x;,citation_volume=75;,citation_journal_title=Endocrine;">
<meta name="citation_reference" content="citation_title=A comprehensive study on the diagnosis and management of noninvasive follicular thyroid neoplasm with papillary-like nuclear features;,citation_author=B. A. Alzumaili;,citation_author=L. N. Krumeich;,citation_author=R. Collins;,citation_author=T. Kravchenko;,citation_author=E. I. Ababneh;,citation_author=A. S. Fisch;,citation_author=W. C. Faquin;,citation_author=V. Nosé;,citation_author=M. Martinez-Lage;,citation_author=G. W. Randolph;,citation_author=R. M. Gartland;,citation_author=C. C. Lubitz;,citation_author=P. M. Sadow;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=5;,citation_doi=10.1089/thy.2023.0035;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=33;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Hashimoto’s thyroiditis is not associated with increased risk of thyroid cancer in patients with thyroid nodules: A single-center prospective study;,citation_author=C. Anil;,citation_author=S. Goksel;,citation_author=A. Gursoy;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=6;,citation_doi=10.1089/thy.2009.0450;,citation_issn=1050-7256;,citation_volume=20;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=A pragmatic assessment of the british thyroid association &amp;amp;amp;quot;u classification&amp;quot; of thyroid nodules with a focus on their follow-up;,citation_author=M. H. Arambewela;,citation_author=A. M. Wijesinghe;,citation_author=K. Randhawa;,citation_author=M. Bull;,citation_author=J. Wadsley;,citation_author=S. P. Balasubramanian;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=6;,citation_doi=10.1016/j.crad.2020.02.012;,citation_issn=0009-9260;,citation_volume=75;,citation_journal_title=Clin Radiol;">
<meta name="citation_reference" content="citation_title=Chronic lymphocytic thyroiditis: Could it be influenced by a petrochemical complex? Data from a cytological study in south-eastern sicily;,citation_author=S. Arena;,citation_author=A. Latina;,citation_author=R. Baratta;,citation_author=G. Burgio;,citation_author=D. Gullo;,citation_author=S. Benvenga;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=4;,citation_doi=10.1530/eje-14-0864;,citation_issn=0804-4643;,citation_volume=172;,citation_journal_title=Eur J Endocrinol;">
<meta name="citation_reference" content="citation_title=Higher TSH level is associated with increased risk of malignancy in thyroid nodules;,citation_author=Mahshid. Jalali Ariannia;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=1;,citation_volume=9;,citation_journal_title=Journal of Pioneering Medical Sciences;">
<meta name="citation_reference" content="citation_title=Management of differentiated thyroid cancer: The standard of care;,citation_author=A. M. Avram;,citation_author=K. Zukotynski;,citation_author=H. R. Nadel;,citation_author=L. Giovanella;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=2;,citation_doi=10.2967/jnumed.121.262402;,citation_issn=0161-5505;,citation_volume=63;,citation_journal_title=J Nucl Med;">
<meta name="citation_reference" content="citation_title=Association of hashimoto’s thyroiditis with thyroid cancer;,citation_author=G. Azizi;,citation_author=J. M. Keller;,citation_author=M. Lewis;,citation_author=K. Piper;,citation_author=D. Puett;,citation_author=K. M. Rivenbark;,citation_author=C. D. Malchoff;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=6;,citation_doi=10.1530/erc-14-0258;,citation_issn=1351-0088 (Print)
1351-0088;,citation_volume=21;,citation_journal_title=Endocr Relat Cancer;">
<meta name="citation_reference" content="citation_title=Autoimmune thyroid disease: A risk factor for thyroid cancer;,citation_author=G. Azizi;,citation_author=C. D. Malchoff;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_issue=2;,citation_doi=10.4158/ep10123.Or;,citation_issn=1530-891x;,citation_volume=17;,citation_journal_title=Endocr Pract;">
<meta name="citation_reference" content="citation_title=A new form of familial multi-nodular goitre with progression to differentiated thyroid cancer;,citation_author=A. Bakhsh;,citation_author=G. Kirov;,citation_author=J. W. Gregory;,citation_author=E. D. Williams;,citation_author=M. Ludgate;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_issue=2;,citation_doi=10.1677/erc.1.01138;,citation_issn=1351-0088 (Print)
1351-0088;,citation_volume=13;,citation_journal_title=Endocr Relat Cancer;">
<meta name="citation_reference" content="citation_title=Risk-stratification of thyroid nodules examined by (18)FDG-PET/CT while ensuring congruity between imaging and histopathological localization;,citation_author=P. Bakkegaard;,citation_author=S. C. Londero;,citation_author=S. J. Bonnema;,citation_author=V. E. Nielsen;,citation_author=M. L. Jespersen;,citation_author=K. Z. Swan;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=12;,citation_doi=10.1007/s00405-021-06733-5;,citation_issn=0937-4477;,citation_volume=278;,citation_journal_title=Eur Arch Otorhinolaryngol;">
<meta name="citation_reference" content="citation_title=Is there any association between insulin resistance and thyroid cancer? : A case control study;,citation_author=F. Balkan;,citation_author=E. D. Onal;,citation_author=A. Usluogullari;,citation_author=D. Tuzun;,citation_author=D. Ozdemir;,citation_author=S. S. Inancli;,citation_author=R. Ersoy;,citation_author=B. Cakir;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=1;,citation_doi=10.1007/s12020-013-9942-x;,citation_issn=1355-008x;,citation_volume=45;,citation_journal_title=Endocrine;">
<meta name="citation_reference" content="citation_title=Overview of the 2022 WHO classification of thyroid neoplasms;,citation_author=Z. W. Baloch;,citation_author=S. L. Asa;,citation_author=J. A. Barletta;,citation_author=R. A. Ghossein;,citation_author=C. C. Juhlin;,citation_author=C. K. Jung;,citation_author=V. A. LiVolsi;,citation_author=M. G. Papotti;,citation_author=M. Sobrinho-Simões;,citation_author=G. Tallini;,citation_author=O. Mete;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=1;,citation_doi=10.1007/s12022-022-09707-3;,citation_issn=1046-3976;,citation_volume=33;,citation_journal_title=Endocr Pathol;">
<meta name="citation_reference" content="citation_title=Benign cytology does not rule out malignancy in thyroid nodules larger than 4&nbsp;cm;,citation_author=O. O. Baser;,citation_author=D. Koseoglu;,citation_author=Z. Cetin;,citation_author=M. Catak;,citation_author=H. Kizilkaya;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=11;,citation_doi=10.1002/dc.25024;,citation_issn=1097-0339;,citation_volume=50;,citation_journal_title=Diagn Cytopathol;">
<meta name="citation_reference" content="citation_title=Cancer risk in patients with cold thyroid nodules: Relevance of iodine intake, sex, age, and multinodularity;,citation_author=A. Belfiore;,citation_author=G. L. La Rosa;,citation_author=G. A. La Porta;,citation_author=D. Giuffrida;,citation_author=G. Milazzo;,citation_author=L. Lupo;,citation_author=C. Regalbuto;,citation_author=R. Vigneri;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_issue=4;,citation_doi=10.1016/0002-9343(92)90164-7;,citation_issn=0002-9343 (Print)
0002-9343;,citation_volume=93;,citation_journal_title=Am J Med;">
<meta name="citation_reference" content="citation_title=High prevalence of suspicious cytology in thyroid nodules associated with positive thyroid autoantibodies;,citation_author=F. Boi;,citation_author=M. L. Lai;,citation_author=B. Marziani;,citation_author=L. Minerba;,citation_author=G. Faa;,citation_author=S. Mariotti;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_issue=5;,citation_doi=10.1530/eje.1.02020;,citation_issn=0804-4643 (Print)
0804-4643;,citation_volume=153;,citation_journal_title=Eur J Endocrinol;">
<meta name="citation_reference" content="citation_title=Both thyroid autoimmunity and increased serum TSH are independent risk factors for malignancy in patients with thyroid nodules;,citation_author=F. Boi;,citation_author=L. Minerba;,citation_author=M. L. Lai;,citation_author=B. Marziani;,citation_author=B. Figus;,citation_author=F. Spanu;,citation_author=A. Borghero;,citation_author=S. Mariotti;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_issue=5;,citation_doi=10.3275/8579;,citation_issn=0391-4097;,citation_volume=36;,citation_journal_title=J Endocrinol Invest;">
<meta name="citation_reference" content="citation_title=The bethesda system for reporting thyroid cytopathology: A meta-analysis;,citation_author=M. Bongiovanni;,citation_author=A. Spitale;,citation_author=W. C. Faquin;,citation_author=L. Mazzucchelli;,citation_author=Z. W. Baloch;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_issue=4;,citation_doi=10.1159/000339959;,citation_issn=0001-5547 (Print)
0001-5547;,citation_volume=56;,citation_journal_title=Acta Cytol;">
<meta name="citation_reference" content="citation_title=The bethesda system for reporting thyroid cytopathology: A meta-analysis;,citation_author=M. Bongiovanni;,citation_author=William C. Spitale A Fau - Faquin;,citation_author=Luca Faquin Wc Fau - Mazzucchelli;,citation_author=Zubair W. Mazzucchelli L Fau - Baloch;,citation_author=Z. W. Baloch;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_issue=333-9;,citation_volume=56;,citation_journal_title=Acta Cytol;">
<meta name="citation_reference" content="citation_title=Thyroid cancer in the democratic republic of the congo: Frequency and risk factors;,citation_author=J. Bukasa Kakamba;,citation_author=N. Sabbah;,citation_author=P. Bayauli;,citation_author=M. Massicard;,citation_author=J. Bidingija;,citation_author=A. Nkodila;,citation_author=B. Mbunga;,citation_author=S. Ditu;,citation_author=A. Beckers;,citation_author=I. Potorac;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=6;,citation_doi=10.1016/j.ando.2021.09.002;,citation_issn=0003-4266;,citation_volume=82;,citation_journal_title=Ann Endocrinol (Paris);">
<meta name="citation_reference" content="citation_title=Papillary thyroid carcinoma in struma ovarii: Management after surgery;,citation_author=C. Chandran;,citation_author=P. Lekprasert;,citation_author=C. Minimo;,citation_author=K. Win;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=11;,citation_doi=10.1136/bcr-2022-252129;,citation_issn=1757-790x;,citation_volume=15;,citation_journal_title=BMJ Case Rep;">
<meta name="citation_reference" content="citation_title=Ascent of machine learning in medicine;,citation_author=C Chen;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=5;,citation_doi=10.1038/s41563-019-0360-1;,citation_issn=1476-1122 (Print)
1476-1122;,citation_volume=18;,citation_journal_title=Nature Materials;">
<meta name="citation_reference" content="citation_title=The utility of serum anti-thyroglobulin antibody and thyroglobulin in the preoperative differential diagnosis of thyroid follicular neoplasms;,citation_author=Z. Chen;,citation_author=Y. Lin;,citation_author=S. Lai;,citation_author=P. Wang;,citation_author=J. Li;,citation_author=L. Wang;,citation_author=H. Guan;,citation_author=J. Kuang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=2;,citation_doi=10.1007/s12020-022-02993-1;,citation_issn=1355-008x;,citation_volume=76;,citation_journal_title=Endocrine;">
<meta name="citation_reference" content="citation_title=The diagnostic predictive value of neutrophil-to-lymphocyte ratio in thyroid cancer adjusted for tumor size;,citation_author=T. Y. Cheong;,citation_author=S. D. Hong;,citation_author=K. W. Jung;,citation_author=Y. K. So;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=5;,citation_doi=10.1371/journal.pone.0251446;,citation_issn=1932-6203;,citation_volume=16;,citation_journal_title=PLoS One;">
<meta name="citation_reference" content="citation_title=The influence of incidental detection of thyroid nodule on thyroid cancer risk and prognosis-a systematic review;,citation_author=J. E. Chooi;,citation_author=A. Ravindiran;,citation_author=S. P. Balasubramanian;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=2;,citation_doi=10.1111/cen.14575;,citation_issn=0300-0664;,citation_volume=96;,citation_journal_title=Clin Endocrinol (Oxf);">
<meta name="citation_reference" content="citation_title=Follicular thyroid carcinoma: How have histologic diagnoses changed in the last half-century and what are the prognostic implications?;,citation_author=N. A. Cipriani;,citation_author=S. Nagar;,citation_author=S. P. Kaplan;,citation_author=M. G. White;,citation_author=T. Antic;,citation_author=P. M. Sadow;,citation_author=B. Aschebrook-Kilfoy;,citation_author=P. Angelos;,citation_author=E. L. Kaplan;,citation_author=R. H. Grogan;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=11;,citation_doi=10.1089/thy.2015.0297;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=25;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.cancerresearchuk.org/health-professional/cancer-statistics/statistics-by-cancer-type/thyroid-cancer/incidence#heading-Zero;">
<meta name="citation_reference" content="citation_title=Increasing incidence of thyroid cancer in the united states, 1973-2002;,citation_author=L. Davies;,citation_author=H. G. Welch;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_issue=18;,citation_doi=10.1001/jama.295.18.2164;,citation_issn=0098-7484;,citation_volume=295;,citation_journal_title=Jama;">
<meta name="citation_reference" content="citation_title=Factors associated with malignancy in patients with maximal thyroid nodules ≥2 cm;,citation_author=S. Dong;,citation_author=J. Pan;,citation_author=Y. B. Shen;,citation_author=L. X. Zhu;,citation_author=Q. Xia;,citation_author=X. J. Xie;,citation_author=Y. J. Wu;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_doi=10.2147/cmar.S303715;,citation_issn=1179-1322 (Print)
1179-1322;,citation_volume=13;,citation_journal_title=Cancer Manag Res;">
<meta name="citation_reference" content="citation_title=High serum thyrotropin concentrations within the reference range: A predictor of malignancy in nodular thyroid disease;,citation_author=K. Duccini;,citation_author=M. V. L. Souza;,citation_author=R. Delfim;,citation_author=A. P. Aguiar;,citation_author=P. Teixeira;,citation_author=M. Vaisman;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=3;,citation_doi=10.1159/000488196;,citation_issn=1011-7571 (Print)
1011-7571;,citation_volume=27;,citation_journal_title=Med Princ Pract;">
<meta name="citation_reference" content="citation_title=Thyroid nodules located in the lower pole have a higher risk of malignancy than located in the isthmus: A single-center experience;,citation_author=G. Duman;,citation_author=B. Sariakcali;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_doi=10.1155/2021/9940995;,citation_issn=1687-8337 (Print)
1687-8337;,citation_volume=2021;,citation_journal_title=Int J Endocrinol;">
<meta name="citation_reference" content="citation_title=The diagnosis and management of thyroid nodules: A review;,citation_author=C. Durante;,citation_author=G. Grani;,citation_author=L. Lamartina;,citation_author=S. Filetti;,citation_author=S. J. Mandel;,citation_author=D. S. Cooper;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=9;,citation_doi=10.1001/jama.2018.0898;,citation_issn=0098-7484;,citation_volume=319;,citation_journal_title=Jama;">
<meta name="citation_reference" content="citation_title=Systematic review and meta-analysis of predictors of post-thyroidectomy hypocalcaemia;,citation_author=O. Edafe;,citation_author=R. Antakia;,citation_author=N. Laskar;,citation_author=L. Uttley;,citation_author=S. P. Balasubramanian;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=4;,citation_doi=10.1002/bjs.9384;,citation_issn=0007-1323;,citation_volume=101;,citation_journal_title=Br J Surg;">
<meta name="citation_reference" content="citation_title=Incidence, prevalence and risk factors for post-surgical hypocalcaemia and hypoparathyroidism;,citation_author=O. Edafe;,citation_author=S. P. Balasubramanian;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=Suppl 1;,citation_doi=10.21037/gs.2017.09.03;,citation_issn=2227-684X (Print)
2227-684x;,citation_volume=6;,citation_journal_title=Gland Surg;">
<meta name="citation_reference" content="citation_title=Reoperation for bleeding after thyroid and parathyroid surgery: Incidence, risk factors, prevention, and management;,citation_author=O. Edafe;,citation_author=E. Cochrane;,citation_author=S. P. Balasubramanian;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=4;,citation_doi=10.1007/s00268-019-05322-2;,citation_issn=0364-2313;,citation_volume=44;,citation_journal_title=World J Surg;">
<meta name="citation_reference" content="citation_title=Application of a machine learning algorithm to predict malignancy in thyroid cytopathology;,citation_author=D. D. Elliott Range;,citation_author=D. Dov;,citation_author=S. Z. Kovalsky;,citation_author=R. Henao;,citation_author=L. Carin;,citation_author=J. Cohen;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=4;,citation_doi=10.1002/cncy.22238;,citation_issn=1934-662x;,citation_volume=128;,citation_journal_title=Cancer Cytopathol;">
<meta name="citation_reference" content="citation_title=Risk of malignancy in thyroid nodules with non-diagnostic fine-needle aspiration: A retrospective cohort study;,citation_author=A. E. Espinosa De Ycaza;,citation_author=K. M. Lowe;,citation_author=D. S. Dean;,citation_author=M. R. Castro;,citation_author=V. Fatourechi;,citation_author=M. Ryder;,citation_author=J. C. Morris;,citation_author=M. N. Stan;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=11;,citation_doi=10.1089/thy.2016.0096;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=26;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Reliability of the ultrasound classification system of thyroid nodules in predicting malignancy;,citation_author=A. G. Farihah;,citation_author=M. I. Nurismah;,citation_author=H. Husyairi;,citation_author=A. S. Shahrun Niza;,citation_author=S. Radhika;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=1;,citation_volume=73;,citation_journal_title=Med J Malaysia;">
<meta name="citation_reference" content="citation_title=TSH levels are associated with increased risk of thyroid carcinoma in patients with nodular disease;,citation_author=T. M. Fighera;,citation_author=C. L. Perez;,citation_author=N. Faris;,citation_author=P. C. Scarabotto;,citation_author=T. T. Silva;,citation_author=T. C. Cavalcanti;,citation_author=C. O. Mesa Junior;,citation_author=F. Miasaki;,citation_author=G. J. Paz Filho;,citation_author=G. A. Carvalho;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=6;,citation_doi=10.5603/EP.a2015.0059;,citation_issn=0423-104x;,citation_volume=66;,citation_journal_title=Endokrynol Pol;">
<meta name="citation_reference" content="citation_title=Hashimoto’s thyroiditis is associated with papillary thyroid carcinoma: Role of TSH and of treatment with l-thyroxine;,citation_author=E. Fiore;,citation_author=T. Rago;,citation_author=F. Latrofa;,citation_author=M. A. Provenzale;,citation_author=P. Piaggi;,citation_author=A. Delitala;,citation_author=M. Scutari;,citation_author=F. Basolo;,citation_author=G. Di Coscio;,citation_author=L. Grasso;,citation_author=A. Pinchera;,citation_author=P. Vitti;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_issue=4;,citation_doi=10.1530/erc-11-0028;,citation_issn=1351-0088;,citation_volume=18;,citation_journal_title=Endocr Relat Cancer;">
<meta name="citation_reference" content="citation_title=Lower levels of TSH are associated with a lower risk of papillary thyroid cancer in patients with thyroid nodular disease: Thyroid autonomy may play a protective role;,citation_author=E. Fiore;,citation_author=T. Rago;,citation_author=M. A. Provenzale;,citation_author=M. Scutari;,citation_author=C. Ugolini;,citation_author=F. Basolo;,citation_author=G. Di Coscio;,citation_author=P. Berti;,citation_author=L. Grasso;,citation_author=R. Elisei;,citation_author=A. Pinchera;,citation_author=P. Vitti;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_issue=4;,citation_doi=10.1677/erc-09-0036;,citation_issn=1351-0088;,citation_volume=16;,citation_journal_title=Endocr Relat Cancer;">
<meta name="citation_reference" content="citation_title=Ultrasound imaging classifications of thyroid nodules for malignancy risk stratification and clinical management: State of the art;,citation_author=C. Floridi;,citation_author=M. Cellina;,citation_author=G. Buccimazza;,citation_author=A. Arrichiello;,citation_author=A. Sacrini;,citation_author=F. Arrigoni;,citation_author=G. Pompili;,citation_author=A. Barile;,citation_author=G. Carrafiello;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=Suppl 3;,citation_doi=10.21037/gs.2019.07.01;,citation_issn=2227-684X (Print)
2227-684x;,citation_volume=8;,citation_journal_title=Gland Surg;">
<meta name="citation_reference" content="citation_title=Prevalence and distribution of carcinoma in patients with solitary and multiple thyroid nodules on sonography;,citation_author=M. C. Frates;,citation_author=C. B. Benson;,citation_author=P. M. Doubilet;,citation_author=E. Kunreuther;,citation_author=M. Contreras;,citation_author=E. S. Cibas;,citation_author=J. Orcutt;,citation_author=Jr. Moore;,citation_author=P. R. Larsen;,citation_author=E. Marqusee;,citation_author=E. K. Alexander;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_issue=9;,citation_doi=10.1210/jc.2006-0690;,citation_issn=0021-972X (Print)
0021-972x;,citation_volume=91;,citation_journal_title=J Clin Endocrinol Metab;">
<meta name="citation_reference" content="citation_title=Prevalence of differentiated thyroid cancer in autopsy studies over six decades: A meta-analysis;,citation_author=L. Furuya-Kanamori;,citation_author=K. J. L. Bell;,citation_author=J. Clark;,citation_author=P. Glasziou;,citation_author=S. A. R. Doi;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=30;,citation_doi=10.1200/jco.2016.67.7419;,citation_issn=0732-183x;,citation_volume=34;,citation_journal_title=J Clin Oncol;">
<meta name="citation_reference" content="citation_title=Germline HABP2 mutation causing familial nonmedullary thyroid cancer;,citation_author=S. K. Gara;,citation_author=Maria J. Jia L Fau - Merino;,citation_author=Sunita K. Merino Mj Fau - Agarwal;,citation_author=Lisa Agarwal Sk Fau - Zhang;,citation_author=Maggie Zhang L Fau - Cam;,citation_author=Dhaval Cam M Fau - Patel;,citation_author=Electron Patel D Fau - Kebebew;,citation_author=E. Kebebew;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=5;,citation_volume=373;,citation_journal_title=N Engl J Med;">
<meta name="citation_reference" content="citation_title=Presence of risk factors for thyroid cancer does not improve the performance of ultrasound screening for thyroid nodules in patients with obesity;,citation_author=G. Gimenez-Perez;,citation_author=A. Recasens;,citation_author=S. Chicharro;,citation_author=O. Simo;,citation_author=J. Murillo;,citation_author=I. Castells;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1742-1241
http://ovidsp.ovid.com/ovidweb.cgi?T=JS&amp;amp;amp;PAGE=reference&amp;D=emed23&amp;NEWS=N&amp;AN=2011156961;,citation_issue=7;,citation_doi=https://dx.doi.org/10.1111/ijcp.14211;,citation_issn=1368-5031
1742-1241;,citation_volume=75;,citation_journal_title=International Journal of Clinical Practice;">
<meta name="citation_reference" content="citation_title=A predictive model to distinguish malignant and benign thyroid nodules based on age, gender and ultrasonographic features;,citation_author=F. M. Girardi;,citation_author=L. M. D. Silva;,citation_author=C. D. Flores;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=1;,citation_doi=10.1016/j.bjorl.2017.10.001;,citation_issn=1808-8694 (Print)
1808-8686;,citation_volume=85;,citation_journal_title=Braz J Otorhinolaryngol;">
<meta name="citation_reference" content="citation_title=Thyroid autoimmunity and risk of malignancy in thyroid nodules submitted to fine-needle aspiration cytology;,citation_author=G. Grani;,citation_author=A. Calvanese;,citation_author=G. Carbotta;,citation_author=M. D’Alessandri;,citation_author=A. Nesca;,citation_author=M. Bianchini;,citation_author=M. Del Sordo;,citation_author=M. Vitale;,citation_author=A. Fumarola;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=2;,citation_doi=10.1002/hed.23587;,citation_issn=1043-3074;,citation_volume=37;,citation_journal_title=Head Neck;">
<meta name="citation_reference" content="citation_title=Lack of association between obesity and aggressiveness of differentiated thyroid cancer;,citation_author=G. Grani;,citation_author=L. Lamartina;,citation_author=T. Montesano;,citation_author=G. Ronga;,citation_author=V. Maggisano;,citation_author=R. Falcone;,citation_author=V. Ramundo;,citation_author=L. Giacomelli;,citation_author=C. Durante;,citation_author=D. Russo;,citation_author=M. Maranghi;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=1;,citation_volume=42;,citation_journal_title=J Endocrinol Invest;">
<meta name="citation_reference" content="citation_title=Are endogenously lower serum thyroid hormones new predictors for thyroid malignancy in addition to higher serum thyrotropin?;,citation_author=K. Gul;,citation_author=D. Ozdemir;,citation_author=A. Dirikoc;,citation_author=A. Oguz;,citation_author=D. Tuzun;,citation_author=H. Baser;,citation_author=R. Ersoy;,citation_author=B. Cakir;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=2;,citation_doi=10.1007/s12020-010-9316-6;,citation_issn=1355-008x;,citation_volume=37;,citation_journal_title=Endocrine;">
<meta name="citation_reference" content="citation_title=Development of a machine learning-based fine-grained risk stratification system for thyroid nodules using predefined clinicoradiological features;,citation_author=E. J. Ha;,citation_author=J. H. Lee;,citation_author=D. H. Lee;,citation_author=D. G. Na;,citation_author=J. H. Kim;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=5;,citation_doi=10.1007/s00330-022-09376-0;,citation_issn=0938-7994;,citation_volume=33;,citation_journal_title=Eur Radiol;">
<meta name="citation_reference" content="citation_title=Prevalence of cancer in patients with thyroid nodules in the island of cyprus: Predictive value of ultrasound features and thyroid autoimmune status;,citation_author=I. S. Hadjisavva;,citation_author=R. Dina;,citation_author=M. A. Talias;,citation_author=P. A. Economides;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=2;,citation_doi=10.1159/000430438;,citation_issn=2235-0640 (Print)
2235-0640;,citation_volume=4;,citation_journal_title=Eur Thyroid J;">
<meta name="citation_reference" content="citation_title=Investigating hypocalcaemia;,citation_author=F. M. Hannan;,citation_author=R. V. Thakker;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_doi=10.1136/bmj.f2213;,citation_issn=0959-8138;,citation_volume=346;,citation_journal_title=Bmj;">
<meta name="citation_reference" content="citation_title=2015 american thyroid association management guidelines for adult patients with thyroid nodules and differentiated thyroid cancer: The american thyroid association guidelines task force on thyroid nodules and differentiated thyroid cancer;,citation_author=B. R. Haugen;,citation_author=E. K. Alexander;,citation_author=K. C. Bible;,citation_author=G. M. Doherty;,citation_author=S. J. Mandel;,citation_author=Y. E. Nikiforov;,citation_author=F. Pacini;,citation_author=G. W. Randolph;,citation_author=A. M. Sawka;,citation_author=M. Schlumberger;,citation_author=K. G. Schuff;,citation_author=S. I. Sherman;,citation_author=J. A. Sosa;,citation_author=D. L. Steward;,citation_author=R. M. Tuttle;,citation_author=L. Wartofsky;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=1;,citation_doi=10.1089/thy.2015.0020;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=26;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Thyroid hormones, autoantibodies, ultrasonography, and clinical parameters for predicting thyroid cancer;,citation_author=L. Z. He;,citation_author=T. S. Zeng;,citation_author=L. Pu;,citation_author=S. X. Pan;,citation_author=W. F. Xia;,citation_author=L. L. Chen;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_doi=10.1155/2016/8215834;,citation_issn=1687-8337 (Print)
1687-8337;,citation_volume=2016;,citation_journal_title=Int J Endocrinol;">
<meta name="citation_reference" content="citation_title=Incidence, risk factors, and natural outcome of vocal fold paresis in 920 thyroid operations with routine pre- and postoperative laryngoscopic evaluation;,citation_author=M. Heikkinen;,citation_author=K. Mäkinen;,citation_author=E. Penttilä;,citation_author=M. Qvarnström;,citation_author=T. Kemppainen;,citation_author=H. Löppönen;,citation_author=J. M. Kärkkäinen;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=9;,citation_doi=10.1007/s00268-019-05021-y;,citation_issn=0364-2313;,citation_volume=43;,citation_journal_title=World J Surg;">
<meta name="citation_reference" content="citation_title=Genetic changes in thyroid cancers and the importance of their preoperative detection in relation to the general treatment and determination of the extent of surgical intervention-a review;,citation_author=J. Hlozek;,citation_author=B. Pekova;,citation_author=J. Rotnágl;,citation_author=R. Holý;,citation_author=J. Astl;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=7;,citation_doi=10.3390/biomedicines10071515;,citation_issn=2227-9059 (Print)
2227-9059;,citation_volume=10;,citation_journal_title=Biomedicines;">
<meta name="citation_reference" content="citation_title=An ultrasonogram reporting system for thyroid nodules stratifying cancer risk for clinical management;,citation_author=E. Horvath;,citation_author=Ricardo Majlis S Fau - Rossi;,citation_author=Carmen Rossi R Fau - Franco;,citation_author=Juan P. Franco C Fau - Niedmann;,citation_author=Alex Niedmann Jp Fau - Castro;,citation_author=Miguel Castro A Fau - Dominguez;,citation_author=M. Dominguez;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_issue=5;,citation_volume=94;,citation_journal_title=J Clin Endocrinol Metab;">
<meta name="citation_reference" content="citation_title=Prospective validation of the ultrasound based TIRADS (thyroid imaging reporting and data system) classification: Results in surgically resected thyroid nodules;,citation_author=E. Horvath;,citation_author=C. F. Silva;,citation_author=S. Majlis;,citation_author=I. Rodriguez;,citation_author=V. Skoknic;,citation_author=A. Castro;,citation_author=H. Rojas;,citation_author=J. P. Niedmann;,citation_author=A. Madrid;,citation_author=F. Capdeville;,citation_author=C. Whittle;,citation_author=R. Rossi;,citation_author=M. Domínguez;,citation_author=H. Tala;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=6;,citation_volume=27;,citation_journal_title=Eur Radiol;">
<meta name="citation_reference" content="citation_title=Epidemiological characteristics of thyroid nodules and risk factors for malignant nodules: A retrospective study from 6,304 surgical cases;,citation_author=Q. Huan;,citation_author=K. Wang;,citation_author=F. Lou;,citation_author=L. Zhang;,citation_author=Q. Huang;,citation_author=Y. Han;,citation_author=H. Sun;,citation_author=L. Zhu;,citation_author=P. Lin;,citation_author=J. Song;,citation_author=F. Liu;,citation_author=Q. Wang;,citation_author=W. Hou;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=12;,citation_issn=0366-6999;,citation_volume=127;,citation_journal_title=Chin Med J (Engl);">
<meta name="citation_reference" content="citation_title=Epidemiological survey of thyroid nodules in 2098 patients for routine physical examination in fujian, china;,citation_author=X. Huang;,citation_author=Y. Qiu;,citation_author=Y. Chen;,citation_author=L. Chen;,citation_author=J. Yi;,citation_author=X. Luo;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_doi=10.1155/2022/2913405;,citation_issn=1555-4309 (Print)
1555-4309;,citation_volume=2022;,citation_journal_title=Contrast Media Mol Imaging;">
<meta name="citation_reference" content="citation_title=Risk of thyroid cancer in euthyroid asymptomatic patients with thyroid nodules with an emphasis on family history of thyroid cancer;,citation_author=S. H. Hwang;,citation_author=E. K. Kim;,citation_author=H. J. Moon;,citation_author=J. H. Yoon;,citation_author=J. Y. Kwak;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=2;,citation_doi=10.3348/kjr.2016.17.2.255;,citation_issn=1229-6929 (Print)
1229-6929;,citation_volume=17;,citation_journal_title=Korean J Radiol;">
<meta name="citation_reference" content="citation_title=Radiation exposure and thyroid cancer: A review;,citation_author=M. L. Iglesias;,citation_author=A. Schmidt;,citation_author=A. A. Ghuzlan;,citation_author=L. Lacroix;,citation_author=F. Vathaire;,citation_author=S. Chevillard;,citation_author=M. Schlumberger;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=2;,citation_volume=61;,citation_journal_title=Arch Endocrinol Metab;">
<meta name="citation_reference" content="citation_title=Neural network analysis for evaluating cancer risk in thyroid nodules with an indeterminate diagnosis at aspiration cytology: Identification of a low-risk subgroup;,citation_author=A. M. Ippolito;,citation_author=M. De Laurentiis;,citation_author=G. L. La Rosa;,citation_author=A. Eleuteri;,citation_author=R. Tagliaferri;,citation_author=S. De Placido;,citation_author=R. Vigneri;,citation_author=A. Belfiore;,citation_publication_date=2004;,citation_cover_date=2004;,citation_year=2004;,citation_issue=12;,citation_doi=10.1089/thy.2004.14.1065;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=14;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Increased incidental thyroid cancer in patients with subclinical chronic lymphocytic thyroiditis;,citation_author=D. Jackson;,citation_author=R. S. Handelsman;,citation_author=J. C. Farrá;,citation_author=J. I. Lew;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_doi=10.1016/j.jss.2019.07.025;,citation_issn=0022-4804;,citation_volume=245;,citation_journal_title=J Surg Res;">
<meta name="citation_reference" content="citation_title=Investigating the effect of thyroid nodule location on the risk of thyroid cancer;,citation_author=S. Jasim;,citation_author=T. J. Baranski;,citation_author=S. A. Teefey;,citation_author=W. D. Middleton;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=3;,citation_doi=10.1089/thy.2019.0478;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=30;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=From overdiagnosis to overtreatment of low-risk thyroid cancer: A thematic analysis of attitudes and beliefs of endocrinologists, surgeons, and patients;,citation_author=C. B. Jensen;,citation_author=M. C. Saucke;,citation_author=D. O. Francis;,citation_author=C. I. Voils;,citation_author=S. C. Pitt;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=5;,citation_doi=10.1089/thy.2019.0587;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=30;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Radiotherapy-induced thyroid disorders;,citation_author=B. A. Jereczek-Fossa;,citation_author=D. Alterio;,citation_author=J. Jassem;,citation_author=B. Gibelli;,citation_author=N. Tradati;,citation_author=R. Orecchia;,citation_publication_date=2004;,citation_cover_date=2004;,citation_year=2004;,citation_issue=4;,citation_doi=10.1016/j.ctrv.2003.12.003;,citation_issn=0305-7372 (Print)
0305-7372;,citation_volume=30;,citation_journal_title=Cancer Treat Rev;">
<meta name="citation_reference" content="citation_title=The prevalence of thyroid nodules and an analysis of related lifestyle factors in beijing communities;,citation_author=H. Jiang;,citation_author=Y. Tian;,citation_author=W. Yan;,citation_author=Y. Kong;,citation_author=H. Wang;,citation_author=A. Wang;,citation_author=J. Dou;,citation_author=P. Liang;,citation_author=Y. Mu;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=4;,citation_doi=10.3390/ijerph13040442;,citation_issn=1661-7827 (Print)
1660-4601;,citation_volume=13;,citation_journal_title=Int J Environ Res Public Health;">
<meta name="citation_reference" content="citation_title=The utility of preoperative serum thyroid-stimulating hormone level for predicting malignant nodular thyroid disease;,citation_author=J. Jin;,citation_author=R. Machekano;,citation_author=C. R. McHenry;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=3;,citation_doi=10.1016/j.amjsurg.2009.08.028;,citation_issn=0002-9610;,citation_volume=199;,citation_journal_title=Am J Surg;">
<meta name="citation_reference" content="citation_title=Preoperative platelet distribution width-to-platelet ratio combined with serum thyroglobulin may be objective and popularizable indicators in predicting papillary thyroid carcinoma;,citation_author=J. Jin;,citation_author=G. Wu;,citation_author=C. Ruan;,citation_author=H. Ling;,citation_author=X. Zheng;,citation_author=C. Ying;,citation_author=Y. Zhang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=6;,citation_doi=10.1002/jcla.24443;,citation_issn=0887-8013 (Print)
0887-8013;,citation_volume=36;,citation_journal_title=J Clin Lab Anal;">
<meta name="citation_reference" content="citation_title=Contrast agent retention features in contrast-enhanced ultrasound: Diagnostic performance for the prediction of papillary thyroid carcinoma;,citation_author=Z. Jin;,citation_author=Y. Zhu;,citation_author=F. Xie;,citation_author=Y. Zhang;,citation_author=N. Li;,citation_author=Y. Luo;,citation_author=J. Cao;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_doi=10.1016/j.clinimag.2021.06.030;,citation_issn=0899-7071;,citation_volume=80;,citation_journal_title=Clin Imaging;">
<meta name="citation_reference" content="citation_title=Postoperative chronic hypoparathyroidism and quality of life after total thyroidectomy;,citation_author=C. U. Jørgensen;,citation_author=P. Homøe;,citation_author=M. Dahl;,citation_author=M. F. Hitz;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=4;,citation_doi=10.1002/jbm4.10479;,citation_issn=2473-4039;,citation_volume=5;,citation_journal_title=JBMR Plus;">
<meta name="citation_reference" content="citation_title=Update from the 2022 world health organization classification of thyroid tumors: A standardized diagnostic approach;,citation_author=C. K. Jung;,citation_author=A. Bychkov;,citation_author=K. Kakudo;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=5;,citation_doi=10.3803/EnM.2022.1553;,citation_issn=2093-596X (Print)
2093-596x;,citation_volume=37;,citation_journal_title=Endocrinol Metab (Seoul);">
<meta name="citation_reference" content="citation_title=Artificial intelligence in thyroid fine needle aspiration biopsies;,citation_author=B. Kezlarian;,citation_author=O. Lin;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_doi=10.1159/000512097;,citation_issn=0001-5547;,citation_journal_title=Acta Cytol;">
<meta name="citation_reference" content="citation_title=Low vitamin d status is not associated with thyroid cancer risk;,citation_author=D. Kim;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_volume=6;,citation_journal_title=Journal of Endocrinology and Metabolism;">
<meta name="citation_reference" content="citation_title=Thyroglobulin antibody is associated with increased cancer risk in thyroid nodules;,citation_author=E. S. Kim;,citation_author=D. J. Lim;,citation_author=K. H. Baek;,citation_author=J. M. Lee;,citation_author=M. K. Kim;,citation_author=H. S. Kwon;,citation_author=K. H. Song;,citation_author=M. I. Kang;,citation_author=B. Y. Cha;,citation_author=K. W. Lee;,citation_author=H. Y. Son;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=8;,citation_doi=10.1089/thy.2009.0384;,citation_issn=1050-7256;,citation_volume=20;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Strong association of relatively low and extremely excessive iodine intakes with thyroid cancer in an iodine-replete area;,citation_author=H. J. Kim;,citation_author=N. K. Kim;,citation_author=H. K. Park;,citation_author=D. W. Byun;,citation_author=K. Suh;,citation_author=M. H. Yoo;,citation_author=Y. K. Min;,citation_author=S. W. Kim;,citation_author=J. H. Chung;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=3;,citation_doi=10.1007/s00394-015-1144-2;,citation_issn=1436-6207;,citation_volume=56;,citation_journal_title=Eur J Nutr;">
<meta name="citation_reference" content="citation_title=Thyroid hürthle cell carcinoma: Clinical, pathological, and molecular features;,citation_author=S. Kure;,citation_author=R. Ohashi;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=1;,citation_doi=10.3390/cancers13010026;,citation_issn=2072-6694 (Print)
2072-6694;,citation_volume=13;,citation_journal_title=Cancers (Basel);">
<meta name="citation_reference" content="citation_title=Ultrasound elastography for thyroid nodules: Recent advances;,citation_author=J. Y. Kwak;,citation_author=E. K. Kim;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=2;,citation_doi=10.14366/usg.13025;,citation_issn=2288-5919 (Print)
2288-5919;,citation_volume=33;,citation_journal_title=Ultrasonography;">
<meta name="citation_reference" content="citation_title=The prevalence of vitamin d deficiency is similar between thyroid nodule and thyroid cancer patients;,citation_author=N. Laney;,citation_author=J. Meza;,citation_author=E. Lyden;,citation_author=J. Erickson;,citation_author=K. Treude;,citation_author=W. Goldner;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_doi=10.1155/2010/805716;,citation_issn=1687-8337 (Print)
1687-8337;,citation_volume=2010;,citation_journal_title=Int J Endocrinol;">
<meta name="citation_reference" content="citation_title=Long-term declines of thyroid cancer mortality: An international age-period-cohort analysis;,citation_author=M. Li;,citation_author=J. P. Brito;,citation_author=S. Vaccarella;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=6;,citation_volume=30;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Platelet distribution width and serum albumin levels for discrimination of thyroid cancer from benign thyroid nodules;,citation_author=N. Li;,citation_author=S. Fu;,citation_author=M. M. Cui;,citation_author=Y. Niu;,citation_author=B. Li;,citation_author=Z. P. Liu;,citation_author=T. Liu;,citation_author=R. T. Wang;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=7;,citation_doi=10.22034/apjcp.2017.18.7.1773;,citation_issn=1513-7368 (Print)
1513-7368;,citation_volume=18;,citation_journal_title=Asian Pac J Cancer Prev;">
<meta name="citation_reference" content="citation_title=A new computational model for human thyroid cancer enhances the preoperative diagnostic efficacy;,citation_author=T. Li;,citation_author=J. Sheng;,citation_author=W. Li;,citation_author=X. Zhang;,citation_author=H. Yu;,citation_author=X. Chen;,citation_author=J. Zhang;,citation_author=Q. Cai;,citation_author=Y. Shi;,citation_author=Z. Liu;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=29;,citation_doi=10.18632/oncotarget.4691;,citation_issn=1949-2553;,citation_volume=6;,citation_journal_title=Oncotarget;">
<meta name="citation_reference" content="citation_title=Trends in thyroid cancer incidence and mortality in the united states, 1974-2013;,citation_author=H. Lim;,citation_author=S. S. Devesa;,citation_author=J. A. Sosa;,citation_author=D. Check;,citation_author=C. M. Kitahara;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=13;,citation_doi=10.1001/jama.2017.2719;,citation_issn=0098-7484 (Print)
0098-7484;,citation_volume=317;,citation_journal_title=Jama;">
<meta name="citation_reference" content="citation_title=Oxidative stress mediates the associations between phthalate exposures and thyroid cancer/benign nodule risk;,citation_author=C. Liu;,citation_author=L. Q. Wang;,citation_author=M. Zhang;,citation_author=Y. L. Deng;,citation_author=Q. Luo;,citation_author=E. N. Liu;,citation_author=P. P. Chen;,citation_author=Y. Miao;,citation_author=P. Yang;,citation_author=Q. Zeng;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_doi=10.1016/j.envpol.2023.121462;,citation_issn=0269-7491;,citation_volume=326;,citation_journal_title=Environ Pollut;">
<meta name="citation_reference" content="citation_title=A predictive model of thyroid malignancy using clinical, biochemical and sonographic parameters for patients in a multi-center setting;,citation_author=J. Liu;,citation_author=D. Zheng;,citation_author=Q. Li;,citation_author=X. Tang;,citation_author=Z. Luo;,citation_author=Z. Yuan;,citation_author=L. Gao;,citation_author=J. Zhao;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=1;,citation_doi=10.1186/s12902-018-0241-7;,citation_issn=1472-6823;,citation_volume=18;,citation_journal_title=BMC Endocr Disord;">
<meta name="citation_reference" content="citation_title=A bayesian network for differentiating benign from malignant thyroid nodules using sonographic and demographic features;,citation_author=Y. I. Liu;,citation_author=A. Kamaya;,citation_author=T. S. Desser;,citation_author=D. L. Rubin;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_issue=5;,citation_doi=10.2214/ajr.09.4037;,citation_issn=0361-803x;,citation_volume=196;,citation_journal_title=AJR Am J Roentgenol;">
<meta name="citation_reference" content="citation_title=Papillary thyroid carcinoma: An update;,citation_author=V. A. LiVolsi;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_doi=10.1038/modpathol.2010.129;,citation_issn=0893-3952;,citation_volume=24 Suppl 2;,citation_journal_title=Mod Pathol;">
<meta name="citation_reference" content="citation_title=RET oncogene in MEN2, MEN2B, MTC and other forms of thyroid cancer;,citation_author=M. B. Lodish;,citation_author=C. A. Stratakis;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;,citation_issue=4;,citation_volume=8;,citation_journal_title=Expert Rev Anticancer Ther;">
<meta name="citation_reference" content="citation_title=The bacterial compositions of nasal septal abscess in patients with or without diabetes;,citation_author=C. W. Luan;,citation_author=M. S. Tsai;,citation_author=Y. T. Tsai;,citation_author=C. M. Hsu;,citation_author=C. Y. Liu;,citation_author=Y. H. Yang;,citation_author=C. Y. Wu;,citation_author=G. H. Chang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=12;,citation_doi=10.3390/life12122093;,citation_issn=2075-1729 (Print)
2075-1729;,citation_volume=12;,citation_journal_title=Life (Basel);">
<meta name="citation_reference" content="citation_title=Role of ultrasound, clinical and scintigraphyc parameters to predict malignancy in thyroid nodule;,citation_author=F. F. Maia;,citation_author=P. S. Matos;,citation_author=B. P. Silva;,citation_author=A. T. Pallone;,citation_author=E. J. Pavin;,citation_author=J. Vassallo;,citation_author=D. E. Zantut-Wittmann;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_doi=10.1186/1758-3284-3-17;,citation_issn=1758-3284;,citation_volume=3;,citation_journal_title=Head Neck Oncol;">
<meta name="citation_reference" content="citation_title=Influence of hashimoto thyroiditis on diagnosis and treatment of thyroid nodules;,citation_author=L. Mao;,citation_author=C. Zheng;,citation_author=S. Ou;,citation_author=Y. He;,citation_author=C. Liao;,citation_author=G. Deng;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_doi=10.3389/fendo.2022.1067390;,citation_issn=1664-2392 (Print)
1664-2392;,citation_volume=13;,citation_journal_title=Front Endocrinol (Lausanne);">
<meta name="citation_reference" content="citation_title=Exposure to bisphenol a increases malignancy risk of thyroid nodules in overweight/obese patients;,citation_author=V. Marotta;,citation_author=L. Grumetto;,citation_author=I. Neri;,citation_author=G. Russo;,citation_author=A. Tortora;,citation_author=G. Izzo;,citation_author=I. Panariello;,citation_author=D. Rocco;,citation_author=L. Pezzullo;,citation_author=M. Vitale;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=Pt 1;,citation_doi=10.1016/j.envpol.2022.120478;,citation_issn=0269-7491;,citation_volume=316;,citation_journal_title=Environ Pollut;">
<meta name="citation_reference" content="citation_title=Obesity as a risk factor for thyroid cancer;,citation_author=A. Matrone;,citation_author=Ferruccio Ferrari F Fau - Santini;,citation_author=Rossella Santini F Fau - Elisei;,citation_author=R. Elisei;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=5;,citation_volume=27;,citation_journal_title=Curr Opin Endocrinol Diabetes Obes;">
<meta name="citation_reference" content="citation_title=Pre-existing thyroid autoimmunity and risk of papillary thyroid cancer: A nested case-control study of US active-duty personnel;,citation_author=D. S. A. McLeod;,citation_author=S. A. Bedno;,citation_author=D. S. Cooper;,citation_author=S. M. Hutfless;,citation_author=S. Ippolito;,citation_author=S. J. Jordan;,citation_author=P. G. Matos;,citation_author=R. E. Neale;,citation_author=E. Sabini;,citation_author=D. C. Whiteman;,citation_author=P. W. Ladenson;,citation_author=P. Caturegli;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=23;,citation_doi=10.1200/jco.21.02618;,citation_issn=0732-183x;,citation_volume=40;,citation_journal_title=J Clin Oncol;">
<meta name="citation_reference" content="citation_title=Management of thyroid cancer: United kingdom national multidisciplinary guidelines;,citation_author=A. L. Mitchell;,citation_author=A. Gandhi;,citation_author=D. Scott-Coombes;,citation_author=P. Perros;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=S2;,citation_doi=10.1017/s0022215116000578;,citation_issn=0022-2151 (Print)
0022-2151;,citation_volume=130;,citation_journal_title=J Laryngol Otol;">
<meta name="citation_reference" content="citation_title=The thyroid incidentaloma: An increasingly frequent consequence of radiologic imaging;,citation_author=J. Mitchell;,citation_author=S. Parangi;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_issue=1;,citation_doi=10.1053/j.sult.2004.10.004;,citation_issn=0887-2171 (Print)
0887-2171;,citation_volume=26;,citation_journal_title=Semin Ultrasound CT MR;">
<meta name="citation_reference" content="citation_title=Serum thyrotropin as a risk factor for thyroid malignancy in euthyroid subjects with thyroid micronodule;,citation_author=S. S. Moon;,citation_author=Y. S. Lee;,citation_author=I. K. Lee;,citation_author=J. G. Kim;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_issue=7;,citation_doi=10.1002/hed.21828;,citation_issn=1043-3074;,citation_volume=34;,citation_journal_title=Head Neck;">
<meta name="citation_reference" content="citation_title=Molecular profiling of thyroid nodules-are these findings meaningful, or merely measurable?: A review;,citation_author=L. G. T. Morris;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=9;,citation_volume=146;,citation_journal_title=JAMA Otolaryngol Head Neck Surg;">
<meta name="citation_reference" content="citation_title=Needle biopsy of routine thyroid nodules should be performed using a capillary action technique with 24- to 27-gauge needles: A systematic review and meta-analysis;,citation_author=W. J. Moss;,citation_author=A. Finegersh;,citation_author=J. Pang;,citation_author=J. A. Califano;,citation_author=C. S. Coffey;,citation_author=R. K. Orosco;,citation_author=K. T. Brumund;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=7;,citation_volume=28;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Nasal septal abscess in adult patients - a single center study;,citation_author=N. H. Ngo;,citation_author=N. V. C. Luong;,citation_author=M. T. Q. Le;,citation_author=H. M. H. Nguyen;,citation_author=L. V. Tran;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=http://www.elsevier.com/inca/publications/store/6/2/3/1/1/9/index.htt
http://ovidsp.ovid.com/ovidweb.cgi?T=JS&amp;amp;amp;PAGE=reference&amp;D=emexa&amp;NEWS=N&amp;AN=2023886465;,citation_issue=4;,citation_doi=https://dx.doi.org/10.1016/j.amjoto.2023.103888;,citation_issn=0196-0709
1532-818X;,citation_volume=44;,citation_journal_title=American Journal of Otolaryngology - Head and Neck Medicine and Surgery;">
<meta name="citation_reference" content="citation_title=Nomogram for predicting malignancy in thyroid nodules using clinical, biochemical, ultrasonographic, and cytologic features;,citation_author=I. J. Nixon;,citation_author=I. Ganly;,citation_author=L. E. Hann;,citation_author=O. Lin;,citation_author=C. Yu;,citation_author=S. Brandt;,citation_author=J. P. Shah;,citation_author=A. Shaha;,citation_author=M. W. Kattan;,citation_author=S. G. Patel;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=6;,citation_doi=10.1016/j.surg.2010.09.030;,citation_issn=0039-6060;,citation_volume=148;,citation_journal_title=Surgery;">
<meta name="citation_reference" content="citation_title=Nomogram for selecting thyroid nodules for ultrasound-guided fine-needle aspiration biopsy based on a quantification of risk of malignancy;,citation_author=I. J. Nixon;,citation_author=I. Ganly;,citation_author=L. E. Hann;,citation_author=C. Yu;,citation_author=F. L. Palmer;,citation_author=M. M. Whitcher;,citation_author=J. P. Shah;,citation_author=A. Shaha;,citation_author=M. W. Kattan;,citation_author=S. G. Patel;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_issue=7;,citation_doi=10.1002/hed.23075;,citation_issn=1043-3074;,citation_volume=35;,citation_journal_title=Head Neck;">
<meta name="citation_reference" content="citation_title=Molecular markers guiding thyroid cancer management;,citation_author=C. Nylén;,citation_author=R. Mechera;,citation_author=I. Maréchal-Ross;,citation_author=V. Tsang;,citation_author=A. Chou;,citation_author=A. J. Gill;,citation_author=R. J. Clifton-Bligh;,citation_author=B. G. Robinson;,citation_author=M. S. Sywak;,citation_author=S. B. Sidhu;,citation_author=A. R. Glover;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=8;,citation_doi=10.3390/cancers12082164;,citation_issn=2072-6694 (Print)
2072-6694;,citation_volume=12;,citation_journal_title=Cancers (Basel);">
<meta name="citation_reference" content="citation_title=Large, slowly growing, benign thyroid nodules frequently coexist with synchronous thyroid cancers;,citation_author=R. D. Paparodis;,citation_author=E. Karvounis;,citation_author=D. Bantouna;,citation_author=C. Chourpiliadis;,citation_author=H. Hourpiliadi;,citation_author=S. Livadas;,citation_author=S. Imam;,citation_author=J. C. Jaume;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=8;,citation_doi=10.1210/clinem/dgac242;,citation_issn=0021-972x;,citation_volume=107;,citation_journal_title=J Clin Endocrinol Metab;">
<meta name="citation_reference" content="citation_title=Clinical features of early and late postoperative hypothyroidism after lobectomy;,citation_author=S. Park;,citation_author=M. J. Jeon;,citation_author=E. Song;,citation_author=H. S. Oh;,citation_author=M. Kim;,citation_author=H. Kwon;,citation_author=T. Y. Kim;,citation_author=S. J. Hong;,citation_author=Y. K. Shong;,citation_author=W. B. Kim;,citation_author=T. Y. Sung;,citation_author=W. G. Kim;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=4;,citation_doi=10.1210/jc.2016-3597;,citation_issn=0021-972x;,citation_volume=102;,citation_journal_title=J Clin Endocrinol Metab;">
<meta name="citation_reference" content="citation_title=Association between screening and the thyroid cancer &amp;amp;amp;quot;epidemic&amp;quot; in south korea: Evidence from a nationwide study;,citation_author=S. Park;,citation_author=C. M. Oh;,citation_author=H. Cho;,citation_author=J. Y. Lee;,citation_author=K. W. Jung;,citation_author=J. K. Jun;,citation_author=Y. J. Won;,citation_author=H. J. Kong;,citation_author=K. S. Choi;,citation_author=Y. J. Lee;,citation_author=J. S. Lee;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_volume=355;,citation_journal_title=BMJ;">
<meta name="citation_reference" content="citation_title=Management of low-risk thyroid cancers: Is active surveillance a valid option? A systematic review of the literature;,citation_author=R. Patrone;,citation_author=N. Velotti;,citation_author=S. Masone;,citation_author=A. Conzo;,citation_author=L. Flagiello;,citation_author=C. Cacciatore;,citation_author=M. Filardo;,citation_author=V. Granata;,citation_author=F. Izzo;,citation_author=D. Testa;,citation_author=S. Avenia;,citation_author=A. Sanguinetti;,citation_author=A. Polistena;,citation_author=G. Conzo;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=16;,citation_doi=10.3390/jcm10163569;,citation_issn=2077-0383 (Print)
2077-0383;,citation_volume=10;,citation_journal_title=Journal of Clinical Medicine;">
<meta name="citation_reference" content="citation_title=Deep learning-based artificial intelligence model to assist thyroid nodule diagnosis and management: A multicentre diagnostic study;,citation_author=S. Peng;,citation_author=Y. Liu;,citation_author=W. Lv;,citation_author=L. Liu;,citation_author=Q. Zhou;,citation_author=H. Yang;,citation_author=J. Ren;,citation_author=G. Liu;,citation_author=X. Wang;,citation_author=X. Zhang;,citation_author=Q. Du;,citation_author=F. Nie;,citation_author=G. Huang;,citation_author=Y. Guo;,citation_author=J. Li;,citation_author=J. Liang;,citation_author=H. Hu;,citation_author=H. Xiao;,citation_author=Z. Liu;,citation_author=F. Lai;,citation_author=Q. Zheng;,citation_author=H. Wang;,citation_author=Y. Li;,citation_author=E. K. Alexander;,citation_author=W. Wang;,citation_author=H. Xiao;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=4;,citation_doi=10.1016/s2589-7500(21)00041-8;,citation_issn=2589-7500;,citation_volume=3;,citation_journal_title=Lancet Digital Health;">
<meta name="citation_reference" content="citation_title=Guidelines for the management of thyroid cancer;,citation_author=P. Perros;,citation_author=K. Boelaert;,citation_author=S. Colley;,citation_author=C. Evans;,citation_author=R. M. Evans;,citation_author=G. Gerrard Ba;,citation_author=J. Gilbert;,citation_author=B. Harrison;,citation_author=S. J. Johnson;,citation_author=T. E. Giles;,citation_author=L. Moss;,citation_author=V. Lewington;,citation_author=K. Newbold;,citation_author=J. Taylor;,citation_author=R. V. Thakker;,citation_author=J. Watkinson;,citation_author=G. R. Williams;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_fulltext_html_url=http://dx.doi.org/10.1111/cen.12515;,citation_doi=10.1111/cen.12515;,citation_issn=0300-0664;,citation_volume=81 Suppl 1;,citation_journal_title=Clin Endocrinol (Oxf);">
<meta name="citation_reference" content="citation_title=Predictive value of malignancy of thyroid nodule ultrasound classification systems: A prospective study;,citation_author=A. Persichetti;,citation_author=E. Di Stasio;,citation_author=R. Guglielmi;,citation_author=G. Bizzarri;,citation_author=S. Taccogna;,citation_author=I. Misischi;,citation_author=F. Graziano;,citation_author=L. Petrucci;,citation_author=A. Bianchini;,citation_author=E. Papini;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=4;,citation_volume=103;,citation_journal_title=J Clin Endocrinol Metab;">
<meta name="citation_reference" content="citation_title=Risk of malignancy in the various categories of the UK royal college of pathologists thy terminology for thyroid FNA cytology: A systematic review and meta-analysis;,citation_author=Dn Auid-Orcid Poller;,citation_author=M. Auid-Orcid Bongiovanni;,citation_author=P. Auid-Orcid Trimboli;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=1;,citation_volume=128;,citation_journal_title=Cancer Cytopathol;">
<meta name="citation_reference" content="citation_title=The use of demographic, ultrasonographic and scintigraphic data in the diagnostic approach of thyroid nodules;,citation_author=S. A. Polyzos;,citation_author=M. Kita;,citation_author=Z. Efstathiadou;,citation_author=D. G. Goulis;,citation_author=A. Benos;,citation_author=N. Flaris;,citation_author=M. Leontsini;,citation_author=A. Avramidis;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_issue=4;,citation_doi=10.1055/s-2008-1080922;,citation_issn=0947-7349;,citation_volume=117;,citation_journal_title=Exp Clin Endocrinol Diabetes;">
<meta name="citation_reference" content="citation_title=Serum thyrotropin concentration as a biochemical predictor of thyroid malignancy in patients presenting with thyroid nodules;,citation_author=S. A. Polyzos;,citation_author=M. Kita;,citation_author=Z. Efstathiadou;,citation_author=P. Poulakos;,citation_author=A. Slavakis;,citation_author=D. Sofianou;,citation_author=N. Flaris;,citation_author=M. Leontsini;,citation_author=A. Kourtis;,citation_author=A. Avramidis;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;,citation_issue=9;,citation_doi=10.1007/s00432-008-0373-7;,citation_issn=0171-5216 (Print)
0171-5216;,citation_volume=134;,citation_journal_title=J Cancer Res Clin Oncol;">
<meta name="citation_reference" content="citation_title=Male sex, single nodularity, and young age are associated with the risk of finding a papillary thyroid cancer on fine-needle aspiration cytology in a large series of patients with nodular thyroid disease;,citation_author=T. Rago;,citation_author=E. Fiore;,citation_author=M. Scutari;,citation_author=F. Santini;,citation_author=G. Di Coscio;,citation_author=R. Romani;,citation_author=P. Piaggi;,citation_author=C. Ugolini;,citation_author=F. Basolo;,citation_author=P. Miccoli;,citation_author=A. Pinchera;,citation_author=P. Vitti;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=4;,citation_doi=10.1530/eje-09-0895;,citation_issn=0804-4643;,citation_volume=162;,citation_journal_title=Eur J Endocrinol;">
<meta name="citation_reference" content="citation_title=Thyroid cancer gender disparity;,citation_author=R. Rahbari;,citation_author=Electron Zhang L Fau - Kebebew;,citation_author=E. Kebebew;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=11;,citation_volume=6;,citation_journal_title=Future Oncol;">
<meta name="citation_reference" content="citation_title=Risk factors for well-differentiated thyroid carcinoma in patients with thyroid nodular disease;,citation_author=S. N. Raza;,citation_author=M. D. Shah;,citation_author=C. E. Palme;,citation_author=F. T. Hall;,citation_author=S. Eski;,citation_author=J. L. Freeman;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;,citation_issue=1;,citation_doi=10.1016/j.otohns.2007.10.021;,citation_issn=0194-5998 (Print)
0194-5998;,citation_volume=139;,citation_journal_title=Otolaryngol Head Neck Surg;">
<meta name="citation_reference" content="citation_title=Thyroid ultrasound features and risk of carcinoma: A systematic review and meta-analysis of observational studies;,citation_author=L. R. Remonti;,citation_author=C. K. Kramer;,citation_author=C. B. Leitão;,citation_author=L. C. Pinto;,citation_author=J. L. Gross;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=5;,citation_doi=10.1089/thy.2014.0353;,citation_issn=1050-7256 (Print)
1050-7256;,citation_volume=25;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Ultrasound-guided fine-needle aspiration of thyroid nodules: Stratification of malignancy risk using follicular proliferation grading, clinical and ultrasonographic features;,citation_author=S. Rorive;,citation_author=N. D’Haene;,citation_author=C. Fossion;,citation_author=I. Delpierre;,citation_author=N. Abarguia;,citation_author=F. Avni;,citation_author=C. Decaestecker;,citation_author=I. Salmon;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=6;,citation_doi=10.1530/eje-09-1103;,citation_issn=0804-4643;,citation_volume=162;,citation_journal_title=Eur J Endocrinol;">
<meta name="citation_reference" content="citation_title=Thyroid incidentalomas: Epidemiology, risk stratification with ultrasound and workup;,citation_author=G. Russ;,citation_author=S. Leboulleux;,citation_author=L. Leenhardt;,citation_author=L. Hegedüs;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=3;,citation_doi=10.1159/000365289;,citation_issn=2235-0640 (Print)
2235-0640;,citation_volume=3;,citation_journal_title=Eur Thyroid J;">
<meta name="citation_reference" content="citation_title=Development and internal validation of a predictive model for individual cancer risk assessment for thyroid nodules;,citation_author=F. C. San Laureano;,citation_author=J. J. F. Alba;,citation_author=J. M. J. Heras;,citation_author=A. I. J. Millán;,citation_author=M. T. Fernández-Ladreda;,citation_author=Mdca Ortega;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=10;,citation_doi=10.4158/ep-2020-0004;,citation_issn=1530-891X (Print)
1530-891x;,citation_volume=26;,citation_journal_title=Endocr Pract;">
<meta name="citation_reference" content="citation_title=McGill thyroid nodule score (MTNS): &amp;amp;amp;quot;Rating the risk,&amp;quot; a novel predictive scheme for cancer risk determination;,citation_author=N. B. Sands;,citation_author=S. Karls;,citation_author=A. Amir;,citation_author=M. Tamilia;,citation_author=O. Gologan;,citation_author=L. Rochon;,citation_author=M. J. Black;,citation_author=M. P. Hier;,citation_author=R. J. Payne;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_issn=1916-0208;,citation_volume=40 Suppl 1;,citation_journal_title=J Otolaryngol Head Neck Surg;">
<meta name="citation_reference" content="citation_title=The T4/T3 quotient as a risk factor for differentiated thyroid cancer: A case control study;,citation_author=M. Sasson;,citation_author=E. Kay-Rivest;,citation_author=R. Shoukrun;,citation_author=A. Florea;,citation_author=M. Hier;,citation_author=V. I. Forest;,citation_author=M. Tamilia;,citation_author=R. J. Payne;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=1;,citation_doi=10.1186/s40463-017-0208-0;,citation_issn=1916-0208 (Print)
1916-0208;,citation_volume=46;,citation_journal_title=J Otolaryngol Head Neck Surg;">
<meta name="citation_reference" content="citation_title=Nontoxic diffuse and nodular goiter and thyroid neoplasia;,citation_author=Martin-Jean Schlumberger;,citation_author=Sebastiano Filetti;,citation_author=Ian D Hay;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_journal_title=Williams textbook of endocrinology;">
<meta name="citation_reference" content="citation_title=Artificial intelligence and personalized medicine;,citation_author=N. J. Schork;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_doi=10.1007/978-3-030-16391-4_11;,citation_issn=0927-3042 (Print)
0927-3042;,citation_volume=178;,citation_journal_title=Cancer Treat Res;">
<meta name="citation_reference" content="citation_title=Usefulness of serum thyrotropin for risk prediction of differentiated thyroid cancers does not apply to microcarcinomas: Results of 1,870 chinese patients with thyroid nodules;,citation_author=L. Shi;,citation_author=Y. Li;,citation_author=H. Guan;,citation_author=C. Li;,citation_author=L. Shi;,citation_author=Z. Shan;,citation_author=W. Teng;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_issue=11;,citation_doi=10.1507/endocrj.ej12-0154;,citation_issn=0918-8959;,citation_volume=59;,citation_journal_title=Endocr J;">
<meta name="citation_reference" content="citation_title=Machine learning in medicine: A practical introduction;,citation_author=J. A. M. Sidey-Gibbons;,citation_author=C. J. Sidey-Gibbons;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=1;,citation_doi=10.1186/s12874-019-0681-4;,citation_issn=1471-2288;,citation_volume=19;,citation_journal_title=BMC Med Res Methodol;">
<meta name="citation_reference" content="citation_title=Thyroid nodules: Diagnostic evaluation based on thyroid cancer risk assessment;,citation_author=N. Singh Ospina;,citation_author=N. M. Iñiguez-Ariza;,citation_author=M. R. Castro;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_volume=368;,citation_journal_title=BMJ;">
<meta name="citation_reference" content="citation_title=The role of graves’ disease in the development of thyroid nodules and thyroid cancer;,citation_author=M. N. Soares;,citation_author=M. Borges-Canha;,citation_author=C. Neves;,citation_author=J. S. Neves;,citation_author=D. Carvalho;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=4;,citation_doi=10.1530/etj-23-0055;,citation_issn=2235-0640;,citation_volume=12;,citation_journal_title=Eur Thyroid J;">
<meta name="citation_reference" content="citation_title=Sensitivity to thyroid hormone indices are associated with papillary thyroid carcinoma in chinese patients with thyroid nodules;,citation_author=J. Sun;,citation_author=J. Liu;,citation_author=T. T. Wu;,citation_author=Z. Y. Gu;,citation_author=X. W. Zhang;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=1;,citation_doi=10.1186/s12902-023-01381-8;,citation_issn=1472-6823;,citation_volume=23;,citation_journal_title=BMC Endocr Disord;">
<meta name="citation_reference" content="citation_title=Malignancy risk assessment in patients with thyroid nodules using classification and regression trees;,citation_author=S. Taghipour Zahir;,citation_author=F. Binesh;,citation_author=M. Mirouliaei;,citation_author=E. Khajeh;,citation_author=S. Noshad;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_doi=10.1155/2013/983953;,citation_issn=2090-8067 (Print);,citation_volume=2013;,citation_journal_title=J Thyroid Res;">
<meta name="citation_reference" content="citation_title=Thyroid nodule update on diagnosis and management;,citation_author=S. Tamhane;,citation_author=H. Gharib;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_doi=10.1186/s40842-016-0035-7;,citation_issn=2055-8260 (Print)
2055-8260;,citation_volume=2;,citation_journal_title=Clin Diabetes Endocrinol;">
<meta name="citation_reference" content="citation_title=Feasibility of intravoxel incoherent motion for differentiating benign and malignant thyroid nodules;,citation_author=H. Tan;,citation_author=J. Chen;,citation_author=Y. L. Zhao;,citation_author=J. H. Liu;,citation_author=L. Zhang;,citation_author=C. S. Liu;,citation_author=D. Huang;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=2;,citation_doi=10.1016/j.acra.2018.05.011;,citation_issn=1076-6332;,citation_volume=26;,citation_journal_title=Acad Radiol;">
<meta name="citation_reference" content="citation_title=Development and validation of a diagnostic nomogram for the preoperative differentiation between follicular thyroid carcinoma and follicular thyroid adenomas;,citation_author=P. Tang;,citation_author=C. Ren;,citation_author=L. Shen;,citation_author=Z. Zhou;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=1;,citation_doi=10.1097/rct.0000000000001078;,citation_issn=0363-8715;,citation_volume=45;,citation_journal_title=J Comput Assist Tomogr;">
<meta name="citation_reference" content="citation_title=AIBx, artificial intelligence model to risk stratify thyroid nodules;,citation_author=J. Thomas;,citation_author=T. Haertling;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=6;,citation_doi=10.1089/thy.2019.0752;,citation_issn=1050-7256;,citation_volume=30;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Strong association of high urinary iodine with thyroid nodule and papillary thyroid cancer;,citation_author=F. Wang;,citation_author=Y. Wang;,citation_author=L. Wang;,citation_author=X. Wang;,citation_author=C. Sun;,citation_author=M. Xing;,citation_author=W. Zhao;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=11;,citation_doi=10.1007/s13277-014-2397-8;,citation_issn=1010-4283;,citation_volume=35;,citation_journal_title=Tumour Biol;">
<meta name="citation_reference" content="citation_title=Value of echogenic foci in diagnosing papillary thyroid carcinoma and predicting aggressive biological behavior;,citation_author=G. Wang;,citation_author=F. Nie;,citation_author=Y. Wang;,citation_author=P. Wang;,citation_author=L. Wang;,citation_author=X. Fan;,citation_author=Z. Ma;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=5;,citation_doi=10.1002/jum.15815;,citation_issn=0278-4297;,citation_volume=41;,citation_journal_title=J Ultrasound Med;">
<meta name="citation_reference" content="citation_title=The risk of perchlorate and iodine on the incidence of thyroid tumors and nodular goiter: A case-control study in southeastern china;,citation_author=H. Wang;,citation_author=Y. Jiang;,citation_author=J. Song;,citation_author=H. Liang;,citation_author=Y. Liu;,citation_author=J. Huang;,citation_author=P. Yin;,citation_author=D. Wu;,citation_author=H. Zhang;,citation_author=X. Liu;,citation_author=D. Zhou;,citation_author=W. Wei;,citation_author=L. Lei;,citation_author=J. Peng;,citation_author=J. Zhang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=1;,citation_doi=10.1186/s12940-021-00818-8;,citation_issn=1476-069x;,citation_volume=21;,citation_journal_title=Environ Health;">
<meta name="citation_reference" content="citation_title=An integrated AI model to improve diagnostic accuracy of ultrasound and output known risk features in suspicious thyroid nodules;,citation_author=J. Wang;,citation_author=J. Jiang;,citation_author=D. Zhang;,citation_author=Y. Z. Zhang;,citation_author=L. Guo;,citation_author=Y. Jiang;,citation_author=S. Du;,citation_author=Q. Zhou;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=3;,citation_doi=10.1007/s00330-021-08298-7;,citation_issn=0938-7994;,citation_volume=32;,citation_journal_title=Eur Radiol;">
<meta name="citation_reference" content="citation_title=Preoperative serum thyrotropin to thyroglobulin ratio is effective for thyroid nodule evaluation in euthyroid patients;,citation_author=L. Wang;,citation_author=H. Li;,citation_author=Z. Yang;,citation_author=Z. Guo;,citation_author=Q. Zhang;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=1;,citation_doi=10.1177/0194599815579877;,citation_issn=0194-5998;,citation_volume=153;,citation_journal_title=Otolaryngol Head Neck Surg;">
<meta name="citation_reference" content="citation_title=Rapid diagnosis of papillary thyroid carcinoma with machine learning and probe electrospray ionization mass spectrometry;,citation_author=Y. Wang;,citation_author=Z. Chen;,citation_author=K. Shima;,citation_author=D. Zhong;,citation_author=L. Yang;,citation_author=Q. Wang;,citation_author=R. Jiang;,citation_author=J. Dong;,citation_author=Y. Lei;,citation_author=X. Li;,citation_author=L. Cao;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=6;,citation_doi=10.1002/jms.4831;,citation_issn=1076-5174;,citation_volume=57;,citation_journal_title=J Mass Spectrom;">
<meta name="citation_reference" content="citation_title=Analysis of the correlation between high iodized salt intake and the risk of thyroid nodules: A large retrospective study;,citation_author=Y. Wang;,citation_author=J. Wang;,citation_author=Z. Chen;,citation_author=M. Ma;,citation_author=C. Lin;,citation_author=Q. He;,citation_author=M. Ye;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=1;,citation_doi=10.1186/s12885-021-08700-z;,citation_issn=1471-2407;,citation_volume=21;,citation_journal_title=BMC Cancer;">
<meta name="citation_reference" content="citation_title=Complement C4-a and plasminogen as potential biomarkers for prediction of papillary thyroid carcinoma;,citation_author=Y. Wang;,citation_author=S. Zhou;,citation_author=D. Wang;,citation_author=T. Wei;,citation_author=J. Zhu;,citation_author=Z. Li;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_doi=10.3389/fendo.2021.737638;,citation_issn=1664-2392 (Print)
1664-2392;,citation_volume=12;,citation_journal_title=Front Endocrinol (Lausanne);">
<meta name="citation_reference" content="citation_title=Abscess and hematoma of the septum - old diagnosis, contemporary treatment options;,citation_author=A. Wasilewska;,citation_author=L. Zawadzka-Glos;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=http://www.newmedicine.pl/wp-content/uploads/2021/11/newmed_2021_003-007.pdf
http://ovidsp.ovid.com/ovidweb.cgi?T=JS&amp;amp;amp;PAGE=reference&amp;D=emexa&amp;NEWS=N&amp;AN=2022506400;,citation_issue=1;,citation_doi=https://dx.doi.org/10.25121/NewMed.2021.25.1.3;,citation_issn=1427-0994
1731-2507;,citation_volume=25;,citation_journal_title=New Medicine;">
<meta name="citation_reference" content="citation_title=Systematic review of trends in the incidence rates of thyroid cancer;,citation_author=J. J. Wiltshire;,citation_author=T. M. Drake;,citation_author=L. Uttley;,citation_author=S. P. Balasubramanian;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=11;,citation_doi=10.1089/thy.2016.0100;,citation_issn=1050-7256;,citation_volume=26;,citation_journal_title=Thyroid;">
<meta name="citation_reference" content="citation_title=Predicting malignancy in thyroid nodules: Feasibility of a predictive model integrating clinical, biochemical, and ultrasound characteristics;,citation_author=J. Witczak;,citation_author=P. Taylor;,citation_author=J. Chai;,citation_author=B. Amphlett;,citation_author=J. M. Soukias;,citation_author=G. Das;,citation_author=B. P. Tennant;,citation_author=J. Geen;,citation_author=O. E. Okosieme;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_doi=10.1186/s13044-016-0033-y;,citation_issn=1756-6614 (Print)
1756-6614;,citation_volume=9;,citation_journal_title=Thyroid Res;">
<meta name="citation_reference" content="citation_title=Risk of thyroid cancer and benign nodules associated with exposure to parabens among chinese adults in wuhan, china;,citation_author=N. X. Wu;,citation_author=L. J. Deng;,citation_author=F. Xiong;,citation_author=J. Y. Xie;,citation_author=X. J. Li;,citation_author=Q. Zeng;,citation_author=J. C. Sun;,citation_author=D. Chen;,citation_author=P. Yang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=46;,citation_doi=10.1007/s11356-022-20741-w;,citation_issn=0944-1344;,citation_volume=29;,citation_journal_title=Environ Sci Pollut Res Int;">
<meta name="citation_reference" content="citation_title=External validation of a nomogram that predicts the pathological diagnosis of thyroid nodules in a chinese population;,citation_author=R. Wu;,citation_author=L. Zhu;,citation_author=W. Li;,citation_author=Q. Tang;,citation_author=F. Pan;,citation_author=W. Wu;,citation_author=J. Liu;,citation_author=C. Yao;,citation_author=S. Wang;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_issue=6;,citation_doi=10.1371/journal.pone.0065162;,citation_issn=1932-6203;,citation_volume=8;,citation_journal_title=PLoS One;">
<meta name="citation_reference" content="citation_title=Improving the diagnosis of thyroid cancer by machine learning and clinical data;,citation_author=N. M. Xi;,citation_author=L. Wang;,citation_author=C. Yang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=1;,citation_doi=10.1038/s41598-022-15342-z;,citation_issn=2045-2322;,citation_volume=12;,citation_journal_title=Sci Rep;">
<meta name="citation_reference" content="citation_title=The inverse association of serum magnesium with papillary thyroid cancer in thyroid nodules: A cross-sectional survey based on thyroidectomy population;,citation_author=H. Xu;,citation_author=X. Hu;,citation_author=J. Li;,citation_author=Z. Nie;,citation_author=S. Kang;,citation_author=H. Liu;,citation_author=Y. Wang;,citation_author=X. Jia;,citation_author=Z. Lyu;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=7;,citation_doi=10.1007/s12011-022-03448-4;,citation_issn=0163-4984;,citation_volume=201;,citation_journal_title=Biol Trace Elem Res;">
<meta name="citation_reference" content="citation_title=Relationship between insulin resistance and thyroid cancer in chinese euthyroid subjects without conditions affecting insulin resistance;,citation_author=N. Xu;,citation_author=H. Liu;,citation_author=Y. Wang;,citation_author=Y. Xue;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=1;,citation_doi=10.1186/s12902-022-00943-6;,citation_issn=1472-6823;,citation_volume=22;,citation_journal_title=BMC Endocr Disord;">
<meta name="citation_reference" content="citation_title=Development and validation of a novel diagnostic tool for predicting the malignancy probability of thyroid nodules: A retrospective study based on clinical, b-mode, color doppler and elastographic ultrasonographic characteristics;,citation_author=S. Xu;,citation_author=X. Ni;,citation_author=W. Zhou;,citation_author=W. Zhan;,citation_author=H. Zhang;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_doi=10.3389/fendo.2022.966572;,citation_issn=1664-2392 (Print)
1664-2392;,citation_volume=13;,citation_journal_title=Front Endocrinol (Lausanne);">
<meta name="citation_reference" content="citation_title=A retrospective analysis of thyroid cancer in china;,citation_author=Y. Yang;,citation_author=Q. Li;,citation_author=L. Guo;,citation_author=M. Cui;,citation_author=T. Bao;,citation_author=Z. Zhang;,citation_author=C. Wang;,citation_author=Y. Ni;,citation_author=J. Zhang;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_issue=9;,citation_issn=1513-7368;,citation_volume=12;,citation_journal_title=Asian Pac J Cancer Prev;">
<meta name="citation_reference" content="citation_title=Diagnosis and management of small thyroid nodules: A comparative study with six guidelines for thyroid nodules;,citation_author=J. H. Yoon;,citation_author=K. Han;,citation_author=E. K. Kim;,citation_author=H. J. Moon;,citation_author=J. Y. Kwak;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=2;,citation_doi=10.1148/radiol.2016160641;,citation_issn=0033-8419;,citation_volume=283;,citation_journal_title=Radiology;">
<meta name="citation_reference" content="citation_title=Malignancy risk stratification of thyroid nodules: Comparison between the thyroid imaging reporting and data system and the 2014 american thyroid association management guidelines;,citation_author=J. H. Yoon;,citation_author=H. S. Lee;,citation_author=E. K. Kim;,citation_author=H. J. Moon;,citation_author=J. Y. Kwak;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=3;,citation_volume=278;,citation_journal_title=Radiology;">
<meta name="citation_reference" content="citation_title=Development and validation of a preoperative prediction model for follicular thyroid carcinoma;,citation_author=Q. Yu;,citation_author=K. Liu;,citation_author=C. Xie;,citation_author=D. Ma;,citation_author=Y. Wu;,citation_author=H. Jiang;,citation_author=W. Dai;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=2;,citation_doi=10.1111/cen.14002;,citation_issn=0300-0664;,citation_volume=91;,citation_journal_title=Clin Endocrinol (Oxf);">
<meta name="citation_reference" content="citation_title=Comparison of natural course between thyroid cancer nodules and thyroid benign nodules;,citation_author=K. J. Yun;,citation_author=J. Ha;,citation_author=M. H. Kim;,citation_author=Y. Y. Seo;,citation_author=M. K. Kim;,citation_author=H. S. Kwon;,citation_author=K. H. Song;,citation_author=M. I. Kang;,citation_author=K. H. Baek;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=2;,citation_doi=10.3803/EnM.2019.34.2.195;,citation_issn=2093-596X (Print)
2093-596x;,citation_volume=34;,citation_journal_title=Endocrinol Metab (Seoul);">
<meta name="citation_reference" content="citation_title=Development and validation of a web-based malignancy risk-stratification system of thyroid nodules;,citation_author=B. Zhang;,citation_author=S. Pei;,citation_author=Q. Chen;,citation_author=Y. Dong;,citation_author=L. Zhang;,citation_author=X. Mo;,citation_author=S. Cong;,citation_author=S. Zhang;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=6;,citation_doi=10.1111/cen.14255;,citation_issn=0300-0664;,citation_volume=93;,citation_journal_title=Clin Endocrinol (Oxf);">
<meta name="citation_reference" content="citation_title=THYROID NODULE LOCATION ON ULTRASONOGRAPHY AS a PREDICTOR OF MALIGNANCY;,citation_author=F. Zhang;,citation_author=O. Oluwo;,citation_author=F. B. Castillo;,citation_author=P. Gangula;,citation_author=M. Castillo;,citation_author=F. Farag;,citation_author=S. Zakaria;,citation_author=T. Zahedi;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=2;,citation_doi=10.4158/ep-2018-0361;,citation_issn=1530-891X (Print)
1530-891x;,citation_volume=25;,citation_journal_title=Endocr Pract;">
<meta name="citation_reference" content="citation_title=A case-control study of urinary concentrations of bisphenol a, bisphenol f, and bisphenol s and the risk of papillary thyroid cancer;,citation_author=L. Zhang;,citation_author=J. Zhang;,citation_author=S. Fan;,citation_author=Y. Zhong;,citation_author=J. Li;,citation_author=Y. Zhao;,citation_author=S. Ni;,citation_author=J. Liu;,citation_author=Y. Wu;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=Pt 1;,citation_doi=10.1016/j.chemosphere.2022.137162;,citation_issn=0045-6535;,citation_volume=312;,citation_journal_title=Chemosphere;">
<meta name="citation_reference" content="citation_title=Risk factors and diagnostic prediction models for papillary thyroid carcinoma;,citation_author=X. Zhang;,citation_author=Y. Ze;,citation_author=J. Sang;,citation_author=X. Shi;,citation_author=Y. Bi;,citation_author=S. Shen;,citation_author=X. Zhang;,citation_author=D. Zhu;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_doi=10.3389/fendo.2022.938008;,citation_issn=1664-2392 (Print)
1664-2392;,citation_volume=13;,citation_journal_title=Front Endocrinol (Lausanne);">
<meta name="citation_reference" content="citation_title=Association of obesity with the clinicopathological features of thyroid cancer in a large, operative population: A retrospective case-control study;,citation_author=S. Zhao;,citation_author=X. Jia;,citation_author=X. Fan;,citation_author=L. Zhao;,citation_author=P. Pang;,citation_author=Y. Wang;,citation_author=Y. Luo;,citation_author=F. Wang;,citation_author=G. Yang;,citation_author=X. Wang;,citation_author=W. Gu;,citation_author=L. Zang;,citation_author=Y. Pei;,citation_author=J. Du;,citation_author=J. Ba;,citation_author=J. Dou;,citation_author=Y. Mu;,citation_author=Z. Lyu;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=50;,citation_doi=10.1097/md.0000000000018213;,citation_issn=0025-7974 (Print)
0025-7974;,citation_volume=98;,citation_journal_title=Medicine (Baltimore);">
<meta name="citation_reference" content="citation_title=Comparing the rate and extent of malignancy in surgically excised thyroid nodules across race and ethnicity;,citation_author=H. Zheng;,citation_author=V. Lai;,citation_author=J. Lu;,citation_author=J. K. Kang;,citation_author=E. A. Felger;,citation_author=N. M. Carroll;,citation_author=K. D. Burman;,citation_author=L. Wartofsky;,citation_author=J. E. Rosen;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=4;,citation_doi=10.1016/j.amjsurg.2021.09.018;,citation_issn=0002-9610;,citation_volume=223;,citation_journal_title=Am J Surg;">
</head>

  <body class="quarto-notebook">
    <div id="quarto-embed-header" class="headroom fixed-top bg-primary">
      
      <a onclick="window.quartoBackToArticle(); return false;" class="btn btn-primary quarto-back-link" href=""><i class="bi bi-caret-left"></i> Back to Article</a>
      <h6><i class="bi bi-journal-code"></i> Article Notebook</h6>

            <a href="./index.qmd" class="btn btn-primary quarto-download-embed" download="index.qmd">Download Source</a>
          </div>

     <header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">An investigation of the predictors of thyroid cancer in patients with thyroid nodules</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Ovie Edafe <a href="mailto:ovieedafe@hotmail.co.uk" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0002-6205-806X" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        Department of Oncology &amp; Metabolism, University of Sheffield
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Neil Shephard <a href="https://orcid.org/000-0001-8301-6857" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        Research Software Engineer, Department of Computer Science, University of Sheffield
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Karen Sisley <a href="https://orcid.org/0000-0002-5565-2618" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        Senior Lecturer, Clinical Medicine, School of Medicine and Population Health, University of Sheffield
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Sabapathy P Balasubramanian <a href="https://orcid.org/0000-0001-5953-2843" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        Directorate of General Surgery, Sheffield Teaching Hospitals NHS Foundation Trust
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
                <div>
            <div class="quarto-title-meta-heading">Published</div>
            <div class="quarto-title-meta-contents">
              <p class="date">April 26, 2024</p>
            </div>
          </div>
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      </div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>An abstract summarising the work undertaken and the overall conclusions can be placed here. Sub-headings are currently removed because they conflict with those in the body of the text and mess up the links in the Table of Contents.</p>
      </div>
    </div>

    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>Thyroid nodules, Thyroid cancer</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    </div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active" data-toc-expanded="3">
    <h2 id="toc-title">Table of Contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">1</span> Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">2</span> Methods</a>
  <ul>
  <li><a href="#study-design" id="toc-study-design" class="nav-link" data-scroll-target="#study-design"><span class="header-section-number">2.1</span> Study design</a></li>
  <li><a href="#setting" id="toc-setting" class="nav-link" data-scroll-target="#setting"><span class="header-section-number">2.2</span> Setting</a></li>
  <li><a href="#participants" id="toc-participants" class="nav-link" data-scroll-target="#participants"><span class="header-section-number">2.3</span> Participants</a></li>
  <li><a href="#variables" id="toc-variables" class="nav-link" data-scroll-target="#variables"><span class="header-section-number">2.4</span> Variables</a></li>
  <li><a href="#data-source" id="toc-data-source" class="nav-link" data-scroll-target="#data-source"><span class="header-section-number">2.5</span> Data source</a></li>
  <li><a href="#study-size" id="toc-study-size" class="nav-link" data-scroll-target="#study-size"><span class="header-section-number">2.6</span> Study size</a></li>
  <li><a href="#data-analysis" id="toc-data-analysis" class="nav-link" data-scroll-target="#data-analysis"><span class="header-section-number">2.7</span> Data analysis</a></li>
  <li><a href="#imputation" id="toc-imputation" class="nav-link" data-scroll-target="#imputation"><span class="header-section-number">2.8</span> Imputation</a></li>
  <li><a href="#modelling" id="toc-modelling" class="nav-link" data-scroll-target="#modelling"><span class="header-section-number">2.9</span> Modelling</a>
  <ul class="collapse">
  <li><a href="#lasso-elastic-net" id="toc-lasso-elastic-net" class="nav-link" data-scroll-target="#lasso-elastic-net"><span class="header-section-number">2.9.1</span> LASSO / Elastic Net</a></li>
  <li><a href="#random-forest" id="toc-random-forest" class="nav-link" data-scroll-target="#random-forest"><span class="header-section-number">2.9.2</span> Random Forest</a></li>
  <li><a href="#gradient-boosting" id="toc-gradient-boosting" class="nav-link" data-scroll-target="#gradient-boosting"><span class="header-section-number">2.9.3</span> Gradient Boosting</a></li>
  <li><a href="#svm" id="toc-svm" class="nav-link" data-scroll-target="#svm"><span class="header-section-number">2.9.4</span> SVM</a></li>
  <li><a href="#comparision" id="toc-comparision" class="nav-link" data-scroll-target="#comparision"><span class="header-section-number">2.9.5</span> Comparision</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">3</span> Results</a>
  <ul>
  <li><a href="#data-description" id="toc-data-description" class="nav-link" data-scroll-target="#data-description"><span class="header-section-number">3.1</span> Data Description</a>
  <ul class="collapse">
  <li><a href="#missing-data" id="toc-missing-data" class="nav-link" data-scroll-target="#missing-data"><span class="header-section-number">3.1.1</span> Missing Data</a></li>
  <li><a href="#imputation-1" id="toc-imputation-1" class="nav-link" data-scroll-target="#imputation-1"><span class="header-section-number">3.1.2</span> Imputation</a></li>
  </ul></li>
  <li><a href="#modelling-1" id="toc-modelling-1" class="nav-link" data-scroll-target="#modelling-1"><span class="header-section-number">3.2</span> Modelling</a>
  <ul class="collapse">
  <li><a href="#logistic-regression" id="toc-logistic-regression" class="nav-link" data-scroll-target="#logistic-regression"><span class="header-section-number">3.2.1</span> Logistic Regression</a></li>
  <li><a href="#lasso" id="toc-lasso" class="nav-link" data-scroll-target="#lasso"><span class="header-section-number">3.2.2</span> LASSO</a></li>
  <li><a href="#elastic-net" id="toc-elastic-net" class="nav-link" data-scroll-target="#elastic-net"><span class="header-section-number">3.2.3</span> Elastic Net</a></li>
  <li><a href="#random-forest-1" id="toc-random-forest-1" class="nav-link" data-scroll-target="#random-forest-1"><span class="header-section-number">3.2.4</span> Random Forest</a></li>
  <li><a href="#gradient-boosting-1" id="toc-gradient-boosting-1" class="nav-link" data-scroll-target="#gradient-boosting-1"><span class="header-section-number">3.2.5</span> Gradient Boosting</a></li>
  </ul></li>
  </ul></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">      

       <div class="cell-container"><div class="cell-decorator"><pre>In [1]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Libraries for data manipulation, plotting and tabulating (sorted alphabetically)</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'dplyr'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:stats':

    filter, lag</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union</code></pre>
</div>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggdark)</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gtsummary)</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(Hmisc)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'Hmisc'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:dplyr':

    src, summarize</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:base':

    format.pval, units</code></pre>
</div>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(knitr)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(mice)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'mice'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:stats':

    filter</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:base':

    cbind, rbind</code></pre>
</div>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readr)</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(rmarkdown)</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(visdat)</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(naniar)</span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a><span class="do">## Libraries for Tidymodelling</span></span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dials)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: scales</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'scales'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:readr':

    col_factor</code></pre>
</div>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(kernlab)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'kernlab'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:scales':

    alpha</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:mice':

    convergence</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:ggplot2':

    alpha</code></pre>
</div>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(knitr)</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidymodels)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching packages ────────────────────────────────────── tidymodels 1.2.0 ──</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ broom        1.0.6     ✔ tibble       3.2.1
✔ infer        1.0.7     ✔ tidyr        1.3.1
✔ modeldata    1.4.0     ✔ tune         1.2.1
✔ parsnip      1.2.1     ✔ workflows    1.1.4
✔ purrr        1.0.2     ✔ workflowsets 1.1.0
✔ recipes      1.1.0     ✔ yardstick    1.3.1
✔ rsample      1.2.1     </code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Conflicts ───────────────────────────────────────── tidymodels_conflicts() ──
✖ recipes::all_double()   masks gtsummary::all_double()
✖ recipes::all_factor()   masks gtsummary::all_factor()
✖ recipes::all_integer()  masks gtsummary::all_integer()
✖ recipes::all_logical()  masks gtsummary::all_logical()
✖ recipes::all_numeric()  masks gtsummary::all_numeric()
✖ kernlab::alpha()        masks scales::alpha(), ggplot2::alpha()
✖ purrr::cross()          masks kernlab::cross()
✖ purrr::discard()        masks scales::discard()
✖ mice::filter()          masks dplyr::filter(), stats::filter()
✖ dplyr::lag()            masks stats::lag()
✖ .GlobalEnv::set_names() masks purrr::set_names()
✖ yardstick::spec()       masks readr::spec()
✖ Hmisc::src()            masks dplyr::src()
✖ recipes::step()         masks stats::step()
✖ Hmisc::summarize()      masks dplyr::summarize()
✖ parsnip::translate()    masks Hmisc::translate()
✖ .GlobalEnv::view()      masks tibble::view()
• Use tidymodels_prefer() to resolve common conflicts.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ lubridate 1.9.3     </code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ kernlab::alpha()     masks scales::alpha(), ggplot2::alpha()
✖ scales::col_factor() masks readr::col_factor()
✖ purrr::cross()       masks kernlab::cross()
✖ purrr::discard()     masks scales::discard()
✖ mice::filter()       masks dplyr::filter(), stats::filter()
✖ stringr::fixed()     masks recipes::fixed()
✖ dplyr::lag()         masks stats::lag()
✖ yardstick::spec()    masks readr::spec()
✖ Hmisc::src()         masks dplyr::src()
✖ Hmisc::summarize()   masks dplyr::summarize()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(vip)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'vip'

The following object is masked from 'package:utils':

    vi</code></pre>
</div>
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Set global options</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">digits =</span> <span class="dv">3</span>)</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>train <span class="ot">&lt;-</span> <span class="fl">0.75</span></span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>test <span class="ot">&lt;-</span> <span class="fl">0.25</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a><span class="do">## Set directories based on current location</span></span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>base_dir <span class="ot">&lt;-</span> <span class="fu">getwd</span>()</span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>data_dir <span class="ot">&lt;-</span> <span class="fu">paste</span>(base_dir, <span class="st">"data"</span>, <span class="at">sep =</span> <span class="st">"/"</span>)</span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a>csv_dir <span class="ot">&lt;-</span> <span class="fu">paste</span>(data_dir, <span class="st">"csv"</span>, <span class="at">sep =</span> <span class="st">"/"</span>)</span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a>r_dir <span class="ot">&lt;-</span> <span class="fu">paste</span>(data_dir, <span class="st">"r"</span>, <span class="at">sep =</span> <span class="st">"/"</span>)</span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a>r_scripts <span class="ot">&lt;-</span> <span class="fu">paste</span>(base_dir, <span class="st">"r"</span>, <span class="at">sep =</span> <span class="st">"/"</span>)</span>
<span id="cb31-12"><a href="#cb31-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-13"><a href="#cb31-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-14"><a href="#cb31-14" aria-hidden="true" tabindex="-1"></a><span class="do">## Load data</span></span>
<span id="cb31-15"><a href="#cb31-15" aria-hidden="true" tabindex="-1"></a><span class="do">##</span></span>
<span id="cb31-16"><a href="#cb31-16" aria-hidden="true" tabindex="-1"></a><span class="do">## The following line runs the `r/shf_thy_nod.R` script which reads the data from CSV and cleans/tidies it.</span></span>
<span id="cb31-17"><a href="#cb31-17" aria-hidden="true" tabindex="-1"></a><span class="do">## If something has changed in that file or the underlying data (in `data/csv/sheffield_thyroid_module.R`) then this</span></span>
<span id="cb31-18"><a href="#cb31-18" aria-hidden="true" tabindex="-1"></a><span class="do">## line should be uncommented and the code will run. At the end of the file it saves the data to `data/r/clean.rds`.</span></span>
<span id="cb31-19"><a href="#cb31-19" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">"r/shf_thy_nod.R"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 1364 Columns: 33
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (26): gender, ethnicity, eligibility, incidental_nodule, palpable_nodule...
dbl  (7): study_id, age_at_scan, albumin, tsh_value, lymphocytes, monocyte, ...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a><span class="do">## If nothing has changed in the underlying data or the cleaning/tidying process then the last version of the data,</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a><span class="do">## saved to `data/r/clean.rds` can be loaded by commenting out the line above and uncommenting the line below.</span></span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a><span class="co">#df &lt;- readRDS(paste(r_dir, "clean.rds", sep="/"))</span></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a><span class="do">## @ns-rse 2024-06-18:</span></span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a><span class="do">## We want this table to match the model, therefore rather than repeat ourselves we move the subsetting to give us</span></span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a><span class="do">## df_complete to here and assign to df_complete. This is then passed into gtsummary::tbl_summary() and is available for</span></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a><span class="do">## the next code chunk.</span></span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a><span class="do">##</span></span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a><span class="do">## This is the point at which you should subset your data for those who have data available for the variables of</span></span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a><span class="do">## interest. The variables here should include the outcome `final_pathology` and the predictor variables that are set in</span></span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a><span class="do">## the code chunk `recipe`. May want to move this to another earlier in the processing so that the number of rows can be</span></span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a><span class="do">## counted and reported.</span></span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>df_complete <span class="ot">&lt;-</span> df <span class="sc">|&gt;</span></span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(</span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a>    age_at_scan,</span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a>    gender,</span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a>    ethnicity,</span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>    incidental_nodule,</span>
<span id="cb33-21"><a href="#cb33-21" aria-hidden="true" tabindex="-1"></a>    palpable_nodule,</span>
<span id="cb33-22"><a href="#cb33-22" aria-hidden="true" tabindex="-1"></a>    rapid_enlargement,</span>
<span id="cb33-23"><a href="#cb33-23" aria-hidden="true" tabindex="-1"></a>    compressive_symptoms,</span>
<span id="cb33-24"><a href="#cb33-24" aria-hidden="true" tabindex="-1"></a>    hypertension,</span>
<span id="cb33-25"><a href="#cb33-25" aria-hidden="true" tabindex="-1"></a>    vocal_cord_paresis,</span>
<span id="cb33-26"><a href="#cb33-26" aria-hidden="true" tabindex="-1"></a>    graves_disease,</span>
<span id="cb33-27"><a href="#cb33-27" aria-hidden="true" tabindex="-1"></a>    hashimotos_thyroiditis,</span>
<span id="cb33-28"><a href="#cb33-28" aria-hidden="true" tabindex="-1"></a>    family_history_thyroid_cancer,</span>
<span id="cb33-29"><a href="#cb33-29" aria-hidden="true" tabindex="-1"></a>    exposure_radiation,</span>
<span id="cb33-30"><a href="#cb33-30" aria-hidden="true" tabindex="-1"></a>    albumin,</span>
<span id="cb33-31"><a href="#cb33-31" aria-hidden="true" tabindex="-1"></a>    tsh_value,</span>
<span id="cb33-32"><a href="#cb33-32" aria-hidden="true" tabindex="-1"></a>    lymphocytes,</span>
<span id="cb33-33"><a href="#cb33-33" aria-hidden="true" tabindex="-1"></a>    monocyte,</span>
<span id="cb33-34"><a href="#cb33-34" aria-hidden="true" tabindex="-1"></a>    bta_u_classification,</span>
<span id="cb33-35"><a href="#cb33-35" aria-hidden="true" tabindex="-1"></a>    solitary_nodule,</span>
<span id="cb33-36"><a href="#cb33-36" aria-hidden="true" tabindex="-1"></a>    size_nodule_mm,</span>
<span id="cb33-37"><a href="#cb33-37" aria-hidden="true" tabindex="-1"></a>    cervical_lymphadenopathy,</span>
<span id="cb33-38"><a href="#cb33-38" aria-hidden="true" tabindex="-1"></a>    thy_classification,</span>
<span id="cb33-39"><a href="#cb33-39" aria-hidden="true" tabindex="-1"></a>    final_pathology) <span class="sc">|&gt;</span></span>
<span id="cb33-40"><a href="#cb33-40" aria-hidden="true" tabindex="-1"></a><span class="do">## @ns-rse 2024-06-14 :</span></span>
<span id="cb33-41"><a href="#cb33-41" aria-hidden="true" tabindex="-1"></a><span class="do">## I would consider removing this dplyr::filter() and instead using the recipes::step_filter_missing() as is now in</span></span>
<span id="cb33-42"><a href="#cb33-42" aria-hidden="true" tabindex="-1"></a><span class="do">## place below</span></span>
<span id="cb33-43"><a href="#cb33-43" aria-hidden="true" tabindex="-1"></a><span class="do">## dplyr::filter(if_any(everything(), is.na))</span></span>
<span id="cb33-44"><a href="#cb33-44" aria-hidden="true" tabindex="-1"></a><span class="do">## Instead I think it might be useful to remove individuals who do not have a value for final_pathology here</span></span>
<span id="cb33-45"><a href="#cb33-45" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(final_pathology))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<section id="introduction" class="level2" data-number="1">
<h2 data-number="1" class="anchored" data-anchor-id="introduction"><span class="header-section-number">1</span> Introduction</h2>
<p>Thyroid nodules are common. The challenge in the management of thyroid nodules is differentiating between benign and malignant nodule thyroid nodules.The use fine needle aspiration and cytology (FNAC) still leaves around 20% of patients that cannot be clearly classified as either benign or malignant. This scenario traditionally leads to diagnostic hemithyroidectomy for definitive histology. Other clinical variables such as patients’ demographics, clinical and biochemical factors have been shown to be associated with thyroid cancer in patients with thyroid nodules. This has been utilised in studies evaluating predictors of thyroid cancer with a view of creating a model to aid prediction. Standard practice on the management of thyroid nodules does not utilise these non ultrasound and non cytological factors. Combination of these variables considered to be significant with ultrasound and cytological characteristics may improve management of patients with thyroid nodules. Thyroid nodules are increasingly being incidentally detected with increased use of imaging in the evaluation of non thyroid related pathologies. Thus, leading to increase investigation of thyroid nodules and subsequent increased number of thyroid operations in non diagnostic cases. There are morbidities associated with thyroid surgery including scar, recurrent laryngeal nerve injury, hypothyroidism and hypoparathyroidism. We performed a systematic review to evaluate for predictors of thyroid cancer specifically in patients presenting with thyroid nodules. The systematic review a number of potential important variables that may be useful in the prediction of thyroid cancer in patients with thyroid nodules. The aim of this study was to evaluate the predictors of thyroid cancer with a view of improving prediction of thyroid cancer using computer age statistical inference techniques (<span class="citation" data-cites="Efron2016Jul">Efron and Hastie (<a href="#ref-Efron2016Jul" role="doc-biblioref">2016</a>)</span>).</p>
</section>
<section id="methods" class="level2" data-number="2">
<h2 data-number="2" class="anchored" data-anchor-id="methods"><span class="header-section-number">2</span> Methods</h2>
<p>This study was reported as per the Strengthening the Reporting of Observational Studies in Epidemiology (STROBE) guidelines</p>
<section id="study-design" class="level3" data-number="2.1">
<h3 data-number="2.1" class="anchored" data-anchor-id="study-design"><span class="header-section-number">2.1</span> Study design</h3>
<p>This was a retrospective cohort study.</p>
</section>
<section id="setting" class="level3" data-number="2.2">
<h3 data-number="2.2" class="anchored" data-anchor-id="setting"><span class="header-section-number">2.2</span> Setting</h3>
<p>The study was conducted at the Sheffield Teaching hospitals NHS Foundation Trusts. This is a tertiary referral centre for the management of thyroid cancer</p>
</section>
<section id="participants" class="level3" data-number="2.3">
<h3 data-number="2.3" class="anchored" data-anchor-id="participants"><span class="header-section-number">2.3</span> Participants</h3>
<p>We included all consecutive patients who presented with thyroid nodule(s) or that were found to have thyroid nodule(s) on ultrasound done for thyroid pathology or for other non thyroid related pathologies</p>
</section>
<section id="variables" class="level3" data-number="2.4">
<h3 data-number="2.4" class="anchored" data-anchor-id="variables"><span class="header-section-number">2.4</span> Variables</h3>
<p>Variable evaluated was based on findings from a systematic review evaluating predictors of thyroid cancer in patients with thyroid nodules. Data on the following variables were collected: patient demographics (age, gender, ethnicity), nodule presentation (incidental nodule, palpable nodule, rapid enlargement, compressive symptoms, vocal paresis), past medical history (hypertension, Graves’ disease, Hashimotos’ thyroiditis, family history of thyroid cancer, exposure to neck radiation), biochemistry (thyroid stimulating hormone, lymphocytes, monocytes), ultrasound characteristics (British Thyroid Association ultrasound (BTA U), nodule size, solitary nodule, nodule consistency, cervical lymphadenopathy), Royal College of Pathology (RCP) FNAC classification, type of thyroid surgery, and histological diagnosis.</p>
</section>
<section id="data-source" class="level3" data-number="2.5">
<h3 data-number="2.5" class="anchored" data-anchor-id="data-source"><span class="header-section-number">2.5</span> Data source</h3>
<p>Data was collected from patients’ case notes and electronic patients’ database using a standardised data collection proforma. This was initially piloted on 30 patients and revised to improve data entry. In addition a number of variables that were not standard collected during workout of patients were not further checked; these include body mass index (BMI), serum thyroglobulin, serum triiodothyronine (T3), thyroxine (T4), thyroglobulin antibody (TgAb), thyroid peroxidase antibody (TP0Ab), and urinary iodine.</p>
</section>
<section id="study-size" class="level3" data-number="2.6">
<h3 data-number="2.6" class="anchored" data-anchor-id="study-size"><span class="header-section-number">2.6</span> Study size</h3>
<p>We sought to have a large data set of at least 100 thyroid nodules with a cancer diagnosis using consecutive sampling technique. We aimed for a total of 1500 patients with thyroid nodules to achieve our target sample size. With the use of modern statistical techniques, we proposed such number will be appropriate to detect important variables if it exists.</p>
</section>
<section id="data-analysis" class="level3" data-number="2.7">
<h3 data-number="2.7" class="anchored" data-anchor-id="data-analysis"><span class="header-section-number">2.7</span> Data analysis</h3>
<p>Data was cleaned and analysed using the R Statistical Software <span class="citation" data-cites="r_citation">R Core Team (<a href="#ref-r_citation" role="doc-biblioref">2023</a>)</span> and the Tidyverse (<span class="citation" data-cites="tidyverse">Wickham et al. (<a href="#ref-tidyverse" role="doc-biblioref">2019</a>)</span>), Tidymodels (<span class="citation" data-cites="tidymodels">Kuhn and Wickham (<a href="#ref-tidymodels" role="doc-biblioref">2020</a>)</span>) collection of packages.</p>
</section>
<section id="imputation" class="level3" data-number="2.8">
<h3 data-number="2.8" class="anchored" data-anchor-id="imputation"><span class="header-section-number">2.8</span> Imputation</h3>
<p>The dataset is incomplete and there are missing observations across all variables to varying degrees. In order to maximise the sample available for analysis imputation was used to infer missing values. The Multivariat Imputation via Chained Equations (<a href="https://amices.org/mice/">MICE</a> and implemented in the eponymous R package <span class="citation" data-cites="vanBuuren2011Dec">Buuren and Groothuis-Oudshoorn (<a href="#ref-vanBuuren2011Dec" role="doc-biblioref">2011</a>)</span>) was employed which assumes data is missing at random (a difficult assumption to formally test). The approach takes each variable with missing data and attempts to predict it using statistical modelling based on the observed values. In essence it is the same approach as the statistical methods being employed to try and predict Thyroid Cancer and there are a range of statistical techniques available which include</p>
</section>
<section id="modelling" class="level3" data-number="2.9">
<h3 data-number="2.9" class="anchored" data-anchor-id="modelling"><span class="header-section-number">2.9</span> Modelling</h3>
<p>We used a selection of statistic modelling techniques to evaluate association between variables and thyroid cancer in patients with thyroid nodules. The patient population was split into training and testing cohorts in a ratio of 0.75:0.25 and each model is fitted using the training cohort. This split ratio is generally used in traditional machine learning techniques. The training set of the data was used to estimate the relation between variables and thyroid cancer. The larger the training data, the better it is for the model to learn the trends. The test set was used to determine the accuracy of the model in predicting thyroid cancer; the bigger the test data the more confidence we have in the model prognostic values. We used simple randomisation technique for the split to prevent bias in the data split. We ensured that there was no duplicate in the data sets so any test data was not accidentally trained. Furthermore, cross validation was used to estimate the accuracy of the various machine learning models. The k-fold techniques splits the data in ?10 folds, and the data was trained on all but one of the the fold, and the one fold not trained is used to test the data. This was repeated multiple times using a different fold for test and the others for training until all the folds is utilised for training and testing. Following multiple training process with k-fold, we selected the model that has the best predictive value for thyroid cancer in the test cohort. We also used the leave one out (loo) cross-validation to train and test the data set.In this technique, all but one observation is use to train the data set and one observation is use to test the data; this is repeated until all the data test is used for testing and training. The model with the best predictive value was selected.</p>
<section id="lasso-elastic-net" class="level4" data-number="2.9.1">
<h4 data-number="2.9.1" class="anchored" data-anchor-id="lasso-elastic-net"><span class="header-section-number">2.9.1</span> LASSO / Elastic Net</h4>
<p>LASSO (Least Absolute Shrinkage and Selection Operatror) and Elastic Net <span class="citation" data-cites="zou2005">Zou and Hastie (<a href="#ref-zou2005" role="doc-biblioref">2005</a>)</span> are regression methods that perform variable selection. The original LASSO method proposed by <span class="citation" data-cites="tibshirani1996"><span>“<span class="nocase">Regression Shrinkage and Selection via the Lasso</span>”</span> (<a href="#ref-tibshirani1996" role="doc-biblioref">1996</a>)</span> allows the coefficients for independent/predictor variables to “shrink” down towards zero, effectively eliminating them from influencing the model, this is often referred to as L<sub>1</sub> regularisation. The Elastic Net <span class="citation" data-cites="zou2005">Zou and Hastie (<a href="#ref-zou2005" role="doc-biblioref">2005</a>)</span> improves on the LASSO by balancing L<sub>1</sub> regularisation with ridge-regression or L<sub>2</sub> regularisation which helps avoid over-fitting.</p>
<p>Both methods avoid many of the shortcomings/pitfalls of stepwise variable selection <span class="citation" data-cites="thompson1995">Thompson (<a href="#ref-thompson1995" role="doc-biblioref">1995</a>)</span> <span class="citation" data-cites="smith2018">Smith (<a href="#ref-smith2018" role="doc-biblioref">2018</a>)</span> and have been shown to be more accurate in clinical decision making in small datasets with well code, externally selected variables <span class="citation" data-cites="steyerberg2001">Steyerberg et al. (<a href="#ref-steyerberg2001" role="doc-biblioref">2001</a>)</span></p>
</section>
<section id="random-forest" class="level4" data-number="2.9.2">
<h4 data-number="2.9.2" class="anchored" data-anchor-id="random-forest"><span class="header-section-number">2.9.2</span> Random Forest</h4>
<p>To add reference The random forest plot is an extension of the decision tree methodology to reduce variance. Decision trees are very sensitive to the training data set and can lead to high variance; thus potential issues with generalisation of the model. The random forest plot selects random observation of the dataset to create multiple decision trees. Random variables are selected for each tree in the training of the data set. The aggregated output of the generated decision trees is then used to create an estimate.</p>
</section>
<section id="gradient-boosting" class="level4" data-number="2.9.3">
<h4 data-number="2.9.3" class="anchored" data-anchor-id="gradient-boosting"><span class="header-section-number">2.9.3</span> Gradient Boosting</h4>
<p>Gradient boosting is a machine learning algorithm that uses decision tree as a base model. The data is initially trained on this decision tree, but the initial prediction is weak, thus termed a weak based model. In gradient boosting the process is iterative; a sequence of decision trees is added to the initial tree. Each tree learns from the prior tree(s) to improve the model, increasing strength and minimising error.</p>
</section>
<section id="svm" class="level4" data-number="2.9.4">
<h4 data-number="2.9.4" class="anchored" data-anchor-id="svm"><span class="header-section-number">2.9.4</span> SVM</h4>
<p>Support Vector Machines is an approach that allows observation with a binary classifications to be separated using a hyperplane. It finds a hyperplane that best stratify the two classes i.e benign versus malignant nodules. SVM finds the hyperplane with the maximum margin of separation between the two classes. The support vectors are the data point that are positioned close to the margin of the hyperplane and these used to select the most appropraite hyperplane. The support vectors are the only data points that have an influence on the maximum margin in SVM.</p>
</section>
<section id="comparision" class="level4" data-number="2.9.5">
<h4 data-number="2.9.5" class="anchored" data-anchor-id="comparision"><span class="header-section-number">2.9.5</span> Comparision</h4>
</section>
</section>
</section>
<section id="results" class="level2" data-number="3">
<h2 data-number="3" class="anchored" data-anchor-id="results"><span class="header-section-number">3</span> Results</h2>
<div class="cell-container"><div class="cell-decorator"><pre>In [2]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb34"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a>n_obs <span class="ot">&lt;-</span> <span class="fu">nrow</span>(df)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" href="">Demographics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false" href="">Clinical Characteristics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false" href="">Biomarkers</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false" href="">Ultrasound</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-5" role="tab" aria-controls="tabset-1-5" aria-selected="false" href="">BTA U</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-6" role="tab" aria-controls="tabset-1-6" aria-selected="false" href="">Thyroid Classification</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-7-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-7" role="tab" aria-controls="tabset-1-7" aria-selected="false" href="">Cytology</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<!-- ```{r} -->
<!-- #| label: tbl-patient-demographics -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Demographics of study population" -->
<!-- patient_demo <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("age_at_scan", "gender", "ethnicity")) |> -->
<!--   gtsummary::tbl_summary() |> -->
<!--   gtsummary::modify_caption("Demographics of study population") -->
<!-- patient_demo -->
<!-- ``` -->
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<!-- ```{r} -->
<!-- #| label: tbl-clinical-characteristics -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Clinical characteristics between benign and malignant thyrioid nodules" -->
<!-- clinical_charac <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("age_at_scan", -->
<!--                   "gender", -->
<!--                   "incidental_nodule", -->
<!--                   "palpable_nodule", -->
<!--                   "rapid_enlargement", -->
<!--                   "compressive_symptoms", -->
<!--                   "hypertension", -->
<!--                   "vocal_cord_paresis", -->
<!--                   "graves_disease", -->
<!--                   "hashimotos_thyroiditis", -->
<!--                   "family_history_thyroid_cancer", -->
<!--                   "exposure_radiation", -->
<!--                   "final_pathology", -->
<!--                   )) |> -->
<!--   gtsummary::tbl_summary(by = final_pathology) |> add_p() |> -->
<!--   gtsummary::modify_caption("Clinical characteristics between benign and malignant thyrioid nodules") -->
<!-- clinical_charac -->
<!-- ``` -->
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<!-- ```{r} -->
<!-- #| label: tbl-biomarkers-variables -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Biochemical variables evaluated between benign and malignant thyroid nodules" -->
<!-- biochem_vars <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("albumin", -->
<!--                   "tsh_value", -->
<!--                   "lymphocytes", -->
<!--                   "monocyte", -->
<!--                   "final_pathology")) |> -->
<!--   gtsummary::tbl_summary(by = final_pathology) |> -->
<!--   gtsummary::modify_caption("Biochemical variables evaluated between benign and malignant thyroid nodules") -->
<!-- biochem_vars -->
<!-- ``` -->
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<!-- ```{r} -->
<!-- #| label: tbl-ultrasound-characteristics -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Ultrasound characteristics of benign and malignant nodules" -->
<!-- ultrasound_char <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("size_nodule_mm", -->
<!--                   "solitary_nodule", -->
<!--                   "bta_u_classification", -->
<!--                   "consistency_nodule", -->
<!--                   "cervical_lymphadenopathy", -->
<!--                   "final_pathology")) |> -->
<!--   gtsummary::tbl_summary(by = final_pathology) |> -->
<!--   gtsummary::modify_caption("Ultrasound characteristics of benign and malignant nodules") -->
<!-- ultrasound_char -->
<!-- ``` -->
</div>
<div id="tabset-1-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-5-tab">
<!-- ```{r} -->
<!-- #| label: u-class-final-path -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "U class path" -->
<!-- ultrasound_final_path <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("bta_u_classification", "final_pathology")) |> -->
<!--   gtsummary::tbl_summary(by = bta_u_classification) |> -->
<!--   gtsummary::modify_caption("BTA U classification by final pathology.") -->
<!-- ultrasound_final_path -->
<!-- ``` -->
</div>
<div id="tabset-1-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-6-tab">
<!-- ```{r} -->
<!-- #| label: thy-class-final-path -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Thyroid classification by Final Pathology" -->
<!-- thy_class_final_path <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("thy_classification", "final_pathology")) |> -->
<!--   gtsummary::tbl_summary(by = thy_classification) |> -->
<!--   gtsummary::modify_caption("Thyroid classification by final pathology.") -->
<!-- thy_class_final_path -->
<!-- ``` -->
</div>
<div id="tabset-1-7" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-7-tab">
<!-- ```{r} -->
<!-- #| label: tbl-cytology-characteristics -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Cytological characteristics of benign and malignant nodules" -->
<!-- cytology_char <- df |> -->
<!--   dplyr::ungroup() |> -->
<!--   dplyr::select(c("thy_classification", -->
<!--                   "final_pathology")) |> -->
<!--   gtsummary::tbl_summary(by = final_pathology) |> -->
<!--   gtsummary::modify_caption("Cytological characteristics of benign and malignant nodules") -->
<!-- cytology_char -->
<!-- ``` -->
</div>
</div>
</div>
<p><strong>?@tbl-patient-demographics</strong> shows the demographics of patients included in this study. A total of 1364 patients were included in this study with a median (IQR) age of 55 ( 41-69). <strong>?@tbl-clinical-characteristics</strong> shows the distribution of clinical variables evaluated between benign and malignant thyroid nodules.</p>
<section id="data-description" class="level3" data-number="3.1">
<h3 data-number="3.1" class="anchored" data-anchor-id="data-description"><span class="header-section-number">3.1</span> Data Description</h3>
<p>A summary of the variables that are available in this data set can be found in <a href="#tbl-variables" class="quarto-xref">Table&nbsp;3</a>.</p>
<!-- ```{r} -->
<!-- #| label: tbl-data-completeness -->
<!-- #| purl: true -->
<!-- #| eval: false -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Overall summary of all variables in the Sheffield dataset." -->
<!-- df_summary <- df |> -->
<!--   ## NB - We drop the study_id its not a variable, rather its an identifier -->
<!--   dplyr::select(!(study_id)) |> -->
<!--   dplyr::ungroup() |> -->
<!--   gtsummary::tbl_summary( -->
<!--     type = all_continuous() ~ "continuous2", -->
<!--     statistic = all_continuous() ~ c( -->
<!--       "{N_nonmiss}", -->
<!--       "{mean} ({sd})", -->
<!--       "{median} ({p25}, {p75})", -->
<!--       "{min}, {max}" -->
<!--     ), -->
<!--     percent="column",      # Include percentages for categorical variables by "column", "row" or "cell" -->
<!--     missing="always"           # Exclude missing, options are "no", "ifany" and "always" -->
<!--   ) |> -->
<!--   gtsummary::modify_caption("Baseline characteristics and referral source of included patients") -->
<!-- df_summary -->
<!-- ``` -->
<p>The completeness of the original data is shown in tables <strong>?@tbl-imputation-summary-pmm</strong>, <strong>?@tbl-imputation-summary-cart</strong>, <strong>?@tbl-imputation-summary-rf</strong>, along with summaries from four rounds of imputation for each of three imputation methods. Where variables continuous (e.g.&nbsp;<code>age</code> or <code>size_nodule_mm</code>) basic summary statistics in the form of mean, standard deviation, median and inter-quartile range are given. For categorical variables that are logical <code>TRUE</code>/<code>FALSE</code> (e.g.&nbsp;<code>palpable_nodule</code>) the number of <code>TRUE</code> observations and the percentage (of those with observed data for that variable) are shown along with the number that are <em>Unknown</em>. For categorical variables such as <code>gender</code> percentages in each category are reported. For all variables an indication of the number of missing observations is also given and it is worth noting that there are 214 instances where the <code>final_pathology</code> is not known which reduces the sample size to 1150.</p>
<section id="missing-data" class="level4" data-number="3.1.1">
<h4 data-number="3.1.1" class="anchored" data-anchor-id="missing-data"><span class="header-section-number">3.1.1</span> Missing Data</h4>
<p>More detailed tabulations of missing data by variable are shown in <a href="#tbl-naniar-miss-var-summary" class="quarto-xref">Table&nbsp;1</a> which shows the number and percentage of missing data for each variable and by case in <a href="#tbl-naniar-miss-case-table" class="quarto-xref">Table&nbsp;2</a> which shows how much missing data each case has. A visualisation of this is shown in <a href="#fig-visdat-vis-missing" class="quarto-xref">Figure&nbsp;1</a> .</p>
<p><strong>NB</strong> - Currently there is a <a href="https://github.com/quarto-dev/quarto-cli/issues/10196">bug in the stable release of Quarto</a> which prevents rendering of the missing data figures. It is fixed in development version <a href="https://github.com/quarto-dev/quarto-cli/releases/tag/v1.6.1"><code>v1.6.1</code></a> (currently available as pre-release, so if things don’t render try upgrading).</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true" href="">Variables</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false" href="">Observations</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [3]:</pre></div><div class="cell" data-tbl-caption="Summary of missing data by variable.">

<div class="cell-container"><div class="cell-decorator"><pre>In [4]:</pre></div><div><div class="sourceCode cell-code" id="cb35"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>naniar<span class="sc">::</span><span class="fu">miss_var_summary</span>(df_complete) <span class="sc">|&gt;</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>  knitr<span class="sc">::</span><span class="fu">kable</span>(<span class="at">col.names=</span><span class="fu">c</span>(<span class="st">"Variable"</span>, <span class="st">"N"</span>, <span class="st">"%"</span>),</span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>               <span class="at">caption=</span><span class="st">"Summary of missing data by variable."</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div><div id="tbl-naniar-miss-var-summary" class="cell quarto-float quarto-figure quarto-figure-center anchored" data-tbl-caption="Summary of missing data by variable.">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-naniar-miss-var-summary-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Summary of missing data by variable.
</figcaption>
<div aria-describedby="tbl-naniar-miss-var-summary-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="do-not-create-environment cell caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">Variable</th>
<th style="text-align: right;">N</th>
<th style="text-align: right;">%</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">thy_classification</td>
<td style="text-align: right;">915</td>
<td style="text-align: right;">79.6</td>
</tr>
<tr class="even">
<td style="text-align: left;">albumin</td>
<td style="text-align: right;">515</td>
<td style="text-align: right;">44.8</td>
</tr>
<tr class="odd">
<td style="text-align: left;">tsh_value</td>
<td style="text-align: right;">413</td>
<td style="text-align: right;">35.9</td>
</tr>
<tr class="even">
<td style="text-align: left;">monocyte</td>
<td style="text-align: right;">363</td>
<td style="text-align: right;">31.6</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lymphocytes</td>
<td style="text-align: right;">359</td>
<td style="text-align: right;">31.2</td>
</tr>
<tr class="even">
<td style="text-align: left;">size_nodule_mm</td>
<td style="text-align: right;">319</td>
<td style="text-align: right;">27.7</td>
</tr>
<tr class="odd">
<td style="text-align: left;">family_history_thyroid_cancer</td>
<td style="text-align: right;">281</td>
<td style="text-align: right;">24.4</td>
</tr>
<tr class="even">
<td style="text-align: left;">hypertension</td>
<td style="text-align: right;">126</td>
<td style="text-align: right;">11.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;">exposure_radiation</td>
<td style="text-align: right;">121</td>
<td style="text-align: right;">10.5</td>
</tr>
<tr class="even">
<td style="text-align: left;">compressive_symptoms</td>
<td style="text-align: right;">106</td>
<td style="text-align: right;">9.22</td>
</tr>
<tr class="odd">
<td style="text-align: left;">vocal_cord_paresis</td>
<td style="text-align: right;">76</td>
<td style="text-align: right;">6.61</td>
</tr>
<tr class="even">
<td style="text-align: left;">hashimotos_thyroiditis</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">6.35</td>
</tr>
<tr class="odd">
<td style="text-align: left;">graves_disease</td>
<td style="text-align: right;">67</td>
<td style="text-align: right;">5.83</td>
</tr>
<tr class="even">
<td style="text-align: left;">palpable_nodule</td>
<td style="text-align: right;">58</td>
<td style="text-align: right;">5.04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">bta_u_classification</td>
<td style="text-align: right;">50</td>
<td style="text-align: right;">4.35</td>
</tr>
<tr class="even">
<td style="text-align: left;">ethnicity</td>
<td style="text-align: right;">48</td>
<td style="text-align: right;">4.17</td>
</tr>
<tr class="odd">
<td style="text-align: left;">rapid_enlargement</td>
<td style="text-align: right;">43</td>
<td style="text-align: right;">3.74</td>
</tr>
<tr class="even">
<td style="text-align: left;">cervical_lymphadenopathy</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">0.783</td>
</tr>
<tr class="odd">
<td style="text-align: left;">solitary_nodule</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.696</td>
</tr>
<tr class="even">
<td style="text-align: left;">incidental_nodule</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">0.435</td>
</tr>
<tr class="odd">
<td style="text-align: left;">age_at_scan</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="even">
<td style="text-align: left;">gender</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="odd">
<td style="text-align: left;">final_pathology</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">0</td>
</tr>
</tbody>
</table>
</div>
</div>
</figure>
</div></div></div>
</div></div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [5]:</pre></div><div class="cell" data-tbl-caption="Summary of missing data by case, how much missing data is there per person?">

<div class="cell-container"><div class="cell-decorator"><pre>In [6]:</pre></div><div><div class="sourceCode cell-code" id="cb36"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a>naniar<span class="sc">::</span><span class="fu">miss_case_table</span>(df_complete) <span class="sc">|&gt;</span></span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>  knitr<span class="sc">::</span><span class="fu">kable</span>(<span class="at">col.names=</span><span class="fu">c</span>(<span class="st">"Missing Variables"</span>, <span class="st">"N"</span>, <span class="st">"%"</span>),</span>
<span id="cb36-3"><a href="#cb36-3" aria-hidden="true" tabindex="-1"></a>               <span class="at">caption=</span><span class="st">"Summary of missing data by case, how much missing data is there per person?"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div><div id="tbl-naniar-miss-case-table" class="cell quarto-float quarto-figure quarto-figure-center anchored" data-tbl-caption="Summary of missing data by case, how much missing data is there per person?">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-naniar-miss-case-table-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Summary of missing data by case, how much missing data is there per person?
</figcaption>
<div aria-describedby="tbl-naniar-miss-case-table-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="do-not-create-environment cell caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: right;">Missing Variables</th>
<th style="text-align: right;">N</th>
<th style="text-align: right;">%</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: right;">0</td>
<td style="text-align: right;">65</td>
<td style="text-align: right;">5.652</td>
</tr>
<tr class="even">
<td style="text-align: right;">1</td>
<td style="text-align: right;">227</td>
<td style="text-align: right;">19.739</td>
</tr>
<tr class="odd">
<td style="text-align: right;">2</td>
<td style="text-align: right;">229</td>
<td style="text-align: right;">19.913</td>
</tr>
<tr class="even">
<td style="text-align: right;">3</td>
<td style="text-align: right;">181</td>
<td style="text-align: right;">15.739</td>
</tr>
<tr class="odd">
<td style="text-align: right;">4</td>
<td style="text-align: right;">139</td>
<td style="text-align: right;">12.087</td>
</tr>
<tr class="even">
<td style="text-align: right;">5</td>
<td style="text-align: right;">102</td>
<td style="text-align: right;">8.870</td>
</tr>
<tr class="odd">
<td style="text-align: right;">6</td>
<td style="text-align: right;">76</td>
<td style="text-align: right;">6.609</td>
</tr>
<tr class="even">
<td style="text-align: right;">7</td>
<td style="text-align: right;">35</td>
<td style="text-align: right;">3.043</td>
</tr>
<tr class="odd">
<td style="text-align: right;">8</td>
<td style="text-align: right;">30</td>
<td style="text-align: right;">2.609</td>
</tr>
<tr class="even">
<td style="text-align: right;">9</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">1.130</td>
</tr>
<tr class="odd">
<td style="text-align: right;">10</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">1.652</td>
</tr>
<tr class="even">
<td style="text-align: right;">11</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">1.304</td>
</tr>
<tr class="odd">
<td style="text-align: right;">12</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">0.783</td>
</tr>
<tr class="even">
<td style="text-align: right;">13</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">0.348</td>
</tr>
<tr class="odd">
<td style="text-align: right;">14</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">0.261</td>
</tr>
<tr class="even">
<td style="text-align: right;">15</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">0.174</td>
</tr>
<tr class="odd">
<td style="text-align: right;">16</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.087</td>
</tr>
</tbody>
</table>
</div>
</div>
</figure>
</div></div></div>
</div></div>
</div>
</div>
</div>
<div class="cell-container"><div class="cell-decorator"><pre>In [7]:</pre></div><div id="cell-fig-visdat-vis-missing" class="cell">
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>visdat<span class="sc">::</span><span class="fu">vis_miss</span>(df_complete)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-visdat-vis-missing" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-visdat-vis-missing-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-visdat-vis-missing-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1: "><img src="index_files/figure-html/fig-visdat-vis-missing-1.png" id="fig-visdat-vis-missing" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-visdat-vis-missing-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
<p>The <a href="https://amices.org/mice/">MICE</a> package also provides tools for visualising missing data and these are shown in figures <a href="#fig-mice-vis-missing-clinical" class="quarto-xref">Figure&nbsp;2</a>, <strong>?@fig-mice-vis-missing-biomarker</strong> and <a href="#fig-mice-vis-missing-ultrasound" class="quarto-xref">Figure&nbsp;4</a>.</p>
<p>The columns of these plots, labelled along the top, show the variable, if a cell is blue it indicates data is present, if it is red it indicates there is missing data. The left-hand side shows the total number of observations for that rows particular combination of variables with number of missing variables indicated on the right. The first row shows that for these variables there are 604 observations with zero missing data across the listed variables, the second row indicates there are 166 observations with <em>just</em> <code>family_history_thyroid_cancer</code> but there are some with this missing <em>and other variables</em>. The numbers on the bottom of the figure indicate the total number of missing observations for that variable (e.g.&nbsp;for <code>family_history_thyroid_cancer</code> there is a total of 281 missing observations).</p>
<p><strong>TODO</strong> - Workout why <code>out-width: "80%"</code> isn’t applied to these figures and/or how to make the <code>All</code> figure readable.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true" href="">Clinical</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false" href="">Biomarkers</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-3" role="tab" aria-controls="tabset-3-3" aria-selected="false" href="">Ultrasound</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-4" role="tab" aria-controls="tabset-3-4" aria-selected="false" href="">All</a></li></ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [8]:</pre></div><div id="cell-fig-mice-vis-missing-clinical" class="cell" data-fig-caption="Patterns of missing data for clinical variables.">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a>mice_missing_clinical <span class="ot">&lt;-</span> df_complete <span class="sc">|&gt;</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(final_pathology,</span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a>                age_at_scan,</span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>                gender,</span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>                incidental_nodule,</span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a>                palpable_nodule,</span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a>                rapid_enlargement,</span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a>                compressive_symptoms,</span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a>                hashimotos_thyroiditis,</span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a>                family_history_thyroid_cancer,</span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>                cervical_lymphadenopathy) <span class="sc">|&gt;</span></span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a>  mice<span class="sc">::</span><span class="fu">md.pattern</span>(<span class="at">rotate.names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-vis-missing-clinical" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-vis-missing-clinical-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-vis-missing-clinical-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;2: "><img src="index_files/figure-html/fig-mice-vis-missing-clinical-1.png" id="fig-mice-vis-missing-clinical" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-vis-missing-clinical-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [9]:</pre></div><div id="cell-fig-mice-vis-missing-biomarkers" class="cell" data-fig-caption="Patterns of missing data for biomarkers.">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>mice_missing_biomarkers <span class="ot">&lt;-</span> df_complete <span class="sc">|&gt;</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(final_pathology,</span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>                age_at_scan,</span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>                gender,</span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>                tsh_value,</span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>                albumin,</span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>                lymphocytes,</span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>                monocyte) <span class="sc">|&gt;</span></span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>  mice<span class="sc">::</span><span class="fu">md.pattern</span>(<span class="at">rotate.names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-vis-missing-biomarkers" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-vis-missing-biomarkers-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-vis-missing-biomarkers-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;3: "><img src="index_files/figure-html/fig-mice-vis-missing-biomarkers-1.png" id="fig-mice-vis-missing-biomarkers" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-vis-missing-biomarkers-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-3-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-3-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [10]:</pre></div><div id="cell-fig-mice-vis-missing-ultrasound" class="cell" data-fig-caption="Patterns of missing data for ultrasound.">
<div class="sourceCode cell-code" id="cb40"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1" aria-hidden="true" tabindex="-1"></a>mice_missing_ultrasound <span class="ot">&lt;-</span> df_complete <span class="sc">|&gt;</span></span>
<span id="cb40-2"><a href="#cb40-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(final_pathology,</span>
<span id="cb40-3"><a href="#cb40-3" aria-hidden="true" tabindex="-1"></a>                age_at_scan,</span>
<span id="cb40-4"><a href="#cb40-4" aria-hidden="true" tabindex="-1"></a>                gender,</span>
<span id="cb40-5"><a href="#cb40-5" aria-hidden="true" tabindex="-1"></a>                bta_u_classification,</span>
<span id="cb40-6"><a href="#cb40-6" aria-hidden="true" tabindex="-1"></a>                thy_classification,</span>
<span id="cb40-7"><a href="#cb40-7" aria-hidden="true" tabindex="-1"></a>                size_nodule_mm,</span>
<span id="cb40-8"><a href="#cb40-8" aria-hidden="true" tabindex="-1"></a>                solitary_nodule) <span class="sc">|&gt;</span></span>
<span id="cb40-9"><a href="#cb40-9" aria-hidden="true" tabindex="-1"></a>  mice<span class="sc">::</span><span class="fu">md.pattern</span>(<span class="at">rotate.names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-vis-missing-ultrasound" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-vis-missing-ultrasound-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-vis-missing-ultrasound-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;4: "><img src="index_files/figure-html/fig-mice-vis-missing-ultrasound-1.png" id="fig-mice-vis-missing-ultrasound" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-vis-missing-ultrasound-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-3-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-4-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [11]:</pre></div><div id="cell-fig-mice-vis-missing-all" class="cell" data-fig-caption="Patterns of missing data across all variables.">
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a>mice_missing_ultrasound <span class="ot">&lt;-</span> df_complete <span class="sc">|&gt;</span></span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(final_pathology,</span>
<span id="cb41-3"><a href="#cb41-3" aria-hidden="true" tabindex="-1"></a>                age_at_scan,</span>
<span id="cb41-4"><a href="#cb41-4" aria-hidden="true" tabindex="-1"></a>                gender,</span>
<span id="cb41-5"><a href="#cb41-5" aria-hidden="true" tabindex="-1"></a>                incidental_nodule,</span>
<span id="cb41-6"><a href="#cb41-6" aria-hidden="true" tabindex="-1"></a>                palpable_nodule,</span>
<span id="cb41-7"><a href="#cb41-7" aria-hidden="true" tabindex="-1"></a>                rapid_enlargement,</span>
<span id="cb41-8"><a href="#cb41-8" aria-hidden="true" tabindex="-1"></a>                compressive_symptoms,</span>
<span id="cb41-9"><a href="#cb41-9" aria-hidden="true" tabindex="-1"></a>                hashimotos_thyroiditis,</span>
<span id="cb41-10"><a href="#cb41-10" aria-hidden="true" tabindex="-1"></a>                family_history_thyroid_cancer,</span>
<span id="cb41-11"><a href="#cb41-11" aria-hidden="true" tabindex="-1"></a>                cervical_lymphadenopathy,</span>
<span id="cb41-12"><a href="#cb41-12" aria-hidden="true" tabindex="-1"></a>                tsh_value,</span>
<span id="cb41-13"><a href="#cb41-13" aria-hidden="true" tabindex="-1"></a>                albumin,</span>
<span id="cb41-14"><a href="#cb41-14" aria-hidden="true" tabindex="-1"></a>                lymphocytes,</span>
<span id="cb41-15"><a href="#cb41-15" aria-hidden="true" tabindex="-1"></a>                monocyte,</span>
<span id="cb41-16"><a href="#cb41-16" aria-hidden="true" tabindex="-1"></a>                bta_u_classification,</span>
<span id="cb41-17"><a href="#cb41-17" aria-hidden="true" tabindex="-1"></a>                thy_classification,</span>
<span id="cb41-18"><a href="#cb41-18" aria-hidden="true" tabindex="-1"></a>                size_nodule_mm,</span>
<span id="cb41-19"><a href="#cb41-19" aria-hidden="true" tabindex="-1"></a>                solitary_nodule) <span class="sc">|&gt;</span></span>
<span id="cb41-20"><a href="#cb41-20" aria-hidden="true" tabindex="-1"></a>  mice<span class="sc">::</span><span class="fu">md.pattern</span>(<span class="at">rotate.names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-vis-missing-all" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-vis-missing-all-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-vis-missing-all-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="Figure&nbsp;5: "><img src="index_files/figure-html/fig-mice-vis-missing-all-1.png" id="fig-mice-vis-missing-all" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-vis-missing-all-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
</div>
</div>
</section>
<section id="imputation-1" class="level4" data-number="3.1.2">
<h4 data-number="3.1.2" class="anchored" data-anchor-id="imputation-1"><span class="header-section-number">3.1.2</span> Imputation</h4>
<p>The <a href="https://amices.org/mice/">MICE</a> package@mice offers a number of different methods for imputing variables (see [documentation][mice_details]) we have investigated Predictive Mean Matching (PMM), Classification and Regression Trees (CART) and Random Forests (RF). Four rounds of imputation using each method were made.</p>
<p>A comparison of distributions/proportions before and after imputation are presented below to allow assessment of the utility of each method.</p>
<div class="cell-container"><div class="cell-decorator"><pre>In [12]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb42"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb42-1"><a href="#cb42-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Setup MICE mids for various methods</span></span>
<span id="cb42-2"><a href="#cb42-2" aria-hidden="true" tabindex="-1"></a><span class="do">##</span></span>
<span id="cb42-3"><a href="#cb42-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb42-4"><a href="#cb42-4" aria-hidden="true" tabindex="-1"></a><span class="co">#' Impute missing data and plot the results using the mice package.</span></span>
<span id="cb42-5"><a href="#cb42-5" aria-hidden="true" tabindex="-1"></a><span class="co">#'</span></span>
<span id="cb42-6"><a href="#cb42-6" aria-hidden="true" tabindex="-1"></a><span class="co">#' This is a wrapper around the functionality of the \href{https://amices.org/mice}{mice} and</span></span>
<span id="cb42-7"><a href="#cb42-7" aria-hidden="true" tabindex="-1"></a><span class="co">#' \href{https://amices.org/ggmice}{ggmice} packages for Multivariate Imputation by Chained Equations and visualisation</span></span>
<span id="cb42-8"><a href="#cb42-8" aria-hidden="true" tabindex="-1"></a><span class="co">#' of the resulting imputed datasets. Users should refer to the documentation for details of the options available.</span></span>
<span id="cb42-9"><a href="#cb42-9" aria-hidden="true" tabindex="-1"></a><span class="co">#'</span></span>
<span id="cb42-10"><a href="#cb42-10" aria-hidden="true" tabindex="-1"></a><span class="co">#' The wrapper uses \code{\link[mice]{futuremice}} to perform imputation in parallel using the same number of cores as</span></span>
<span id="cb42-11"><a href="#cb42-11" aria-hidden="true" tabindex="-1"></a><span class="co">#' the requested number of imputations. This speeds up the process but be wary if your computer has limited cores.</span></span>
<span id="cb42-12"><a href="#cb42-12" aria-hidden="true" tabindex="-1"></a><span class="co">#'</span></span>
<span id="cb42-13"><a href="#cb42-13" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param df data.frame|tibble Data frame or tibble of original data.</span></span>
<span id="cb42-14"><a href="#cb42-14" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param imputations int Number of imputations to perform.</span></span>
<span id="cb42-15"><a href="#cb42-15" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param iterations int Number of iterations to perform for imputation.</span></span>
<span id="cb42-16"><a href="#cb42-16" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param method str Method of imputation to use, see the Details section of \code{\link[mice]{mice}}.</span></span>
<span id="cb42-17"><a href="#cb42-17" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param action str Action to take when running \code{\link[mice]{complete}}.</span></span>
<span id="cb42-18"><a href="#cb42-18" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param include bool Logical of whether to include the original dataset in the output.</span></span>
<span id="cb42-19"><a href="#cb42-19" aria-hidden="true" tabindex="-1"></a><span class="co">#' @param seed int Seed for random number generation</span></span>
<span id="cb42-20"><a href="#cb42-20" aria-hidden="true" tabindex="-1"></a><span class="co">#'</span></span>
<span id="cb42-21"><a href="#cb42-21" aria-hidden="true" tabindex="-1"></a><span class="co">#'</span></span>
<span id="cb42-22"><a href="#cb42-22" aria-hidden="true" tabindex="-1"></a>impute_data <span class="ot">&lt;-</span> <span class="cf">function</span>(<span class="at">df =</span> df_complete,</span>
<span id="cb42-23"><a href="#cb42-23" aria-hidden="true" tabindex="-1"></a>                        <span class="at">outcome_var =</span> <span class="st">"final_pathology"</span>,</span>
<span id="cb42-24"><a href="#cb42-24" aria-hidden="true" tabindex="-1"></a>                        <span class="at">imputations =</span> <span class="dv">4</span>,</span>
<span id="cb42-25"><a href="#cb42-25" aria-hidden="true" tabindex="-1"></a>                        <span class="at">iterations =</span> <span class="dv">5</span>,</span>
<span id="cb42-26"><a href="#cb42-26" aria-hidden="true" tabindex="-1"></a>                        <span class="at">method =</span> <span class="st">"pmm"</span>,</span>
<span id="cb42-27"><a href="#cb42-27" aria-hidden="true" tabindex="-1"></a>                        <span class="at">action =</span> <span class="st">"long"</span>,</span>
<span id="cb42-28"><a href="#cb42-28" aria-hidden="true" tabindex="-1"></a>                        <span class="at">include =</span> <span class="cn">TRUE</span>,</span>
<span id="cb42-29"><a href="#cb42-29" aria-hidden="true" tabindex="-1"></a>                        <span class="at">continuous =</span> <span class="fu">c</span>(<span class="st">"albumin"</span>, <span class="st">"tsh_value"</span>, <span class="st">"lymphocytes"</span>, <span class="st">"monocyte"</span>, <span class="st">"size_nodule_mm"</span>),</span>
<span id="cb42-30"><a href="#cb42-30" aria-hidden="true" tabindex="-1"></a>                        <span class="at">categorical =</span> <span class="fu">c</span>(<span class="st">"ethnicity"</span>,</span>
<span id="cb42-31"><a href="#cb42-31" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"incidental_nodule"</span>,</span>
<span id="cb42-32"><a href="#cb42-32" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"palpable_nodule"</span>,</span>
<span id="cb42-33"><a href="#cb42-33" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"rapid_enlargement"</span>,</span>
<span id="cb42-34"><a href="#cb42-34" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"compressive_symptoms"</span>,</span>
<span id="cb42-35"><a href="#cb42-35" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"hypertension"</span>,</span>
<span id="cb42-36"><a href="#cb42-36" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"vocal_cord_paresis"</span>,</span>
<span id="cb42-37"><a href="#cb42-37" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"graves_disease"</span>,</span>
<span id="cb42-38"><a href="#cb42-38" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"hashimotos_thyroiditis"</span>,</span>
<span id="cb42-39"><a href="#cb42-39" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"family_history_thyroid_cancer"</span>,</span>
<span id="cb42-40"><a href="#cb42-40" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"exposure_radiation"</span>,</span>
<span id="cb42-41"><a href="#cb42-41" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"bta_u_classification"</span>,</span>
<span id="cb42-42"><a href="#cb42-42" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"solitary_nodule"</span>,</span>
<span id="cb42-43"><a href="#cb42-43" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"cervical_lymphadenopathy"</span>,</span>
<span id="cb42-44"><a href="#cb42-44" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"thy_classification"</span>),</span>
<span id="cb42-45"><a href="#cb42-45" aria-hidden="true" tabindex="-1"></a>                        <span class="at">seed =</span> <span class="dv">123</span>) {</span>
<span id="cb42-46"><a href="#cb42-46" aria-hidden="true" tabindex="-1"></a>    results <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb42-47"><a href="#cb42-47" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Setup imputation</span></span>
<span id="cb42-48"><a href="#cb42-48" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>mids <span class="ot">&lt;-</span> df <span class="sc">|&gt;</span></span>
<span id="cb42-49"><a href="#cb42-49" aria-hidden="true" tabindex="-1"></a>        dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="sc">-</span>{{ outcome_var }}) <span class="sc">|&gt;</span></span>
<span id="cb42-50"><a href="#cb42-50" aria-hidden="true" tabindex="-1"></a>        mice<span class="sc">::</span><span class="fu">futuremice</span>(<span class="at">m =</span> imputations,</span>
<span id="cb42-51"><a href="#cb42-51" aria-hidden="true" tabindex="-1"></a>                         <span class="at">method =</span> method,</span>
<span id="cb42-52"><a href="#cb42-52" aria-hidden="true" tabindex="-1"></a>                         <span class="at">n.core =</span> imputations,</span>
<span id="cb42-53"><a href="#cb42-53" aria-hidden="true" tabindex="-1"></a>                         <span class="at">iterations =</span> iterations,</span>
<span id="cb42-54"><a href="#cb42-54" aria-hidden="true" tabindex="-1"></a>                         <span class="at">parallelseed  =</span> seed)</span>
<span id="cb42-55"><a href="#cb42-55" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Generate output dataset,</span></span>
<span id="cb42-56"><a href="#cb42-56" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>imputed <span class="ot">&lt;-</span> results<span class="sc">$</span>mids <span class="sc">|&gt;</span></span>
<span id="cb42-57"><a href="#cb42-57" aria-hidden="true" tabindex="-1"></a>        mice<span class="sc">::</span><span class="fu">complete</span>(<span class="at">action =</span> <span class="st">"long"</span>, <span class="at">include =</span> include)</span>
<span id="cb42-58"><a href="#cb42-58" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Convert the .imp variable which indicates the imputation set to factor with original dataset labelled as such</span></span>
<span id="cb42-59"><a href="#cb42-59" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>imputed <span class="ot">&lt;-</span> results<span class="sc">$</span>imputed <span class="sc">|&gt;</span></span>
<span id="cb42-60"><a href="#cb42-60" aria-hidden="true" tabindex="-1"></a>         dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">.imp =</span> <span class="fu">factor</span>(.imp,</span>
<span id="cb42-61"><a href="#cb42-61" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">levels =</span> <span class="fu">seq</span>(<span class="dv">0</span>, imputations, <span class="dv">1</span>),</span>
<span id="cb42-62"><a href="#cb42-62" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"Original"</span>, <span class="fu">as.character</span>(<span class="fu">seq</span>(<span class="dv">1</span>, imputations, <span class="dv">1</span>)))))</span>
<span id="cb42-63"><a href="#cb42-63" aria-hidden="true" tabindex="-1"></a>    <span class="do">## We need to bind the outcome variable to each imputed dataset so they can be used in analyses</span></span>
<span id="cb42-64"><a href="#cb42-64" aria-hidden="true" tabindex="-1"></a>    outcome <span class="ot">=</span> df[[outcome_var]]</span>
<span id="cb42-65"><a href="#cb42-65" aria-hidden="true" tabindex="-1"></a>    n <span class="ot">=</span> imputations</span>
<span id="cb42-66"><a href="#cb42-66" aria-hidden="true" tabindex="-1"></a>    <span class="cf">while</span>(n <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb42-67"><a href="#cb42-67" aria-hidden="true" tabindex="-1"></a>        outcome <span class="ot">=</span> <span class="fu">append</span>(outcome, df[[outcome_var]])</span>
<span id="cb42-68"><a href="#cb42-68" aria-hidden="true" tabindex="-1"></a>        n <span class="ot">&lt;-</span> n <span class="sc">-</span> <span class="dv">1</span></span>
<span id="cb42-69"><a href="#cb42-69" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb42-70"><a href="#cb42-70" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>imputed <span class="ot">&lt;-</span> <span class="fu">cbind</span>(results<span class="sc">$</span>imputed, outcome)</span>
<span id="cb42-71"><a href="#cb42-71" aria-hidden="true" tabindex="-1"></a>    <span class="fu">colnames</span>(results<span class="sc">$</span>imputed) <span class="ot">&lt;-</span> stringr<span class="sc">::</span><span class="fu">str_replace</span>(<span class="fu">colnames</span>(results<span class="sc">$</span>imputed), <span class="st">"outcome"</span>, outcome_var)</span>
<span id="cb42-72"><a href="#cb42-72" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Plot traces of the imputation over iteration</span></span>
<span id="cb42-73"><a href="#cb42-73" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>trace <span class="ot">&lt;-</span> ggmice<span class="sc">::</span><span class="fu">plot_trace</span>(results<span class="sc">$</span>mid)</span>
<span id="cb42-74"><a href="#cb42-74" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Plot correlation between variables</span></span>
<span id="cb42-75"><a href="#cb42-75" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>corr <span class="ot">&lt;-</span> ggmice<span class="sc">::</span><span class="fu">plot_corr</span>(df,</span>
<span id="cb42-76"><a href="#cb42-76" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">label =</span> <span class="cn">TRUE</span>,</span>
<span id="cb42-77"><a href="#cb42-77" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">square =</span> <span class="cn">TRUE</span>,</span>
<span id="cb42-78"><a href="#cb42-78" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">rotate =</span> <span class="cn">TRUE</span>,</span>
<span id="cb42-79"><a href="#cb42-79" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">diagonal =</span> <span class="cn">FALSE</span>)</span>
<span id="cb42-80"><a href="#cb42-80" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Plot histograms of continuous variables</span></span>
<span id="cb42-81"><a href="#cb42-81" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>histogram <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb42-82"><a href="#cb42-82" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> (var <span class="cf">in</span> continuous) {</span>
<span id="cb42-83"><a href="#cb42-83" aria-hidden="true" tabindex="-1"></a>        results<span class="sc">$</span>histogram[[var]] <span class="ot">&lt;-</span> ggmice<span class="sc">::</span><span class="fu">ggmice</span>(results<span class="sc">$</span>mids,</span>
<span id="cb42-84"><a href="#cb42-84" aria-hidden="true" tabindex="-1"></a>                                                   ggplot2<span class="sc">::</span><span class="fu">aes</span>(<span class="at">x =</span> .data[[var]],</span>
<span id="cb42-85"><a href="#cb42-85" aria-hidden="true" tabindex="-1"></a>                                                                <span class="at">group =</span> .imp)) <span class="sc">+</span></span>
<span id="cb42-86"><a href="#cb42-86" aria-hidden="true" tabindex="-1"></a>                                        ggplot2<span class="sc">::</span><span class="fu">geom_density</span>()</span>
<span id="cb42-87"><a href="#cb42-87" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb42-88"><a href="#cb42-88" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Scatterplots and bar charts for categorical variables</span></span>
<span id="cb42-89"><a href="#cb42-89" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>scatter <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb42-90"><a href="#cb42-90" aria-hidden="true" tabindex="-1"></a>    results<span class="sc">$</span>bar_chart <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb42-91"><a href="#cb42-91" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> (var <span class="cf">in</span> categorical) {</span>
<span id="cb42-92"><a href="#cb42-92" aria-hidden="true" tabindex="-1"></a>        results<span class="sc">$</span>scatter[[var]] <span class="ot">&lt;-</span> ggmice<span class="sc">::</span><span class="fu">ggmice</span>(results<span class="sc">$</span>mids,</span>
<span id="cb42-93"><a href="#cb42-93" aria-hidden="true" tabindex="-1"></a>                                                 ggplot2<span class="sc">::</span><span class="fu">aes</span>(<span class="at">x =</span> .imp,</span>
<span id="cb42-94"><a href="#cb42-94" aria-hidden="true" tabindex="-1"></a>                                                              <span class="at">y =</span> .data[[var]])) <span class="sc">+</span></span>
<span id="cb42-95"><a href="#cb42-95" aria-hidden="true" tabindex="-1"></a>                                        ggplot2<span class="sc">::</span><span class="fu">geom_jitter</span>()</span>
<span id="cb42-96"><a href="#cb42-96" aria-hidden="true" tabindex="-1"></a>        results<span class="sc">$</span>bar_chart[[var]] <span class="ot">&lt;-</span> ggmice<span class="sc">::</span><span class="fu">ggmice</span>(results<span class="sc">$</span>mids,</span>
<span id="cb42-97"><a href="#cb42-97" aria-hidden="true" tabindex="-1"></a>                                                   ggplot2<span class="sc">::</span><span class="fu">aes</span>(<span class="at">x =</span> .data[[var]],</span>
<span id="cb42-98"><a href="#cb42-98" aria-hidden="true" tabindex="-1"></a>                                                                <span class="at">fill =</span> .imp)) <span class="sc">+</span></span>
<span id="cb42-99"><a href="#cb42-99" aria-hidden="true" tabindex="-1"></a>                                        ggplot2<span class="sc">::</span><span class="fu">geom_bar</span>(<span class="at">position =</span> <span class="st">"dodge"</span>)</span>
<span id="cb42-100"><a href="#cb42-100" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb42-101"><a href="#cb42-101" aria-hidden="true" tabindex="-1"></a>    results</span>
<span id="cb42-102"><a href="#cb42-102" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb42-103"><a href="#cb42-103" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb42-104"><a href="#cb42-104" aria-hidden="true" tabindex="-1"></a><span class="do">## Impute using three different methods using the above impute_data() wrapper</span></span>
<span id="cb42-105"><a href="#cb42-105" aria-hidden="true" tabindex="-1"></a>imputations <span class="ot">=</span> <span class="dv">5</span></span>
<span id="cb42-106"><a href="#cb42-106" aria-hidden="true" tabindex="-1"></a>iterations <span class="ot">=</span> <span class="dv">5</span></span>
<span id="cb42-107"><a href="#cb42-107" aria-hidden="true" tabindex="-1"></a>mice_pmm <span class="ot">&lt;-</span> <span class="fu">impute_data</span>(<span class="at">method =</span> <span class="st">"pmm"</span>,</span>
<span id="cb42-108"><a href="#cb42-108" aria-hidden="true" tabindex="-1"></a>                        <span class="at">imputations =</span> imputations,</span>
<span id="cb42-109"><a href="#cb42-109" aria-hidden="true" tabindex="-1"></a>                        <span class="at">iterations =</span> iterations,</span>
<span id="cb42-110"><a href="#cb42-110" aria-hidden="true" tabindex="-1"></a>                        <span class="at">seed =</span> <span class="dv">684613</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25
Warning: Number of logged events: 25
Warning: Number of logged events: 25</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 24
Warning: Number of logged events: 24</code></pre>
</div>
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a>mice_cart <span class="ot">&lt;-</span> <span class="fu">impute_data</span>(<span class="at">method =</span> <span class="st">"cart"</span>,</span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>                        <span class="at">imputations =</span> imputations,</span>
<span id="cb45-3"><a href="#cb45-3" aria-hidden="true" tabindex="-1"></a>                        <span class="at">iterations =</span> iterations,</span>
<span id="cb45-4"><a href="#cb45-4" aria-hidden="true" tabindex="-1"></a>                        <span class="at">seed =</span> <span class="dv">1388466</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 24</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 23</code></pre>
</div>
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a>mice_rf <span class="ot">&lt;-</span> <span class="fu">impute_data</span>(<span class="at">method =</span> <span class="st">"rf"</span>,</span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a>                        <span class="at">imputations =</span> imputations,</span>
<span id="cb51-3"><a href="#cb51-3" aria-hidden="true" tabindex="-1"></a>                        <span class="at">iterations =</span> iterations,</span>
<span id="cb51-4"><a href="#cb51-4" aria-hidden="true" tabindex="-1"></a>                        <span class="at">seed =</span> <span class="dv">3151358</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 23</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Number of logged events: 25
Warning: Number of logged events: 25</code></pre>
</div>
</div></div>
<p>The convergence of the imputation methods are shown in figues <strong>?@fig-mice-convergence-pmm</strong>, <strong>?@fig-mice-convergence-cart</strong>, and <strong>?@fig-mice-convergence-rf</strong>.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"></ul>
<div class="tab-content">

</div>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-5-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-1" role="tab" aria-controls="tabset-5-1" aria-selected="true" href="">Albumin</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-2" role="tab" aria-controls="tabset-5-2" aria-selected="false" href="">Monocyte</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-3" role="tab" aria-controls="tabset-5-3" aria-selected="false" href="">Lymphocytes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-4" role="tab" aria-controls="tabset-5-4" aria-selected="false" href="">TSH Value</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-5" role="tab" aria-controls="tabset-5-5" aria-selected="false" href="">Nodule Size</a></li></ul>
<div class="tab-content">
<div id="tabset-5-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-5-1-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [13]:</pre></div><div id="cell-fig-mice-imputation-albumin" class="cell">
<div class="sourceCode cell-code" id="cb56"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb56-1"><a href="#cb56-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>histogram<span class="sc">$</span>albumin,</span>
<span id="cb56-2"><a href="#cb56-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>histogram<span class="sc">$</span>albumin,</span>
<span id="cb56-3"><a href="#cb56-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>histogram<span class="sc">$</span>albumin,</span>
<span id="cb56-4"><a href="#cb56-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">1</span>,</span>
<span id="cb56-5"><a href="#cb56-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-albumin" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-albumin-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-albumin-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="Figure&nbsp;6: "><img src="index_files/figure-html/fig-mice-imputation-albumin-1.png" id="fig-mice-imputation-albumin" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-albumin-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-5-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-2-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [14]:</pre></div><div id="cell-fig-mice-imputation-monocyte" class="cell">
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>histogram<span class="sc">$</span>monocyte,</span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>histogram<span class="sc">$</span>monocyte,</span>
<span id="cb57-3"><a href="#cb57-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>histogram<span class="sc">$</span>monocyte,</span>
<span id="cb57-4"><a href="#cb57-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">1</span>,</span>
<span id="cb57-5"><a href="#cb57-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-monocyte" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-monocyte-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-monocyte-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-7" title="Figure&nbsp;7: "><img src="index_files/figure-html/fig-mice-imputation-monocyte-1.png" id="fig-mice-imputation-monocyte" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-monocyte-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-5-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-3-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [15]:</pre></div><div id="cell-fig-mice-imputation-lymphocytes" class="cell">
<div class="sourceCode cell-code" id="cb58"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>histogram<span class="sc">$</span>lymphocytes,</span>
<span id="cb58-2"><a href="#cb58-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>histogram<span class="sc">$</span>lymphocytes,</span>
<span id="cb58-3"><a href="#cb58-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>histogram<span class="sc">$</span>lymphocytes,</span>
<span id="cb58-4"><a href="#cb58-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">1</span>,</span>
<span id="cb58-5"><a href="#cb58-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-lymphocytes" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-lymphocytes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-lymphocytes-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-8" title="Figure&nbsp;8: "><img src="index_files/figure-html/fig-mice-imputation-lymphocytes-1.png" id="fig-mice-imputation-lymphocytes" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-lymphocytes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-5-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-4-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [16]:</pre></div><div id="cell-fig-mice-imputation-tsh-value" class="cell">
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>histogram<span class="sc">$</span>tsh_value,</span>
<span id="cb59-2"><a href="#cb59-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>histogram<span class="sc">$</span>tsh_value,</span>
<span id="cb59-3"><a href="#cb59-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>histogram<span class="sc">$</span>tsh_value,</span>
<span id="cb59-4"><a href="#cb59-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">1</span>,</span>
<span id="cb59-5"><a href="#cb59-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-tsh-value" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-tsh-value-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-tsh-value-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-9" title="Figure&nbsp;9: "><img src="index_files/figure-html/fig-mice-imputation-tsh-value-1.png" id="fig-mice-imputation-tsh-value" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-tsh-value-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-5-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-5-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [17]:</pre></div><div id="cell-fig-mice-imputation-size-nodule-mm" class="cell">
<div class="sourceCode cell-code" id="cb60"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>histogram<span class="sc">$</span>size_nodule_mm,</span>
<span id="cb60-2"><a href="#cb60-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>histogram<span class="sc">$</span>size_nodule_mm,</span>
<span id="cb60-3"><a href="#cb60-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>histogram<span class="sc">$</span>size_nodule_mm,</span>
<span id="cb60-4"><a href="#cb60-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">1</span>,</span>
<span id="cb60-5"><a href="#cb60-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-size-nodule-mm" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-size-nodule-mm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-size-nodule-mm-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-10" title="Figure&nbsp;10: "><img src="index_files/figure-html/fig-mice-imputation-size-nodule-mm-1.png" id="fig-mice-imputation-size-nodule-mm" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-size-nodule-mm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
</div>
</div>
<p><strong>TODO</strong> - Extract the legends from individual plots and add them to the end of each row, see the <a href="https://wilkelab.org/cowplot/articles/shared_legends.html">cowplot shared legends article</a> for pointers on how to do this. Should ideally also get the <code>fill</code> colours to align with those used by <code>ggmice</code>.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-6-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-1" role="tab" aria-controls="tabset-6-1" aria-selected="true" href="">Incidental Nodule</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-2" role="tab" aria-controls="tabset-6-2" aria-selected="false" href="">Palpable Nodule</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-3" role="tab" aria-controls="tabset-6-3" aria-selected="false" href="">Rapid Enlargement</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-4" role="tab" aria-controls="tabset-6-4" aria-selected="false" href="">Compressive Symptoms</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-5" role="tab" aria-controls="tabset-6-5" aria-selected="false" href="">Hypertension</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-6" role="tab" aria-controls="tabset-6-6" aria-selected="false" href="">Vocal Cord Paresis</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-7-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-7" role="tab" aria-controls="tabset-6-7" aria-selected="false" href="">Graves Disease</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-8-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-8" role="tab" aria-controls="tabset-6-8" aria-selected="false" href="">Hashimotos Thyroiditis</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-9-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-9" role="tab" aria-controls="tabset-6-9" aria-selected="false" href="">Family History</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-10-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-10" role="tab" aria-controls="tabset-6-10" aria-selected="false" href="">Exposure Radiation</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-11-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-11" role="tab" aria-controls="tabset-6-11" aria-selected="false" href="">Solitary Nodule</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-12-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-12" role="tab" aria-controls="tabset-6-12" aria-selected="false" href="">BTA U-Classification</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-13-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-13" role="tab" aria-controls="tabset-6-13" aria-selected="false" href="">Cervical Lymphadenopathy</a></li></ul>
<div class="tab-content">
<div id="tabset-6-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-6-1-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [18]:</pre></div><div id="cell-fig-mice-imputation-incidental-nodule" class="cell">
<div class="sourceCode cell-code" id="cb61"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb61-1"><a href="#cb61-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>incidental_nodule <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"None"</span>),</span>
<span id="cb61-2"><a href="#cb61-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>incidental_nodule <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"None"</span>),</span>
<span id="cb61-3"><a href="#cb61-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>incidental_nodule <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"None"</span>),</span>
<span id="cb61-4"><a href="#cb61-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>incidental_nodule <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"None"</span>),</span>
<span id="cb61-5"><a href="#cb61-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>incidental_nodule <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"None"</span>),</span>
<span id="cb61-6"><a href="#cb61-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>incidental_nodule <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"None"</span>),</span>
<span id="cb61-7"><a href="#cb61-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb61-8"><a href="#cb61-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-incidental-nodule" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-incidental-nodule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-incidental-nodule-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-11" title="Figure&nbsp;11: "><img src="index_files/figure-html/fig-mice-imputation-incidental-nodule-1.png" id="fig-mice-imputation-incidental-nodule" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-incidental-nodule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-2-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [19]:</pre></div><div id="cell-fig-mice-imputation-palpable-nodule" class="cell">
<div class="sourceCode cell-code" id="cb62"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>palpable_nodule,</span>
<span id="cb62-2"><a href="#cb62-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>palpable_nodule,</span>
<span id="cb62-3"><a href="#cb62-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>palpable_nodule,</span>
<span id="cb62-4"><a href="#cb62-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>palpable_nodule,</span>
<span id="cb62-5"><a href="#cb62-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>palpable_nodule,</span>
<span id="cb62-6"><a href="#cb62-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>palpable_nodule,</span>
<span id="cb62-7"><a href="#cb62-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb62-8"><a href="#cb62-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-palpable-nodule" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-palpable-nodule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-palpable-nodule-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-12" title="Figure&nbsp;12: "><img src="index_files/figure-html/fig-mice-imputation-palpable-nodule-1.png" id="fig-mice-imputation-palpable-nodule" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-palpable-nodule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-3-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [20]:</pre></div><div id="cell-fig-mice-imputation-rapid-enlargement" class="cell">
<div class="sourceCode cell-code" id="cb63"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>rapid_enlargement,</span>
<span id="cb63-2"><a href="#cb63-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>rapid_enlargement,</span>
<span id="cb63-3"><a href="#cb63-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>rapid_enlargement,</span>
<span id="cb63-4"><a href="#cb63-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>rapid_enlargement,</span>
<span id="cb63-5"><a href="#cb63-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>rapid_enlargement,</span>
<span id="cb63-6"><a href="#cb63-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>rapid_enlargement,</span>
<span id="cb63-7"><a href="#cb63-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb63-8"><a href="#cb63-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-rapid-enlargement" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-rapid-enlargement-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-rapid-enlargement-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-13" title="Figure&nbsp;13: "><img src="index_files/figure-html/fig-mice-imputation-rapid-enlargement-1.png" id="fig-mice-imputation-rapid-enlargement" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-rapid-enlargement-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-4-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [21]:</pre></div><div id="cell-fig-mice-imputation-compressive-symptoms" class="cell">
<div class="sourceCode cell-code" id="cb64"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>compressive_symptoms,</span>
<span id="cb64-2"><a href="#cb64-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>compressive_symptoms,</span>
<span id="cb64-3"><a href="#cb64-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>compressive_symptoms,</span>
<span id="cb64-4"><a href="#cb64-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>compressive_symptoms,</span>
<span id="cb64-5"><a href="#cb64-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>compressive_symptoms,</span>
<span id="cb64-6"><a href="#cb64-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>compressive_symptoms,</span>
<span id="cb64-7"><a href="#cb64-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb64-8"><a href="#cb64-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-compressive-symptoms" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-compressive-symptoms-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-compressive-symptoms-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-14" title="Figure&nbsp;14: "><img src="index_files/figure-html/fig-mice-imputation-compressive-symptoms-1.png" id="fig-mice-imputation-compressive-symptoms" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-compressive-symptoms-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-5-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [22]:</pre></div><div id="cell-fig-mice-imputation-hypertension" class="cell">
<div class="sourceCode cell-code" id="cb65"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb65-1"><a href="#cb65-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>hypertension,</span>
<span id="cb65-2"><a href="#cb65-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>hypertension,</span>
<span id="cb65-3"><a href="#cb65-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>hypertension,</span>
<span id="cb65-4"><a href="#cb65-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>hypertension,</span>
<span id="cb65-5"><a href="#cb65-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>hypertension,</span>
<span id="cb65-6"><a href="#cb65-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>hypertension,</span>
<span id="cb65-7"><a href="#cb65-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb65-8"><a href="#cb65-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-hypertension" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-hypertension-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-hypertension-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-15" title="Figure&nbsp;15: "><img src="index_files/figure-html/fig-mice-imputation-hypertension-1.png" id="fig-mice-imputation-hypertension" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-hypertension-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-6-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [23]:</pre></div><div id="cell-fig-mice-imputation-vocal-cord-paresis" class="cell">
<div class="sourceCode cell-code" id="cb66"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb66-1"><a href="#cb66-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>vocal_cord_paresis,</span>
<span id="cb66-2"><a href="#cb66-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>vocal_cord_paresis,</span>
<span id="cb66-3"><a href="#cb66-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>vocal_cord_paresis,</span>
<span id="cb66-4"><a href="#cb66-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>vocal_cord_paresis,</span>
<span id="cb66-5"><a href="#cb66-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>vocal_cord_paresis,</span>
<span id="cb66-6"><a href="#cb66-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>vocal_cord_paresis,</span>
<span id="cb66-7"><a href="#cb66-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb66-8"><a href="#cb66-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-vocal-cord-paresis" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-vocal-cord-paresis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-vocal-cord-paresis-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-16" title="Figure&nbsp;16: "><img src="index_files/figure-html/fig-mice-imputation-vocal-cord-paresis-1.png" id="fig-mice-imputation-vocal-cord-paresis" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-vocal-cord-paresis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-7" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-7-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [24]:</pre></div><div id="cell-fig-mice-imputation-graves-disease" class="cell">
<div class="sourceCode cell-code" id="cb67"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb67-1"><a href="#cb67-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>graves_disease,</span>
<span id="cb67-2"><a href="#cb67-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>graves_disease,</span>
<span id="cb67-3"><a href="#cb67-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>graves_disease,</span>
<span id="cb67-4"><a href="#cb67-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>graves_disease,</span>
<span id="cb67-5"><a href="#cb67-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>graves_disease,</span>
<span id="cb67-6"><a href="#cb67-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>graves_disease,</span>
<span id="cb67-7"><a href="#cb67-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb67-8"><a href="#cb67-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-graves-disease" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-graves-disease-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-graves-disease-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-17" title="Figure&nbsp;17: "><img src="index_files/figure-html/fig-mice-imputation-graves-disease-1.png" id="fig-mice-imputation-graves-disease" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-graves-disease-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-8" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-8-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [25]:</pre></div><div id="cell-fig-mice-imputation-hashimotos-thyroiditis" class="cell">
<div class="sourceCode cell-code" id="cb68"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb68-1"><a href="#cb68-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>hashimotos_thyroiditis,</span>
<span id="cb68-2"><a href="#cb68-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>hashimotos_thyroiditis,</span>
<span id="cb68-3"><a href="#cb68-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>hashimotos_thyroiditis,</span>
<span id="cb68-4"><a href="#cb68-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>hashimotos_thyroiditis,</span>
<span id="cb68-5"><a href="#cb68-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>hashimotos_thyroiditis,</span>
<span id="cb68-6"><a href="#cb68-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>hashimotos_thyroiditis,</span>
<span id="cb68-7"><a href="#cb68-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb68-8"><a href="#cb68-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-hashimotos-thyroiditis" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-hashimotos-thyroiditis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-hashimotos-thyroiditis-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-18" title="Figure&nbsp;18: "><img src="index_files/figure-html/fig-mice-imputation-hashimotos-thyroiditis-1.png" id="fig-mice-imputation-hashimotos-thyroiditis" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-hashimotos-thyroiditis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-9" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-9-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [26]:</pre></div><div id="cell-fig-mice-imputation-family-history-thyroid-cancer" class="cell">
<div class="sourceCode cell-code" id="cb69"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>family_history,</span>
<span id="cb69-2"><a href="#cb69-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>family_history,</span>
<span id="cb69-3"><a href="#cb69-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>family_history,</span>
<span id="cb69-4"><a href="#cb69-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>family_history,</span>
<span id="cb69-5"><a href="#cb69-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>family_history,</span>
<span id="cb69-6"><a href="#cb69-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>family_history,</span>
<span id="cb69-7"><a href="#cb69-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb69-8"><a href="#cb69-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-family-history-thyroid-cancer" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-family-history-thyroid-cancer-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-family-history-thyroid-cancer-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-19" title="Figure&nbsp;19: "><img src="index_files/figure-html/fig-mice-imputation-family-history-thyroid-cancer-1.png" id="fig-mice-imputation-family-history-thyroid-cancer" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-family-history-thyroid-cancer-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-10" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-10-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [27]:</pre></div><div id="cell-fig-mice-imputation-exposure-radiation" class="cell">
<div class="sourceCode cell-code" id="cb70"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb70-1"><a href="#cb70-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>exposure_radiation,</span>
<span id="cb70-2"><a href="#cb70-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>exposure_radiation,</span>
<span id="cb70-3"><a href="#cb70-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>exposure_radiation,</span>
<span id="cb70-4"><a href="#cb70-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>exposure_radiation,</span>
<span id="cb70-5"><a href="#cb70-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>exposure_radiation,</span>
<span id="cb70-6"><a href="#cb70-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>exposure_radiation,</span>
<span id="cb70-7"><a href="#cb70-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb70-8"><a href="#cb70-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-exposure-radiation" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-exposure-radiation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-exposure-radiation-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-20" title="Figure&nbsp;20: "><img src="index_files/figure-html/fig-mice-imputation-exposure-radiation-1.png" id="fig-mice-imputation-exposure-radiation" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-exposure-radiation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-11" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-11-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [28]:</pre></div><div id="cell-fig-mice-imputation-solitary-nodule" class="cell">
<div class="sourceCode cell-code" id="cb71"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>solitary_nodule,</span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>solitary_nodule,</span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>solitary_nodule,</span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>solitary_nodule,</span>
<span id="cb71-5"><a href="#cb71-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>solitary_nodule,</span>
<span id="cb71-6"><a href="#cb71-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>solitary_nodule,</span>
<span id="cb71-7"><a href="#cb71-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb71-8"><a href="#cb71-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-solitary-nodule" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-solitary-nodule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-solitary-nodule-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-21" title="Figure&nbsp;21: "><img src="index_files/figure-html/fig-mice-imputation-solitary-nodule-1.png" id="fig-mice-imputation-solitary-nodule" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-solitary-nodule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-12" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-12-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [29]:</pre></div><div id="cell-fig-mice-imputation-bta-u-classification" class="cell">
<div class="sourceCode cell-code" id="cb72"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb72-1"><a href="#cb72-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>bta_u_classification,</span>
<span id="cb72-2"><a href="#cb72-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>bta_u_classification,</span>
<span id="cb72-3"><a href="#cb72-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>bta_u_classification,</span>
<span id="cb72-4"><a href="#cb72-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>bta_u_classification,</span>
<span id="cb72-5"><a href="#cb72-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>bta_u_classification,</span>
<span id="cb72-6"><a href="#cb72-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>bta_u_classification,</span>
<span id="cb72-7"><a href="#cb72-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb72-8"><a href="#cb72-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-bta-u-classification" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-bta-u-classification-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-bta-u-classification-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-22" title="Figure&nbsp;22: "><img src="index_files/figure-html/fig-mice-imputation-bta-u-classification-1.png" id="fig-mice-imputation-bta-u-classification" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-bta-u-classification-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
<div id="tabset-6-13" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-13-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [30]:</pre></div><div id="cell-fig-mice-imputation-rapid-cervical-lymphadenopathy" class="cell">
<div class="sourceCode cell-code" id="cb73"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(mice_pmm<span class="sc">$</span>scatter<span class="sc">$</span>cervical_lymphadenopathy,</span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>scatter<span class="sc">$</span>cervical_lymphadenopathy,</span>
<span id="cb73-3"><a href="#cb73-3" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>scatter<span class="sc">$</span>cervical_lymphadenopathy,</span>
<span id="cb73-4"><a href="#cb73-4" aria-hidden="true" tabindex="-1"></a>                   mice_pmm<span class="sc">$</span>bar_chart<span class="sc">$</span>cervical_lymphadenopathy,</span>
<span id="cb73-5"><a href="#cb73-5" aria-hidden="true" tabindex="-1"></a>                   mice_cart<span class="sc">$</span>bar_chart<span class="sc">$</span>cervical_lymphadenopathy,</span>
<span id="cb73-6"><a href="#cb73-6" aria-hidden="true" tabindex="-1"></a>                   mice_rf<span class="sc">$</span>bar_chart<span class="sc">$</span>cervical_lymphadenopathy,</span>
<span id="cb73-7"><a href="#cb73-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">nrow =</span> <span class="dv">2</span>,</span>
<span id="cb73-8"><a href="#cb73-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">ncol =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mice-imputation-rapid-cervical-lymphadenopathy" class="quarto-float quarto-figure quarto-figure-center anchored" width="80%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mice-imputation-rapid-cervical-lymphadenopathy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-mice-imputation-rapid-cervical-lymphadenopathy-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-23" title="Figure&nbsp;23: "><img src="index_files/figure-html/fig-mice-imputation-rapid-cervical-lymphadenopathy-1.png" id="fig-mice-imputation-rapid-cervical-lymphadenopathy" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-mice-imputation-rapid-cervical-lymphadenopathy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
</figcaption>
</figure>
</div>
</div>
</div></div>
</div>
</div>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-7-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-7-1" role="tab" aria-controls="tabset-7-1" aria-selected="true" href="">PMM</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-7-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-7-2" role="tab" aria-controls="tabset-7-2" aria-selected="false" href="">CART</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-7-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-7-3" role="tab" aria-controls="tabset-7-3" aria-selected="false" href="">RF</a></li></ul>
<div class="tab-content">
<div id="tabset-7-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-7-1-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [31]:</pre></div><div id="tbl-imputation-summary-pmm" class="cell anchored" data-tbl-caption="Summary of data imputed via PMM.">
<div class="sourceCode cell-code" id="cb74"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb74-1"><a href="#cb74-1" aria-hidden="true" tabindex="-1"></a>mice_pmm<span class="sc">$</span>imputed <span class="sc">|&gt;</span> gtsummary<span class="sc">::</span><span class="fu">tbl_summary</span>(<span class="at">by=</span><span class="st">".imp"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="vqkuwhxojr" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#vqkuwhxojr table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#vqkuwhxojr thead, #vqkuwhxojr tbody, #vqkuwhxojr tfoot, #vqkuwhxojr tr, #vqkuwhxojr td, #vqkuwhxojr th {
  border-style: none;
}

#vqkuwhxojr p {
  margin: 0;
  padding: 0;
}

#vqkuwhxojr .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#vqkuwhxojr .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#vqkuwhxojr .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#vqkuwhxojr .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#vqkuwhxojr .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#vqkuwhxojr .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vqkuwhxojr .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#vqkuwhxojr .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#vqkuwhxojr .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#vqkuwhxojr .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#vqkuwhxojr .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#vqkuwhxojr .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#vqkuwhxojr .gt_spanner_row {
  border-bottom-style: hidden;
}

#vqkuwhxojr .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#vqkuwhxojr .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#vqkuwhxojr .gt_from_md > :first-child {
  margin-top: 0;
}

#vqkuwhxojr .gt_from_md > :last-child {
  margin-bottom: 0;
}

#vqkuwhxojr .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#vqkuwhxojr .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#vqkuwhxojr .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#vqkuwhxojr .gt_row_group_first td {
  border-top-width: 2px;
}

#vqkuwhxojr .gt_row_group_first th {
  border-top-width: 2px;
}

#vqkuwhxojr .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#vqkuwhxojr .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#vqkuwhxojr .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#vqkuwhxojr .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vqkuwhxojr .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#vqkuwhxojr .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#vqkuwhxojr .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#vqkuwhxojr .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#vqkuwhxojr .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vqkuwhxojr .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#vqkuwhxojr .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#vqkuwhxojr .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#vqkuwhxojr .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#vqkuwhxojr .gt_left {
  text-align: left;
}

#vqkuwhxojr .gt_center {
  text-align: center;
}

#vqkuwhxojr .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#vqkuwhxojr .gt_font_normal {
  font-weight: normal;
}

#vqkuwhxojr .gt_font_bold {
  font-weight: bold;
}

#vqkuwhxojr .gt_font_italic {
  font-style: italic;
}

#vqkuwhxojr .gt_super {
  font-size: 65%;
}

#vqkuwhxojr .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#vqkuwhxojr .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#vqkuwhxojr .gt_indent_1 {
  text-indent: 5px;
}

#vqkuwhxojr .gt_indent_2 {
  text-indent: 10px;
}

#vqkuwhxojr .gt_indent_3 {
  text-indent: 15px;
}

#vqkuwhxojr .gt_indent_4 {
  text-indent: 20px;
}

#vqkuwhxojr .gt_indent_5 {
  text-indent: 25px;
}

#vqkuwhxojr .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#vqkuwhxojr div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPcmlnaW5hbCoqLCBOID0gMSwxNTA=&quot;><div class='gt_from_md'><p><strong>Original</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>Original</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KioxKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>1</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>1</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KioyKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>2</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>2</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KiozKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>3</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>3</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio0KiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>4</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>4</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio1KiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>5</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>5</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">.id</td>
<td class="gt_row gt_center" headers="stat_1">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_2">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_3">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_4">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_5">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_6">576 (288, 863)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="stat_1">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_2">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_3">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_4">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_5">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_6">55 (41, 68)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;F</td>
<td class="gt_row gt_center" headers="stat_1">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_2">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_3">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_4">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_5">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_6">903 (79%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;M</td>
<td class="gt_row gt_center" headers="stat_1">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_2">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_3">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_4">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_5">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_6">247 (21%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">ethnicity</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;A</td>
<td class="gt_row gt_center" headers="stat_1">812 (74%)</td>
<td class="gt_row gt_center" headers="stat_2">843 (73%)</td>
<td class="gt_row gt_center" headers="stat_3">841 (73%)</td>
<td class="gt_row gt_center" headers="stat_4">839 (73%)</td>
<td class="gt_row gt_center" headers="stat_5">845 (73%)</td>
<td class="gt_row gt_center" headers="stat_6">842 (73%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;B</td>
<td class="gt_row gt_center" headers="stat_1">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_2">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">3 (0.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;C</td>
<td class="gt_row gt_center" headers="stat_1">44 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_2">45 (3.9%)</td>
<td class="gt_row gt_center" headers="stat_3">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_4">44 (3.8%)</td>
<td class="gt_row gt_center" headers="stat_5">48 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_6">45 (3.9%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;D</td>
<td class="gt_row gt_center" headers="stat_1">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_2">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_3">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_4">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_5">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_6">2 (0.2%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;F</td>
<td class="gt_row gt_center" headers="stat_1">4 (0.4%)</td>
<td class="gt_row gt_center" headers="stat_2">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">4 (0.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;G</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_5">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;H</td>
<td class="gt_row gt_center" headers="stat_1">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_2">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_3">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_4">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">8 (0.7%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;J</td>
<td class="gt_row gt_center" headers="stat_1">46 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_2">50 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_3">49 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_4">51 (4.4%)</td>
<td class="gt_row gt_center" headers="stat_5">50 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_6">49 (4.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;K</td>
<td class="gt_row gt_center" headers="stat_1">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_2">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_3">13 (1.1%)</td>
<td class="gt_row gt_center" headers="stat_4">12 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_5">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_6">12 (1.0%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;L</td>
<td class="gt_row gt_center" headers="stat_1">16 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_4">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_5">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_6">17 (1.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;M</td>
<td class="gt_row gt_center" headers="stat_1">16 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_2">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_3">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_4">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_5">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_6">18 (1.6%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;N</td>
<td class="gt_row gt_center" headers="stat_1">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_4">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_5">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">20 (1.7%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;P</td>
<td class="gt_row gt_center" headers="stat_1">14 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_2">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_3">15 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_4">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_5">15 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_6">14 (1.2%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;R</td>
<td class="gt_row gt_center" headers="stat_1">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_4">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_5">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;S</td>
<td class="gt_row gt_center" headers="stat_1">38 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_2">40 (3.5%)</td>
<td class="gt_row gt_center" headers="stat_3">43 (3.7%)</td>
<td class="gt_row gt_center" headers="stat_4">42 (3.7%)</td>
<td class="gt_row gt_center" headers="stat_5">40 (3.5%)</td>
<td class="gt_row gt_center" headers="stat_6">40 (3.5%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Z</td>
<td class="gt_row gt_center" headers="stat_1">57 (5.2%)</td>
<td class="gt_row gt_center" headers="stat_2">61 (5.3%)</td>
<td class="gt_row gt_center" headers="stat_3">61 (5.3%)</td>
<td class="gt_row gt_center" headers="stat_4">59 (5.1%)</td>
<td class="gt_row gt_center" headers="stat_5">59 (5.1%)</td>
<td class="gt_row gt_center" headers="stat_6">62 (5.4%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">48</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">incidental_nodule</td>
<td class="gt_row gt_center" headers="stat_1">620 (54%)</td>
<td class="gt_row gt_center" headers="stat_2">623 (54%)</td>
<td class="gt_row gt_center" headers="stat_3">622 (54%)</td>
<td class="gt_row gt_center" headers="stat_4">623 (54%)</td>
<td class="gt_row gt_center" headers="stat_5">624 (54%)</td>
<td class="gt_row gt_center" headers="stat_6">623 (54%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">5</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">palpable_nodule</td>
<td class="gt_row gt_center" headers="stat_1">441 (40%)</td>
<td class="gt_row gt_center" headers="stat_2">471 (41%)</td>
<td class="gt_row gt_center" headers="stat_3">470 (41%)</td>
<td class="gt_row gt_center" headers="stat_4">470 (41%)</td>
<td class="gt_row gt_center" headers="stat_5">471 (41%)</td>
<td class="gt_row gt_center" headers="stat_6">470 (41%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">58</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">rapid_enlargement</td>
<td class="gt_row gt_center" headers="stat_1">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">21 (1.8%)</td>
<td class="gt_row gt_center" headers="stat_4">22 (1.9%)</td>
<td class="gt_row gt_center" headers="stat_5">20 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">19 (1.7%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">43</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">compressive_symptoms</td>
<td class="gt_row gt_center" headers="stat_1">88 (8.4%)</td>
<td class="gt_row gt_center" headers="stat_2">107 (9.3%)</td>
<td class="gt_row gt_center" headers="stat_3">103 (9.0%)</td>
<td class="gt_row gt_center" headers="stat_4">102 (8.9%)</td>
<td class="gt_row gt_center" headers="stat_5">98 (8.5%)</td>
<td class="gt_row gt_center" headers="stat_6">101 (8.8%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">106</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">hypertension</td>
<td class="gt_row gt_center" headers="stat_1">262 (26%)</td>
<td class="gt_row gt_center" headers="stat_2">287 (25%)</td>
<td class="gt_row gt_center" headers="stat_3">301 (26%)</td>
<td class="gt_row gt_center" headers="stat_4">292 (25%)</td>
<td class="gt_row gt_center" headers="stat_5">293 (25%)</td>
<td class="gt_row gt_center" headers="stat_6">291 (25%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">126</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">vocal_cord_paresis</td>
<td class="gt_row gt_center" headers="stat_1">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_2">5 (0.4%)</td>
<td class="gt_row gt_center" headers="stat_3">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">3 (0.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">76</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">graves_disease</td>
<td class="gt_row gt_center" headers="stat_1">17 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_2">20 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">20 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_4">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_5">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">18 (1.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">67</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">hashimotos_thyroiditis</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_5">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">73</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">family_history_thyroid_cancer</td>
<td class="gt_row gt_center" headers="stat_1">8 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_2">10 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_3">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_4">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_5">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_6">16 (1.4%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">281</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">exposure_radiation</td>
<td class="gt_row gt_center" headers="stat_1">9 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_2">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_3">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_4">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_5">10 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_6">10 (0.9%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">121</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Albumin</td>
<td class="gt_row gt_center" headers="stat_1">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_2">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_3">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_4">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_5">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_6">45.0 (43.0, 47.0)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">515</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">TSH value</td>
<td class="gt_row gt_center" headers="stat_1">1.48 (0.85, 2.30)</td>
<td class="gt_row gt_center" headers="stat_2">1.50 (0.89, 2.30)</td>
<td class="gt_row gt_center" headers="stat_3">1.50 (0.86, 2.40)</td>
<td class="gt_row gt_center" headers="stat_4">1.50 (0.87, 2.50)</td>
<td class="gt_row gt_center" headers="stat_5">1.50 (0.89, 2.50)</td>
<td class="gt_row gt_center" headers="stat_6">1.40 (0.85, 2.30)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">413</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Lymphocytes</td>
<td class="gt_row gt_center" headers="stat_1">1.94 (1.51, 2.43)</td>
<td class="gt_row gt_center" headers="stat_2">1.96 (1.54, 2.46)</td>
<td class="gt_row gt_center" headers="stat_3">1.95 (1.53, 2.45)</td>
<td class="gt_row gt_center" headers="stat_4">1.94 (1.50, 2.42)</td>
<td class="gt_row gt_center" headers="stat_5">1.95 (1.53, 2.42)</td>
<td class="gt_row gt_center" headers="stat_6">1.93 (1.49, 2.42)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">359</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Monocytes</td>
<td class="gt_row gt_center" headers="stat_1">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_2">0.53 (0.43, 0.66)</td>
<td class="gt_row gt_center" headers="stat_3">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_4">0.53 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_5">0.52 (0.43, 0.66)</td>
<td class="gt_row gt_center" headers="stat_6">0.52 (0.42, 0.66)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">363</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">bta_u_classification</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U1</td>
<td class="gt_row gt_center" headers="stat_1">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_2">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_3">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_4">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_5">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_6">1 (&lt;0.1%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U2</td>
<td class="gt_row gt_center" headers="stat_1">860 (78%)</td>
<td class="gt_row gt_center" headers="stat_2">902 (78%)</td>
<td class="gt_row gt_center" headers="stat_3">895 (78%)</td>
<td class="gt_row gt_center" headers="stat_4">898 (78%)</td>
<td class="gt_row gt_center" headers="stat_5">893 (78%)</td>
<td class="gt_row gt_center" headers="stat_6">893 (78%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U3</td>
<td class="gt_row gt_center" headers="stat_1">210 (19%)</td>
<td class="gt_row gt_center" headers="stat_2">214 (19%)</td>
<td class="gt_row gt_center" headers="stat_3">222 (19%)</td>
<td class="gt_row gt_center" headers="stat_4">220 (19%)</td>
<td class="gt_row gt_center" headers="stat_5">223 (19%)</td>
<td class="gt_row gt_center" headers="stat_6">224 (19%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U4</td>
<td class="gt_row gt_center" headers="stat_1">22 (2.0%)</td>
<td class="gt_row gt_center" headers="stat_2">25 (2.2%)</td>
<td class="gt_row gt_center" headers="stat_3">23 (2.0%)</td>
<td class="gt_row gt_center" headers="stat_4">24 (2.1%)</td>
<td class="gt_row gt_center" headers="stat_5">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_6">22 (1.9%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U5</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_3">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_4">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_5">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_6">10 (0.9%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">50</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">solitary_nodule</td>
<td class="gt_row gt_center" headers="stat_1">320 (28%)</td>
<td class="gt_row gt_center" headers="stat_2">323 (28%)</td>
<td class="gt_row gt_center" headers="stat_3">322 (28%)</td>
<td class="gt_row gt_center" headers="stat_4">323 (28%)</td>
<td class="gt_row gt_center" headers="stat_5">323 (28%)</td>
<td class="gt_row gt_center" headers="stat_6">323 (28%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">8</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Nodule size (mm)</td>
<td class="gt_row gt_center" headers="stat_1">14 (7, 28)</td>
<td class="gt_row gt_center" headers="stat_2">13 (6, 27)</td>
<td class="gt_row gt_center" headers="stat_3">13 (6, 27)</td>
<td class="gt_row gt_center" headers="stat_4">13 (6, 27)</td>
<td class="gt_row gt_center" headers="stat_5">12 (6, 26)</td>
<td class="gt_row gt_center" headers="stat_6">13 (6, 27)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">319</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">cervical_lymphadenopathy</td>
<td class="gt_row gt_center" headers="stat_1">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_2">27 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_3">27 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_4">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_5">27 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_6">26 (2.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">9</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">thy_classification</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1</td>
<td class="gt_row gt_center" headers="stat_1">34 (14%)</td>
<td class="gt_row gt_center" headers="stat_2">362 (31%)</td>
<td class="gt_row gt_center" headers="stat_3">174 (15%)</td>
<td class="gt_row gt_center" headers="stat_4">242 (21%)</td>
<td class="gt_row gt_center" headers="stat_5">179 (16%)</td>
<td class="gt_row gt_center" headers="stat_6">220 (19%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1c</td>
<td class="gt_row gt_center" headers="stat_1">8 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">35 (3.0%)</td>
<td class="gt_row gt_center" headers="stat_4">39 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_5">44 (3.8%)</td>
<td class="gt_row gt_center" headers="stat_6">28 (2.4%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2</td>
<td class="gt_row gt_center" headers="stat_1">63 (27%)</td>
<td class="gt_row gt_center" headers="stat_2">403 (35%)</td>
<td class="gt_row gt_center" headers="stat_3">412 (36%)</td>
<td class="gt_row gt_center" headers="stat_4">360 (31%)</td>
<td class="gt_row gt_center" headers="stat_5">373 (32%)</td>
<td class="gt_row gt_center" headers="stat_6">360 (31%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2c</td>
<td class="gt_row gt_center" headers="stat_1">11 (4.7%)</td>
<td class="gt_row gt_center" headers="stat_2">29 (2.5%)</td>
<td class="gt_row gt_center" headers="stat_3">34 (3.0%)</td>
<td class="gt_row gt_center" headers="stat_4">44 (3.8%)</td>
<td class="gt_row gt_center" headers="stat_5">35 (3.0%)</td>
<td class="gt_row gt_center" headers="stat_6">38 (3.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3a</td>
<td class="gt_row gt_center" headers="stat_1">18 (7.7%)</td>
<td class="gt_row gt_center" headers="stat_2">36 (3.1%)</td>
<td class="gt_row gt_center" headers="stat_3">44 (3.8%)</td>
<td class="gt_row gt_center" headers="stat_4">60 (5.2%)</td>
<td class="gt_row gt_center" headers="stat_5">61 (5.3%)</td>
<td class="gt_row gt_center" headers="stat_6">69 (6.0%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3f</td>
<td class="gt_row gt_center" headers="stat_1">74 (31%)</td>
<td class="gt_row gt_center" headers="stat_2">193 (17%)</td>
<td class="gt_row gt_center" headers="stat_3">302 (26%)</td>
<td class="gt_row gt_center" headers="stat_4">296 (26%)</td>
<td class="gt_row gt_center" headers="stat_5">335 (29%)</td>
<td class="gt_row gt_center" headers="stat_6">336 (29%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy4</td>
<td class="gt_row gt_center" headers="stat_1">10 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_2">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_3">42 (3.7%)</td>
<td class="gt_row gt_center" headers="stat_4">24 (2.1%)</td>
<td class="gt_row gt_center" headers="stat_5">37 (3.2%)</td>
<td class="gt_row gt_center" headers="stat_6">29 (2.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy5</td>
<td class="gt_row gt_center" headers="stat_1">17 (7.2%)</td>
<td class="gt_row gt_center" headers="stat_2">91 (7.9%)</td>
<td class="gt_row gt_center" headers="stat_3">107 (9.3%)</td>
<td class="gt_row gt_center" headers="stat_4">85 (7.4%)</td>
<td class="gt_row gt_center" headers="stat_5">86 (7.5%)</td>
<td class="gt_row gt_center" headers="stat_6">70 (6.1%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">915</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">final_pathology</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Benign</td>
<td class="gt_row gt_center" headers="stat_1">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_2">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_3">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_4">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_5">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_6">1,050 (91%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Cancer</td>
<td class="gt_row gt_center" headers="stat_1">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_2">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_3">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_4">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_5">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_6">100 (8.7%)</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="even">
<td colspan="7" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>Median (IQR); n (%)</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</div>
<div id="tabset-7-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-7-2-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [32]:</pre></div><div id="tbl-imputation-summary-cart" class="cell anchored" data-tbl-caption="Summary of data imputed via CART.">
<div class="sourceCode cell-code" id="cb75"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb75-1"><a href="#cb75-1" aria-hidden="true" tabindex="-1"></a>mice_cart<span class="sc">$</span>imputed <span class="sc">|&gt;</span> gtsummary<span class="sc">::</span><span class="fu">tbl_summary</span>(<span class="at">by=</span><span class="st">".imp"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="qobgamgxwi" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#qobgamgxwi table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#qobgamgxwi thead, #qobgamgxwi tbody, #qobgamgxwi tfoot, #qobgamgxwi tr, #qobgamgxwi td, #qobgamgxwi th {
  border-style: none;
}

#qobgamgxwi p {
  margin: 0;
  padding: 0;
}

#qobgamgxwi .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#qobgamgxwi .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#qobgamgxwi .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#qobgamgxwi .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#qobgamgxwi .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#qobgamgxwi .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qobgamgxwi .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#qobgamgxwi .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#qobgamgxwi .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#qobgamgxwi .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#qobgamgxwi .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#qobgamgxwi .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#qobgamgxwi .gt_spanner_row {
  border-bottom-style: hidden;
}

#qobgamgxwi .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#qobgamgxwi .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#qobgamgxwi .gt_from_md > :first-child {
  margin-top: 0;
}

#qobgamgxwi .gt_from_md > :last-child {
  margin-bottom: 0;
}

#qobgamgxwi .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#qobgamgxwi .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#qobgamgxwi .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#qobgamgxwi .gt_row_group_first td {
  border-top-width: 2px;
}

#qobgamgxwi .gt_row_group_first th {
  border-top-width: 2px;
}

#qobgamgxwi .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#qobgamgxwi .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#qobgamgxwi .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#qobgamgxwi .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qobgamgxwi .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#qobgamgxwi .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#qobgamgxwi .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#qobgamgxwi .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#qobgamgxwi .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qobgamgxwi .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#qobgamgxwi .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#qobgamgxwi .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#qobgamgxwi .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#qobgamgxwi .gt_left {
  text-align: left;
}

#qobgamgxwi .gt_center {
  text-align: center;
}

#qobgamgxwi .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#qobgamgxwi .gt_font_normal {
  font-weight: normal;
}

#qobgamgxwi .gt_font_bold {
  font-weight: bold;
}

#qobgamgxwi .gt_font_italic {
  font-style: italic;
}

#qobgamgxwi .gt_super {
  font-size: 65%;
}

#qobgamgxwi .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#qobgamgxwi .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#qobgamgxwi .gt_indent_1 {
  text-indent: 5px;
}

#qobgamgxwi .gt_indent_2 {
  text-indent: 10px;
}

#qobgamgxwi .gt_indent_3 {
  text-indent: 15px;
}

#qobgamgxwi .gt_indent_4 {
  text-indent: 20px;
}

#qobgamgxwi .gt_indent_5 {
  text-indent: 25px;
}

#qobgamgxwi .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#qobgamgxwi div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPcmlnaW5hbCoqLCBOID0gMSwxNTA=&quot;><div class='gt_from_md'><p><strong>Original</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>Original</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KioxKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>1</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>1</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KioyKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>2</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>2</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KiozKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>3</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>3</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio0KiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>4</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>4</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio1KiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>5</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>5</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">.id</td>
<td class="gt_row gt_center" headers="stat_1">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_2">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_3">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_4">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_5">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_6">576 (288, 863)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="stat_1">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_2">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_3">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_4">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_5">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_6">55 (41, 68)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;F</td>
<td class="gt_row gt_center" headers="stat_1">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_2">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_3">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_4">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_5">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_6">903 (79%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;M</td>
<td class="gt_row gt_center" headers="stat_1">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_2">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_3">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_4">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_5">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_6">247 (21%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">ethnicity</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;A</td>
<td class="gt_row gt_center" headers="stat_1">812 (74%)</td>
<td class="gt_row gt_center" headers="stat_2">843 (73%)</td>
<td class="gt_row gt_center" headers="stat_3">842 (73%)</td>
<td class="gt_row gt_center" headers="stat_4">843 (73%)</td>
<td class="gt_row gt_center" headers="stat_5">845 (73%)</td>
<td class="gt_row gt_center" headers="stat_6">845 (73%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;B</td>
<td class="gt_row gt_center" headers="stat_1">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_2">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">3 (0.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;C</td>
<td class="gt_row gt_center" headers="stat_1">44 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_2">46 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_3">46 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_4">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_5">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_6">47 (4.1%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;D</td>
<td class="gt_row gt_center" headers="stat_1">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_2">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_3">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_4">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_5">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_6">2 (0.2%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;F</td>
<td class="gt_row gt_center" headers="stat_1">4 (0.4%)</td>
<td class="gt_row gt_center" headers="stat_2">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">5 (0.4%)</td>
<td class="gt_row gt_center" headers="stat_5">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">4 (0.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;G</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;H</td>
<td class="gt_row gt_center" headers="stat_1">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_2">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_3">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_4">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">8 (0.7%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;J</td>
<td class="gt_row gt_center" headers="stat_1">46 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_2">48 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_3">48 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_4">52 (4.5%)</td>
<td class="gt_row gt_center" headers="stat_5">49 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_6">46 (4.0%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;K</td>
<td class="gt_row gt_center" headers="stat_1">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_2">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_3">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_4">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_5">12 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_6">11 (1.0%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;L</td>
<td class="gt_row gt_center" headers="stat_1">16 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_2">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_3">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_4">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_5">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_6">17 (1.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;M</td>
<td class="gt_row gt_center" headers="stat_1">16 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_2">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_3">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_4">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_5">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_6">19 (1.7%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;N</td>
<td class="gt_row gt_center" headers="stat_1">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">21 (1.8%)</td>
<td class="gt_row gt_center" headers="stat_4">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_5">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">18 (1.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;P</td>
<td class="gt_row gt_center" headers="stat_1">14 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_2">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_3">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_4">15 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_5">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_6">15 (1.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;R</td>
<td class="gt_row gt_center" headers="stat_1">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_2">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_5">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_6">6 (0.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;S</td>
<td class="gt_row gt_center" headers="stat_1">38 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_2">41 (3.6%)</td>
<td class="gt_row gt_center" headers="stat_3">39 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_4">38 (3.3%)</td>
<td class="gt_row gt_center" headers="stat_5">41 (3.6%)</td>
<td class="gt_row gt_center" headers="stat_6">42 (3.7%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Z</td>
<td class="gt_row gt_center" headers="stat_1">57 (5.2%)</td>
<td class="gt_row gt_center" headers="stat_2">61 (5.3%)</td>
<td class="gt_row gt_center" headers="stat_3">63 (5.5%)</td>
<td class="gt_row gt_center" headers="stat_4">60 (5.2%)</td>
<td class="gt_row gt_center" headers="stat_5">58 (5.0%)</td>
<td class="gt_row gt_center" headers="stat_6">60 (5.2%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">48</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">incidental_nodule</td>
<td class="gt_row gt_center" headers="stat_1">620 (54%)</td>
<td class="gt_row gt_center" headers="stat_2">623 (54%)</td>
<td class="gt_row gt_center" headers="stat_3">624 (54%)</td>
<td class="gt_row gt_center" headers="stat_4">624 (54%)</td>
<td class="gt_row gt_center" headers="stat_5">623 (54%)</td>
<td class="gt_row gt_center" headers="stat_6">624 (54%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">5</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">palpable_nodule</td>
<td class="gt_row gt_center" headers="stat_1">441 (40%)</td>
<td class="gt_row gt_center" headers="stat_2">470 (41%)</td>
<td class="gt_row gt_center" headers="stat_3">472 (41%)</td>
<td class="gt_row gt_center" headers="stat_4">468 (41%)</td>
<td class="gt_row gt_center" headers="stat_5">472 (41%)</td>
<td class="gt_row gt_center" headers="stat_6">468 (41%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">58</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">rapid_enlargement</td>
<td class="gt_row gt_center" headers="stat_1">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">20 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_4">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_5">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">21 (1.8%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">43</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">compressive_symptoms</td>
<td class="gt_row gt_center" headers="stat_1">88 (8.4%)</td>
<td class="gt_row gt_center" headers="stat_2">102 (8.9%)</td>
<td class="gt_row gt_center" headers="stat_3">105 (9.1%)</td>
<td class="gt_row gt_center" headers="stat_4">104 (9.0%)</td>
<td class="gt_row gt_center" headers="stat_5">101 (8.8%)</td>
<td class="gt_row gt_center" headers="stat_6">99 (8.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">106</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">hypertension</td>
<td class="gt_row gt_center" headers="stat_1">262 (26%)</td>
<td class="gt_row gt_center" headers="stat_2">291 (25%)</td>
<td class="gt_row gt_center" headers="stat_3">293 (25%)</td>
<td class="gt_row gt_center" headers="stat_4">292 (25%)</td>
<td class="gt_row gt_center" headers="stat_5">291 (25%)</td>
<td class="gt_row gt_center" headers="stat_6">292 (25%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">126</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">vocal_cord_paresis</td>
<td class="gt_row gt_center" headers="stat_1">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_2">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">3 (0.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">76</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">graves_disease</td>
<td class="gt_row gt_center" headers="stat_1">17 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_2">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_3">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_4">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_5">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_6">17 (1.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">67</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">hashimotos_thyroiditis</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">8 (0.7%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">73</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">family_history_thyroid_cancer</td>
<td class="gt_row gt_center" headers="stat_1">8 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_2">12 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_3">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_4">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_5">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_6">16 (1.4%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">281</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">exposure_radiation</td>
<td class="gt_row gt_center" headers="stat_1">9 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_2">10 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_3">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_4">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_5">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_6">10 (0.9%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">121</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Albumin</td>
<td class="gt_row gt_center" headers="stat_1">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_2">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_3">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_4">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_5">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_6">45.0 (43.0, 47.0)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">515</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">TSH value</td>
<td class="gt_row gt_center" headers="stat_1">1.48 (0.85, 2.30)</td>
<td class="gt_row gt_center" headers="stat_2">1.50 (0.88, 2.39)</td>
<td class="gt_row gt_center" headers="stat_3">1.50 (0.88, 2.50)</td>
<td class="gt_row gt_center" headers="stat_4">1.50 (0.91, 2.50)</td>
<td class="gt_row gt_center" headers="stat_5">1.50 (0.90, 2.40)</td>
<td class="gt_row gt_center" headers="stat_6">1.42 (0.85, 2.30)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">413</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Lymphocytes</td>
<td class="gt_row gt_center" headers="stat_1">1.94 (1.51, 2.43)</td>
<td class="gt_row gt_center" headers="stat_2">1.94 (1.54, 2.43)</td>
<td class="gt_row gt_center" headers="stat_3">1.94 (1.51, 2.43)</td>
<td class="gt_row gt_center" headers="stat_4">1.95 (1.53, 2.44)</td>
<td class="gt_row gt_center" headers="stat_5">1.91 (1.50, 2.41)</td>
<td class="gt_row gt_center" headers="stat_6">1.94 (1.50, 2.43)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">359</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Monocytes</td>
<td class="gt_row gt_center" headers="stat_1">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_2">0.53 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_3">0.53 (0.43, 0.66)</td>
<td class="gt_row gt_center" headers="stat_4">0.53 (0.43, 0.66)</td>
<td class="gt_row gt_center" headers="stat_5">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_6">0.52 (0.42, 0.65)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">363</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">bta_u_classification</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U1</td>
<td class="gt_row gt_center" headers="stat_1">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_2">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_3">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_4">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_5">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_6">2 (0.2%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U2</td>
<td class="gt_row gt_center" headers="stat_1">860 (78%)</td>
<td class="gt_row gt_center" headers="stat_2">894 (78%)</td>
<td class="gt_row gt_center" headers="stat_3">899 (78%)</td>
<td class="gt_row gt_center" headers="stat_4">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_5">897 (78%)</td>
<td class="gt_row gt_center" headers="stat_6">897 (78%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U3</td>
<td class="gt_row gt_center" headers="stat_1">210 (19%)</td>
<td class="gt_row gt_center" headers="stat_2">226 (20%)</td>
<td class="gt_row gt_center" headers="stat_3">219 (19%)</td>
<td class="gt_row gt_center" headers="stat_4">215 (19%)</td>
<td class="gt_row gt_center" headers="stat_5">220 (19%)</td>
<td class="gt_row gt_center" headers="stat_6">220 (19%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U4</td>
<td class="gt_row gt_center" headers="stat_1">22 (2.0%)</td>
<td class="gt_row gt_center" headers="stat_2">22 (1.9%)</td>
<td class="gt_row gt_center" headers="stat_3">24 (2.1%)</td>
<td class="gt_row gt_center" headers="stat_4">23 (2.0%)</td>
<td class="gt_row gt_center" headers="stat_5">24 (2.1%)</td>
<td class="gt_row gt_center" headers="stat_6">24 (2.1%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U5</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">50</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">solitary_nodule</td>
<td class="gt_row gt_center" headers="stat_1">320 (28%)</td>
<td class="gt_row gt_center" headers="stat_2">322 (28%)</td>
<td class="gt_row gt_center" headers="stat_3">322 (28%)</td>
<td class="gt_row gt_center" headers="stat_4">322 (28%)</td>
<td class="gt_row gt_center" headers="stat_5">322 (28%)</td>
<td class="gt_row gt_center" headers="stat_6">323 (28%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">8</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Nodule size (mm)</td>
<td class="gt_row gt_center" headers="stat_1">14 (7, 28)</td>
<td class="gt_row gt_center" headers="stat_2">13 (6, 26)</td>
<td class="gt_row gt_center" headers="stat_3">12 (6, 27)</td>
<td class="gt_row gt_center" headers="stat_4">13 (6, 26)</td>
<td class="gt_row gt_center" headers="stat_5">12 (6, 27)</td>
<td class="gt_row gt_center" headers="stat_6">13 (6, 26)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">319</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">cervical_lymphadenopathy</td>
<td class="gt_row gt_center" headers="stat_1">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_2">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_3">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_4">27 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_5">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_6">26 (2.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">9</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">thy_classification</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1</td>
<td class="gt_row gt_center" headers="stat_1">34 (14%)</td>
<td class="gt_row gt_center" headers="stat_2">203 (18%)</td>
<td class="gt_row gt_center" headers="stat_3">192 (17%)</td>
<td class="gt_row gt_center" headers="stat_4">187 (16%)</td>
<td class="gt_row gt_center" headers="stat_5">221 (19%)</td>
<td class="gt_row gt_center" headers="stat_6">188 (16%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1c</td>
<td class="gt_row gt_center" headers="stat_1">8 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_2">46 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_3">35 (3.0%)</td>
<td class="gt_row gt_center" headers="stat_4">36 (3.1%)</td>
<td class="gt_row gt_center" headers="stat_5">59 (5.1%)</td>
<td class="gt_row gt_center" headers="stat_6">63 (5.5%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2</td>
<td class="gt_row gt_center" headers="stat_1">63 (27%)</td>
<td class="gt_row gt_center" headers="stat_2">278 (24%)</td>
<td class="gt_row gt_center" headers="stat_3">272 (24%)</td>
<td class="gt_row gt_center" headers="stat_4">281 (24%)</td>
<td class="gt_row gt_center" headers="stat_5">258 (22%)</td>
<td class="gt_row gt_center" headers="stat_6">237 (21%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2c</td>
<td class="gt_row gt_center" headers="stat_1">11 (4.7%)</td>
<td class="gt_row gt_center" headers="stat_2">45 (3.9%)</td>
<td class="gt_row gt_center" headers="stat_3">70 (6.1%)</td>
<td class="gt_row gt_center" headers="stat_4">58 (5.0%)</td>
<td class="gt_row gt_center" headers="stat_5">70 (6.1%)</td>
<td class="gt_row gt_center" headers="stat_6">76 (6.6%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3a</td>
<td class="gt_row gt_center" headers="stat_1">18 (7.7%)</td>
<td class="gt_row gt_center" headers="stat_2">60 (5.2%)</td>
<td class="gt_row gt_center" headers="stat_3">75 (6.5%)</td>
<td class="gt_row gt_center" headers="stat_4">74 (6.4%)</td>
<td class="gt_row gt_center" headers="stat_5">59 (5.1%)</td>
<td class="gt_row gt_center" headers="stat_6">125 (11%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3f</td>
<td class="gt_row gt_center" headers="stat_1">74 (31%)</td>
<td class="gt_row gt_center" headers="stat_2">347 (30%)</td>
<td class="gt_row gt_center" headers="stat_3">321 (28%)</td>
<td class="gt_row gt_center" headers="stat_4">317 (28%)</td>
<td class="gt_row gt_center" headers="stat_5">325 (28%)</td>
<td class="gt_row gt_center" headers="stat_6">264 (23%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy4</td>
<td class="gt_row gt_center" headers="stat_1">10 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_2">54 (4.7%)</td>
<td class="gt_row gt_center" headers="stat_3">69 (6.0%)</td>
<td class="gt_row gt_center" headers="stat_4">79 (6.9%)</td>
<td class="gt_row gt_center" headers="stat_5">68 (5.9%)</td>
<td class="gt_row gt_center" headers="stat_6">74 (6.4%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy5</td>
<td class="gt_row gt_center" headers="stat_1">17 (7.2%)</td>
<td class="gt_row gt_center" headers="stat_2">117 (10%)</td>
<td class="gt_row gt_center" headers="stat_3">116 (10%)</td>
<td class="gt_row gt_center" headers="stat_4">118 (10%)</td>
<td class="gt_row gt_center" headers="stat_5">90 (7.8%)</td>
<td class="gt_row gt_center" headers="stat_6">123 (11%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">915</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">final_pathology</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Benign</td>
<td class="gt_row gt_center" headers="stat_1">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_2">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_3">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_4">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_5">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_6">1,050 (91%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Cancer</td>
<td class="gt_row gt_center" headers="stat_1">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_2">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_3">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_4">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_5">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_6">100 (8.7%)</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="even">
<td colspan="7" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>Median (IQR); n (%)</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</div>
<div id="tabset-7-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-7-3-tab">
<div class="cell-container"><div class="cell-decorator"><pre>In [33]:</pre></div><div id="tbl-imputation-summary-rf" class="cell anchored" data-tbl-caption="Summary of data imputed via Random Forest.">
<div class="sourceCode cell-code" id="cb76"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb76-1"><a href="#cb76-1" aria-hidden="true" tabindex="-1"></a>mice_rf<span class="sc">$</span>imputed <span class="sc">|&gt;</span> gtsummary<span class="sc">::</span><span class="fu">tbl_summary</span>(<span class="at">by =</span> <span class="st">".imp"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="glmvntnqgq" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#glmvntnqgq table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#glmvntnqgq thead, #glmvntnqgq tbody, #glmvntnqgq tfoot, #glmvntnqgq tr, #glmvntnqgq td, #glmvntnqgq th {
  border-style: none;
}

#glmvntnqgq p {
  margin: 0;
  padding: 0;
}

#glmvntnqgq .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#glmvntnqgq .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#glmvntnqgq .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#glmvntnqgq .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#glmvntnqgq .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#glmvntnqgq .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#glmvntnqgq .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#glmvntnqgq .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#glmvntnqgq .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#glmvntnqgq .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#glmvntnqgq .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#glmvntnqgq .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#glmvntnqgq .gt_spanner_row {
  border-bottom-style: hidden;
}

#glmvntnqgq .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#glmvntnqgq .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#glmvntnqgq .gt_from_md > :first-child {
  margin-top: 0;
}

#glmvntnqgq .gt_from_md > :last-child {
  margin-bottom: 0;
}

#glmvntnqgq .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#glmvntnqgq .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#glmvntnqgq .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#glmvntnqgq .gt_row_group_first td {
  border-top-width: 2px;
}

#glmvntnqgq .gt_row_group_first th {
  border-top-width: 2px;
}

#glmvntnqgq .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#glmvntnqgq .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#glmvntnqgq .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#glmvntnqgq .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#glmvntnqgq .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#glmvntnqgq .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#glmvntnqgq .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#glmvntnqgq .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#glmvntnqgq .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#glmvntnqgq .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#glmvntnqgq .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#glmvntnqgq .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#glmvntnqgq .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#glmvntnqgq .gt_left {
  text-align: left;
}

#glmvntnqgq .gt_center {
  text-align: center;
}

#glmvntnqgq .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#glmvntnqgq .gt_font_normal {
  font-weight: normal;
}

#glmvntnqgq .gt_font_bold {
  font-weight: bold;
}

#glmvntnqgq .gt_font_italic {
  font-style: italic;
}

#glmvntnqgq .gt_super {
  font-size: 65%;
}

#glmvntnqgq .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#glmvntnqgq .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#glmvntnqgq .gt_indent_1 {
  text-indent: 5px;
}

#glmvntnqgq .gt_indent_2 {
  text-indent: 10px;
}

#glmvntnqgq .gt_indent_3 {
  text-indent: 15px;
}

#glmvntnqgq .gt_indent_4 {
  text-indent: 20px;
}

#glmvntnqgq .gt_indent_5 {
  text-indent: 25px;
}

#glmvntnqgq .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#glmvntnqgq div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPcmlnaW5hbCoqLCBOID0gMSwxNTA=&quot;><div class='gt_from_md'><p><strong>Original</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>Original</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KioxKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>1</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>1</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KioyKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>2</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>2</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KiozKiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>3</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>3</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio0KiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>4</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>4</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio1KiosIE4gPSAxLDE1MA==&quot;><div class='gt_from_md'><p><strong>5</strong>, N = 1,150</p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>5</strong>, N = 1,150</p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">.id</td>
<td class="gt_row gt_center" headers="stat_1">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_2">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_3">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_4">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_5">576 (288, 863)</td>
<td class="gt_row gt_center" headers="stat_6">576 (288, 863)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="stat_1">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_2">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_3">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_4">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_5">55 (41, 68)</td>
<td class="gt_row gt_center" headers="stat_6">55 (41, 68)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;F</td>
<td class="gt_row gt_center" headers="stat_1">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_2">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_3">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_4">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_5">903 (79%)</td>
<td class="gt_row gt_center" headers="stat_6">903 (79%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;M</td>
<td class="gt_row gt_center" headers="stat_1">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_2">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_3">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_4">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_5">247 (21%)</td>
<td class="gt_row gt_center" headers="stat_6">247 (21%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">ethnicity</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;A</td>
<td class="gt_row gt_center" headers="stat_1">812 (74%)</td>
<td class="gt_row gt_center" headers="stat_2">851 (74%)</td>
<td class="gt_row gt_center" headers="stat_3">845 (73%)</td>
<td class="gt_row gt_center" headers="stat_4">850 (74%)</td>
<td class="gt_row gt_center" headers="stat_5">849 (74%)</td>
<td class="gt_row gt_center" headers="stat_6">853 (74%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;B</td>
<td class="gt_row gt_center" headers="stat_1">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_2">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">3 (0.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;C</td>
<td class="gt_row gt_center" headers="stat_1">44 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_2">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_3">46 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_4">46 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_5">45 (3.9%)</td>
<td class="gt_row gt_center" headers="stat_6">45 (3.9%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;D</td>
<td class="gt_row gt_center" headers="stat_1">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_2">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_3">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_4">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_5">2 (0.2%)</td>
<td class="gt_row gt_center" headers="stat_6">2 (0.2%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;F</td>
<td class="gt_row gt_center" headers="stat_1">4 (0.4%)</td>
<td class="gt_row gt_center" headers="stat_2">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">4 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">4 (0.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;G</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_5">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_6">8 (0.7%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;H</td>
<td class="gt_row gt_center" headers="stat_1">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_2">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_3">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_4">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">8 (0.7%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;J</td>
<td class="gt_row gt_center" headers="stat_1">46 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_2">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_3">49 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_4">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_5">49 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_6">48 (4.2%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;K</td>
<td class="gt_row gt_center" headers="stat_1">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_2">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_3">15 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_4">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_5">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_6">11 (1.0%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;L</td>
<td class="gt_row gt_center" headers="stat_1">16 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_2">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_3">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_4">16 (1.4%)</td>
<td class="gt_row gt_center" headers="stat_5">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_6">16 (1.4%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;M</td>
<td class="gt_row gt_center" headers="stat_1">16 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_2">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_3">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_4">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_5">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_6">16 (1.4%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;N</td>
<td class="gt_row gt_center" headers="stat_1">18 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_4">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_5">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">20 (1.7%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;P</td>
<td class="gt_row gt_center" headers="stat_1">14 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_2">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_3">15 (1.3%)</td>
<td class="gt_row gt_center" headers="stat_4">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_5">14 (1.2%)</td>
<td class="gt_row gt_center" headers="stat_6">14 (1.2%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;R</td>
<td class="gt_row gt_center" headers="stat_1">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_2">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_3">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_4">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_5">6 (0.5%)</td>
<td class="gt_row gt_center" headers="stat_6">6 (0.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;S</td>
<td class="gt_row gt_center" headers="stat_1">38 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_2">40 (3.5%)</td>
<td class="gt_row gt_center" headers="stat_3">39 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_4">39 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_5">40 (3.5%)</td>
<td class="gt_row gt_center" headers="stat_6">39 (3.4%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Z</td>
<td class="gt_row gt_center" headers="stat_1">57 (5.2%)</td>
<td class="gt_row gt_center" headers="stat_2">58 (5.0%)</td>
<td class="gt_row gt_center" headers="stat_3">59 (5.1%)</td>
<td class="gt_row gt_center" headers="stat_4">59 (5.1%)</td>
<td class="gt_row gt_center" headers="stat_5">58 (5.0%)</td>
<td class="gt_row gt_center" headers="stat_6">57 (5.0%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">48</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">incidental_nodule</td>
<td class="gt_row gt_center" headers="stat_1">620 (54%)</td>
<td class="gt_row gt_center" headers="stat_2">623 (54%)</td>
<td class="gt_row gt_center" headers="stat_3">623 (54%)</td>
<td class="gt_row gt_center" headers="stat_4">624 (54%)</td>
<td class="gt_row gt_center" headers="stat_5">624 (54%)</td>
<td class="gt_row gt_center" headers="stat_6">622 (54%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">5</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">palpable_nodule</td>
<td class="gt_row gt_center" headers="stat_1">441 (40%)</td>
<td class="gt_row gt_center" headers="stat_2">462 (40%)</td>
<td class="gt_row gt_center" headers="stat_3">472 (41%)</td>
<td class="gt_row gt_center" headers="stat_4">465 (40%)</td>
<td class="gt_row gt_center" headers="stat_5">469 (41%)</td>
<td class="gt_row gt_center" headers="stat_6">467 (41%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">58</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">rapid_enlargement</td>
<td class="gt_row gt_center" headers="stat_1">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_2">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_3">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_4">19 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_5">20 (1.7%)</td>
<td class="gt_row gt_center" headers="stat_6">20 (1.7%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">43</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">compressive_symptoms</td>
<td class="gt_row gt_center" headers="stat_1">88 (8.4%)</td>
<td class="gt_row gt_center" headers="stat_2">94 (8.2%)</td>
<td class="gt_row gt_center" headers="stat_3">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_4">94 (8.2%)</td>
<td class="gt_row gt_center" headers="stat_5">89 (7.7%)</td>
<td class="gt_row gt_center" headers="stat_6">95 (8.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">106</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">hypertension</td>
<td class="gt_row gt_center" headers="stat_1">262 (26%)</td>
<td class="gt_row gt_center" headers="stat_2">282 (25%)</td>
<td class="gt_row gt_center" headers="stat_3">284 (25%)</td>
<td class="gt_row gt_center" headers="stat_4">285 (25%)</td>
<td class="gt_row gt_center" headers="stat_5">281 (24%)</td>
<td class="gt_row gt_center" headers="stat_6">286 (25%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">126</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">vocal_cord_paresis</td>
<td class="gt_row gt_center" headers="stat_1">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_2">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_3">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_4">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (0.3%)</td>
<td class="gt_row gt_center" headers="stat_6">3 (0.3%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">76</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">graves_disease</td>
<td class="gt_row gt_center" headers="stat_1">17 (1.6%)</td>
<td class="gt_row gt_center" headers="stat_2">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_3">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_4">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_5">17 (1.5%)</td>
<td class="gt_row gt_center" headers="stat_6">17 (1.5%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">67</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">hashimotos_thyroiditis</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_5">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">73</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">family_history_thyroid_cancer</td>
<td class="gt_row gt_center" headers="stat_1">8 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_2">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_3">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_4">11 (1.0%)</td>
<td class="gt_row gt_center" headers="stat_5">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_6">9 (0.8%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">281</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">exposure_radiation</td>
<td class="gt_row gt_center" headers="stat_1">9 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_2">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_3">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_4">10 (0.9%)</td>
<td class="gt_row gt_center" headers="stat_5">9 (0.8%)</td>
<td class="gt_row gt_center" headers="stat_6">9 (0.8%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">121</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Albumin</td>
<td class="gt_row gt_center" headers="stat_1">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_2">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_3">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_4">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_5">45.0 (43.0, 47.0)</td>
<td class="gt_row gt_center" headers="stat_6">45.0 (43.0, 47.0)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">515</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">TSH value</td>
<td class="gt_row gt_center" headers="stat_1">1.48 (0.85, 2.30)</td>
<td class="gt_row gt_center" headers="stat_2">1.50 (0.87, 2.40)</td>
<td class="gt_row gt_center" headers="stat_3">1.50 (0.90, 2.30)</td>
<td class="gt_row gt_center" headers="stat_4">1.50 (0.88, 2.30)</td>
<td class="gt_row gt_center" headers="stat_5">1.50 (0.87, 2.30)</td>
<td class="gt_row gt_center" headers="stat_6">1.50 (0.85, 2.40)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">413</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Lymphocytes</td>
<td class="gt_row gt_center" headers="stat_1">1.94 (1.51, 2.43)</td>
<td class="gt_row gt_center" headers="stat_2">1.94 (1.54, 2.43)</td>
<td class="gt_row gt_center" headers="stat_3">1.94 (1.51, 2.44)</td>
<td class="gt_row gt_center" headers="stat_4">1.95 (1.54, 2.44)</td>
<td class="gt_row gt_center" headers="stat_5">1.95 (1.53, 2.42)</td>
<td class="gt_row gt_center" headers="stat_6">1.94 (1.51, 2.44)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">359</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Monocytes</td>
<td class="gt_row gt_center" headers="stat_1">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_2">0.53 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_3">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_4">0.53 (0.43, 0.66)</td>
<td class="gt_row gt_center" headers="stat_5">0.52 (0.42, 0.66)</td>
<td class="gt_row gt_center" headers="stat_6">0.52 (0.42, 0.66)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">363</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">bta_u_classification</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U1</td>
<td class="gt_row gt_center" headers="stat_1">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_2">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_3">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_4">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_5">1 (&lt;0.1%)</td>
<td class="gt_row gt_center" headers="stat_6">1 (&lt;0.1%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U2</td>
<td class="gt_row gt_center" headers="stat_1">860 (78%)</td>
<td class="gt_row gt_center" headers="stat_2">902 (78%)</td>
<td class="gt_row gt_center" headers="stat_3">904 (79%)</td>
<td class="gt_row gt_center" headers="stat_4">904 (79%)</td>
<td class="gt_row gt_center" headers="stat_5">904 (79%)</td>
<td class="gt_row gt_center" headers="stat_6">899 (78%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U3</td>
<td class="gt_row gt_center" headers="stat_1">210 (19%)</td>
<td class="gt_row gt_center" headers="stat_2">216 (19%)</td>
<td class="gt_row gt_center" headers="stat_3">216 (19%)</td>
<td class="gt_row gt_center" headers="stat_4">216 (19%)</td>
<td class="gt_row gt_center" headers="stat_5">215 (19%)</td>
<td class="gt_row gt_center" headers="stat_6">219 (19%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U4</td>
<td class="gt_row gt_center" headers="stat_1">22 (2.0%)</td>
<td class="gt_row gt_center" headers="stat_2">23 (2.0%)</td>
<td class="gt_row gt_center" headers="stat_3">22 (1.9%)</td>
<td class="gt_row gt_center" headers="stat_4">22 (1.9%)</td>
<td class="gt_row gt_center" headers="stat_5">22 (1.9%)</td>
<td class="gt_row gt_center" headers="stat_6">24 (2.1%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U5</td>
<td class="gt_row gt_center" headers="stat_1">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_2">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_3">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_4">7 (0.6%)</td>
<td class="gt_row gt_center" headers="stat_5">8 (0.7%)</td>
<td class="gt_row gt_center" headers="stat_6">7 (0.6%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">50</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">solitary_nodule</td>
<td class="gt_row gt_center" headers="stat_1">320 (28%)</td>
<td class="gt_row gt_center" headers="stat_2">321 (28%)</td>
<td class="gt_row gt_center" headers="stat_3">321 (28%)</td>
<td class="gt_row gt_center" headers="stat_4">320 (28%)</td>
<td class="gt_row gt_center" headers="stat_5">323 (28%)</td>
<td class="gt_row gt_center" headers="stat_6">321 (28%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">8</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Nodule size (mm)</td>
<td class="gt_row gt_center" headers="stat_1">14 (7, 28)</td>
<td class="gt_row gt_center" headers="stat_2">13 (6, 27)</td>
<td class="gt_row gt_center" headers="stat_3">12 (6, 26)</td>
<td class="gt_row gt_center" headers="stat_4">12 (6, 26)</td>
<td class="gt_row gt_center" headers="stat_5">13 (6, 26)</td>
<td class="gt_row gt_center" headers="stat_6">13 (6, 26)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">319</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">cervical_lymphadenopathy</td>
<td class="gt_row gt_center" headers="stat_1">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_2">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_3">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_4">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_5">26 (2.3%)</td>
<td class="gt_row gt_center" headers="stat_6">26 (2.3%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">9</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">thy_classification</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1</td>
<td class="gt_row gt_center" headers="stat_1">34 (14%)</td>
<td class="gt_row gt_center" headers="stat_2">206 (18%)</td>
<td class="gt_row gt_center" headers="stat_3">173 (15%)</td>
<td class="gt_row gt_center" headers="stat_4">136 (12%)</td>
<td class="gt_row gt_center" headers="stat_5">175 (15%)</td>
<td class="gt_row gt_center" headers="stat_6">147 (13%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1c</td>
<td class="gt_row gt_center" headers="stat_1">8 (3.4%)</td>
<td class="gt_row gt_center" headers="stat_2">46 (4.0%)</td>
<td class="gt_row gt_center" headers="stat_3">38 (3.3%)</td>
<td class="gt_row gt_center" headers="stat_4">69 (6.0%)</td>
<td class="gt_row gt_center" headers="stat_5">36 (3.1%)</td>
<td class="gt_row gt_center" headers="stat_6">32 (2.8%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2</td>
<td class="gt_row gt_center" headers="stat_1">63 (27%)</td>
<td class="gt_row gt_center" headers="stat_2">299 (26%)</td>
<td class="gt_row gt_center" headers="stat_3">283 (25%)</td>
<td class="gt_row gt_center" headers="stat_4">323 (28%)</td>
<td class="gt_row gt_center" headers="stat_5">333 (29%)</td>
<td class="gt_row gt_center" headers="stat_6">341 (30%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2c</td>
<td class="gt_row gt_center" headers="stat_1">11 (4.7%)</td>
<td class="gt_row gt_center" headers="stat_2">57 (5.0%)</td>
<td class="gt_row gt_center" headers="stat_3">78 (6.8%)</td>
<td class="gt_row gt_center" headers="stat_4">71 (6.2%)</td>
<td class="gt_row gt_center" headers="stat_5">64 (5.6%)</td>
<td class="gt_row gt_center" headers="stat_6">40 (3.5%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3a</td>
<td class="gt_row gt_center" headers="stat_1">18 (7.7%)</td>
<td class="gt_row gt_center" headers="stat_2">105 (9.1%)</td>
<td class="gt_row gt_center" headers="stat_3">115 (10%)</td>
<td class="gt_row gt_center" headers="stat_4">117 (10%)</td>
<td class="gt_row gt_center" headers="stat_5">85 (7.4%)</td>
<td class="gt_row gt_center" headers="stat_6">97 (8.4%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3f</td>
<td class="gt_row gt_center" headers="stat_1">74 (31%)</td>
<td class="gt_row gt_center" headers="stat_2">291 (25%)</td>
<td class="gt_row gt_center" headers="stat_3">299 (26%)</td>
<td class="gt_row gt_center" headers="stat_4">306 (27%)</td>
<td class="gt_row gt_center" headers="stat_5">325 (28%)</td>
<td class="gt_row gt_center" headers="stat_6">359 (31%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy4</td>
<td class="gt_row gt_center" headers="stat_1">10 (4.3%)</td>
<td class="gt_row gt_center" headers="stat_2">48 (4.2%)</td>
<td class="gt_row gt_center" headers="stat_3">52 (4.5%)</td>
<td class="gt_row gt_center" headers="stat_4">47 (4.1%)</td>
<td class="gt_row gt_center" headers="stat_5">61 (5.3%)</td>
<td class="gt_row gt_center" headers="stat_6">48 (4.2%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy5</td>
<td class="gt_row gt_center" headers="stat_1">17 (7.2%)</td>
<td class="gt_row gt_center" headers="stat_2">98 (8.5%)</td>
<td class="gt_row gt_center" headers="stat_3">112 (9.7%)</td>
<td class="gt_row gt_center" headers="stat_4">81 (7.0%)</td>
<td class="gt_row gt_center" headers="stat_5">71 (6.2%)</td>
<td class="gt_row gt_center" headers="stat_6">86 (7.5%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Unknown</td>
<td class="gt_row gt_center" headers="stat_1">915</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
<td class="gt_row gt_center" headers="stat_4">0</td>
<td class="gt_row gt_center" headers="stat_5">0</td>
<td class="gt_row gt_center" headers="stat_6">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">final_pathology</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
<td class="gt_row gt_center" headers="stat_4"><br>
</td>
<td class="gt_row gt_center" headers="stat_5"><br>
</td>
<td class="gt_row gt_center" headers="stat_6"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Benign</td>
<td class="gt_row gt_center" headers="stat_1">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_2">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_3">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_4">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_5">1,050 (91%)</td>
<td class="gt_row gt_center" headers="stat_6">1,050 (91%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Cancer</td>
<td class="gt_row gt_center" headers="stat_1">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_2">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_3">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_4">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_5">100 (8.7%)</td>
<td class="gt_row gt_center" headers="stat_6">100 (8.7%)</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="even">
<td colspan="7" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>Median (IQR); n (%)</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</div>
</div>
</div>
</section>
</section>
<section id="modelling-1" class="level3" data-number="3.2">
<h3 data-number="3.2" class="anchored" data-anchor-id="modelling-1"><span class="header-section-number">3.2</span> Modelling</h3>
<p><strong>TODO</strong> - And in light of having removed <strong>?@tbl-data-completness</strong> in favour of the imputed datesets this too has been removed? (<code>@ns-rse</code> 2024-07-11). <strong>TODO</strong> - This table feels like duplication of <strong>?@tbl-data-completeness</strong>, perhaps have just one? (<code>@ns-rse</code> 2024-07-11).</p>
<p>The predictor variables selected to predict <code>final_pathology</code> are shown in <strong>?@tbl-predictors</strong></p>
<!-- ```{r} -->
<!-- #| label: tbl-predictors -->
<!-- #| purl: true -->
<!-- #| eval: true -->
<!-- #| echo: false -->
<!-- #| warning: false -->
<!-- #| tbl-caption: "Predictors evaluated in modelling." -->
<!-- df_predictors_evaluated <- df_complete |> -->
<!--   dplyr::ungroup() |> -->
<!--   gtsummary::tbl_summary( -->
<!--     type = all_continuous() ~ "continuous2", -->
<!--     statistic = all_continuous() ~ c( -->
<!--       "{N_nonmiss}", -->
<!--       "{mean} ({sd})", -->
<!--       "{median} ({p25}, {p75})", -->
<!--       "{min}, {max}" -->
<!--     ), -->
<!--     percent="column",      # Include percentages for categorical variables by "column", "row" or "cell" -->
<!--     missing="always"           # Exclude missing, options are "no", "ifany" and "always" -->
<!--   ) |> -->
<!--   gtsummary::modify_caption("Variables evaluated in the model") -->
<!-- df_predictors_evaluated -->
<!-- ``` -->
<p>Section that sets up the modelling</p>
<div class="cell-container"><div class="cell-decorator"><pre>In [34]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb77"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Prefer tidymodel commands (although in most places we use the convention &lt;pkg&gt;::&lt;function&gt;())</span></span>
<span id="cb77-2"><a href="#cb77-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb77-3"><a href="#cb77-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidymodels)</span>
<span id="cb77-4"><a href="#cb77-4" aria-hidden="true" tabindex="-1"></a>tidymodels<span class="sc">::</span><span class="fu">tidymodels_prefer</span>()</span>
<span id="cb77-5"><a href="#cb77-5" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">5039378</span>)</span>
<span id="cb77-6"><a href="#cb77-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb77-7"><a href="#cb77-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Use df_complete rather than df as this subset have data for all the variables of interest.</span></span>
<span id="cb77-8"><a href="#cb77-8" aria-hidden="true" tabindex="-1"></a><span class="do">## split &lt;- rsample::initial_split(df_complete, prop = 0.75)</span></span>
<span id="cb77-9"><a href="#cb77-9" aria-hidden="true" tabindex="-1"></a><span class="do">## @ns-rse (2024-07-18) - Use an imputed dataset instead</span></span>
<span id="cb77-10"><a href="#cb77-10" aria-hidden="true" tabindex="-1"></a>split <span class="ot">&lt;-</span> mice_rf<span class="sc">$</span>imputed <span class="sc">|&gt;</span></span>
<span id="cb77-11"><a href="#cb77-11" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">filter</span>(.imp <span class="sc">==</span> <span class="dv">1</span>) <span class="sc">|&gt;</span></span>
<span id="cb77-12"><a href="#cb77-12" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="sc">-</span>.imp, <span class="sc">-</span>.id) <span class="sc">|&gt;</span></span>
<span id="cb77-13"><a href="#cb77-13" aria-hidden="true" tabindex="-1"></a>    rsample<span class="sc">::</span><span class="fu">initial_split</span>(<span class="at">prop =</span> <span class="fl">0.75</span>)</span>
<span id="cb77-14"><a href="#cb77-14" aria-hidden="true" tabindex="-1"></a>train <span class="ot">&lt;-</span> rsample<span class="sc">::</span><span class="fu">training</span>(split)</span>
<span id="cb77-15"><a href="#cb77-15" aria-hidden="true" tabindex="-1"></a>test <span class="ot">&lt;-</span> rsample<span class="sc">::</span><span class="fu">testing</span>(split)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [35]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb78"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb78-1"><a href="#cb78-1" aria-hidden="true" tabindex="-1"></a>cv_folds <span class="ot">&lt;-</span> rsample<span class="sc">::</span><span class="fu">vfold_cv</span>(train, <span class="at">v =</span> <span class="dv">10</span>, <span class="at">repeats =</span> <span class="dv">10</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [36]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb79"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb79-1"><a href="#cb79-1" aria-hidden="true" tabindex="-1"></a>cv_loo <span class="ot">&lt;-</span> rsample<span class="sc">::</span><span class="fu">loo_cv</span>(train)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [37]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb80"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb80-1"><a href="#cb80-1" aria-hidden="true" tabindex="-1"></a><span class="do">## NB This is the key section where the variables that are to be used in the model are defined. A dependent variable</span></span>
<span id="cb80-2"><a href="#cb80-2" aria-hidden="true" tabindex="-1"></a><span class="do">## (the outcome of interest) is in this case the `final_pathology`, whether individuals have malignant or benign tumors,</span></span>
<span id="cb80-3"><a href="#cb80-3" aria-hidden="true" tabindex="-1"></a><span class="do">## this appears on the left-hand side of the equation (before the tilde `~`). On the right of the equation are the</span></span>
<span id="cb80-4"><a href="#cb80-4" aria-hidden="true" tabindex="-1"></a><span class="do">## predictor or dependant variables</span></span>
<span id="cb80-5"><a href="#cb80-5" aria-hidden="true" tabindex="-1"></a><span class="do">##</span></span>
<span id="cb80-6"><a href="#cb80-6" aria-hidden="true" tabindex="-1"></a><span class="do">## @ns-rse 2024-06-14 :</span></span>
<span id="cb80-7"><a href="#cb80-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Because we have used dplyr::select() to choose _just_ the columns of interest we can use the '.'</span></span>
<span id="cb80-8"><a href="#cb80-8" aria-hidden="true" tabindex="-1"></a><span class="do">## notation to refer to "all other variables" as being predictors. This is useful as it saves duplication of writing</span></span>
<span id="cb80-9"><a href="#cb80-9" aria-hidden="true" tabindex="-1"></a><span class="do">## everything out which leaves scope for some being missed.</span></span>
<span id="cb80-10"><a href="#cb80-10" aria-hidden="true" tabindex="-1"></a>thyroid_recipe <span class="ot">&lt;-</span> recipes<span class="sc">::</span><span class="fu">recipe</span>(final_pathology <span class="sc">~</span> ., <span class="at">data =</span> train) <span class="sc">|&gt;</span></span>
<span id="cb80-11"><a href="#cb80-11" aria-hidden="true" tabindex="-1"></a>  <span class="do">## @ns-rse 2024-06-14 :</span></span>
<span id="cb80-12"><a href="#cb80-12" aria-hidden="true" tabindex="-1"></a>  <span class="do">## This step can be used to filter observations with missing data, see the manual pages for more details</span></span>
<span id="cb80-13"><a href="#cb80-13" aria-hidden="true" tabindex="-1"></a>  <span class="do">## https://recipes.tidymodels.org/reference/step_filter_missing.html</span></span>
<span id="cb80-14"><a href="#cb80-14" aria-hidden="true" tabindex="-1"></a>  recipes<span class="sc">::</span><span class="fu">step_filter_missing</span>(recipes<span class="sc">::</span><span class="fu">all_predictors</span>(), <span class="at">threshold =</span> <span class="dv">0</span>) <span class="sc">|&gt;</span></span>
<span id="cb80-15"><a href="#cb80-15" aria-hidden="true" tabindex="-1"></a>  <span class="do">## @ns-rse 2024-06-14 :</span></span>
<span id="cb80-16"><a href="#cb80-16" aria-hidden="true" tabindex="-1"></a>  <span class="do">## We first normalise the data _before_ we generate dummies otherwise the dummies, which are numerical, get normalised</span></span>
<span id="cb80-17"><a href="#cb80-17" aria-hidden="true" tabindex="-1"></a>  <span class="do">## too</span></span>
<span id="cb80-18"><a href="#cb80-18" aria-hidden="true" tabindex="-1"></a>  recipes<span class="sc">::</span><span class="fu">step_normalize</span>(recipes<span class="sc">::</span><span class="fu">all_numeric_predictors</span>()) <span class="sc">|&gt;</span></span>
<span id="cb80-19"><a href="#cb80-19" aria-hidden="true" tabindex="-1"></a>  recipes<span class="sc">::</span><span class="fu">step_dummy</span>(recipes<span class="sc">::</span><span class="fu">all_nominal_predictors</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [38]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb81"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb81-1"><a href="#cb81-1" aria-hidden="true" tabindex="-1"></a>thyroid_workflow <span class="ot">&lt;-</span> workflows<span class="sc">::</span><span class="fu">workflow</span>() <span class="sc">|&gt;</span></span>
<span id="cb81-2"><a href="#cb81-2" aria-hidden="true" tabindex="-1"></a>  workflows<span class="sc">::</span><span class="fu">add_recipe</span>(thyroid_recipe)</span>
<span id="cb81-3"><a href="#cb81-3" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(thyroid_workflow, <span class="at">file =</span> <span class="fu">paste</span>(r_dir, <span class="st">"thyroid_workflow.rds"</span>, <span class="at">sep =</span> <span class="st">"/"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<p>The following section is output from a Tidymodel approach to logistic regression to try and work out why variables are not being included.</p>
<div class="cell-container"><div class="cell-decorator"><pre>In [39]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb82"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a><span class="do">## define binary logistic regression model</span></span>
<span id="cb82-2"><a href="#cb82-2" aria-hidden="true" tabindex="-1"></a>logistic_model <span class="ot">&lt;-</span> <span class="fu">logistic_reg</span>() <span class="sc">|&gt;</span></span>
<span id="cb82-3"><a href="#cb82-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_engine</span>(<span class="st">"glm"</span>)</span>
<span id="cb82-4"><a href="#cb82-4" aria-hidden="true" tabindex="-1"></a><span class="do">## add the binary logistic regression model to the thyroid workflow</span></span>
<span id="cb82-5"><a href="#cb82-5" aria-hidden="true" tabindex="-1"></a>log_thyroid_workflow <span class="ot">&lt;-</span> thyroid_workflow <span class="sc">|&gt;</span></span>
<span id="cb82-6"><a href="#cb82-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_model</span>(logistic_model)</span>
<span id="cb82-7"><a href="#cb82-7" aria-hidden="true" tabindex="-1"></a><span class="do">## fit the the workflow to the training data</span></span>
<span id="cb82-8"><a href="#cb82-8" aria-hidden="true" tabindex="-1"></a>log_thyroid_fit <span class="ot">&lt;-</span> <span class="fu">fit</span>(log_thyroid_workflow, <span class="at">data =</span> train)</span>
<span id="cb82-9"><a href="#cb82-9" aria-hidden="true" tabindex="-1"></a>log_thyroid_fit</span>
<span id="cb82-10"><a href="#cb82-10" aria-hidden="true" tabindex="-1"></a><span class="do">## to inspect the fit object</span></span>
<span id="cb82-11"><a href="#cb82-11" aria-hidden="true" tabindex="-1"></a><span class="fu">str</span>(log_thyroid_fit)</span>
<span id="cb82-12"><a href="#cb82-12" aria-hidden="true" tabindex="-1"></a><span class="do">## use fitted model to make prediction</span></span>
<span id="cb82-13"><a href="#cb82-13" aria-hidden="true" tabindex="-1"></a>logistic_model_predictions <span class="ot">&lt;-</span> <span class="fu">predict</span>(log_thyroid_fit, test) <span class="sc">|&gt;</span> <span class="fu">bind_cols</span>(test)</span>
<span id="cb82-14"><a href="#cb82-14" aria-hidden="true" tabindex="-1"></a><span class="do">## examine the processing steps</span></span>
<span id="cb82-15"><a href="#cb82-15" aria-hidden="true" tabindex="-1"></a>log_thyroid_fit <span class="sc">|&gt;</span> <span class="fu">extract_recipe</span>()</span>
<span id="cb82-16"><a href="#cb82-16" aria-hidden="true" tabindex="-1"></a><span class="do">## examine the model</span></span>
<span id="cb82-17"><a href="#cb82-17" aria-hidden="true" tabindex="-1"></a>log_thyroid_fit <span class="sc">|&gt;</span> <span class="fu">extract_fit_parsnip</span>()</span>
<span id="cb82-18"><a href="#cb82-18" aria-hidden="true" tabindex="-1"></a><span class="do">## to check if the workflow has been trained</span></span>
<span id="cb82-19"><a href="#cb82-19" aria-hidden="true" tabindex="-1"></a>log_thyroid_fit<span class="sc">$</span>trained</span>
<span id="cb82-20"><a href="#cb82-20" aria-hidden="true" tabindex="-1"></a><span class="do">## I can only see age and gender, unsure why the other variable not present.? excluded due to lack of data ?because they are simply not important ? issue with workflow set up</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<p>A total of 1150 patients had complete data for the selected predictor variables (see <strong>?@tbl-predictors</strong>). Because of the volume of missing data which if a saturated model were used would include only ~350 people with complete data across all co-variates imputed datasets were analysed instead.</p>
<section id="logistic-regression" class="level4" data-number="3.2.1">
<h4 data-number="3.2.1" class="anchored" data-anchor-id="logistic-regression"><span class="header-section-number">3.2.1</span> Logistic Regression</h4>
<div class="cell-container"><div class="cell-decorator"><pre>In [40]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb83"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb83-1"><a href="#cb83-1" aria-hidden="true" tabindex="-1"></a>train <span class="sc">|&gt;</span> <span class="fu">colnames</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] "age_at_scan"                   "gender"                       
 [3] "ethnicity"                     "incidental_nodule"            
 [5] "palpable_nodule"               "rapid_enlargement"            
 [7] "compressive_symptoms"          "hypertension"                 
 [9] "vocal_cord_paresis"            "graves_disease"               
[11] "hashimotos_thyroiditis"        "family_history_thyroid_cancer"
[13] "exposure_radiation"            "albumin"                      
[15] "tsh_value"                     "lymphocytes"                  
[17] "monocyte"                      "bta_u_classification"         
[19] "solitary_nodule"               "size_nodule_mm"               
[21] "cervical_lymphadenopathy"      "thy_classification"           
[23] "final_pathology"              </code></pre>
</div>
</div></div>
<section id="clinical-characteristics-1" class="level5" data-number="3.2.1.1">
<h5 data-number="3.2.1.1" class="anchored" data-anchor-id="clinical-characteristics-1"><span class="header-section-number">3.2.1.1</span> Clinical Characteristics</h5>
<div class="cell-container"><div class="cell-decorator"><pre>In [41]:</pre></div><div id="tbl-logistic-regression" class="cell anchored" data-tbl-caption="Logistic regression of Clinical characteristics between benign and malignant thyrioid nodules.">
<div class="sourceCode cell-code" id="cb85"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb85-1"><a href="#cb85-1" aria-hidden="true" tabindex="-1"></a>glm_clin <span class="ot">&lt;-</span> <span class="fu">glm</span>(final_pathology <span class="sc">~</span> age_at_scan <span class="sc">+</span> gender <span class="sc">+</span> incidental_nodule <span class="sc">+</span> palpable_nodule <span class="sc">+</span></span>
<span id="cb85-2"><a href="#cb85-2" aria-hidden="true" tabindex="-1"></a>rapid_enlargement <span class="sc">+</span> compressive_symptoms <span class="sc">+</span> hashimotos_thyroiditis <span class="sc">+</span> family_history_thyroid_cancer <span class="sc">+</span> exposure_radiation <span class="sc">+</span></span>
<span id="cb85-3"><a href="#cb85-3" aria-hidden="true" tabindex="-1"></a>tsh_value <span class="sc">+</span> size_nodule_mm <span class="sc">+</span> solitary_nodule <span class="sc">+</span> cervical_lymphadenopathy,</span>
<span id="cb85-4"><a href="#cb85-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">data =</span> train,</span>
<span id="cb85-5"><a href="#cb85-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">family =</span> <span class="fu">binomial</span>(<span class="at">link =</span> <span class="st">"logit"</span>))</span>
<span id="cb85-6"><a href="#cb85-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-7"><a href="#cb85-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Use the packages to summarise the output</span></span>
<span id="cb85-8"><a href="#cb85-8" aria-hidden="true" tabindex="-1"></a>gtsummary<span class="sc">::</span><span class="fu">tbl_regression</span>(glm_clin,</span>
<span id="cb85-9"><a href="#cb85-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">exponentiate =</span> <span class="cn">TRUE</span>,</span>
<span id="cb85-10"><a href="#cb85-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">show_single_row =</span> <span class="fu">c</span>(gender,</span>
<span id="cb85-11"><a href="#cb85-11" aria-hidden="true" tabindex="-1"></a>                        incidental_nodule,</span>
<span id="cb85-12"><a href="#cb85-12" aria-hidden="true" tabindex="-1"></a>                        palpable_nodule,</span>
<span id="cb85-13"><a href="#cb85-13" aria-hidden="true" tabindex="-1"></a>                        rapid_enlargement,</span>
<span id="cb85-14"><a href="#cb85-14" aria-hidden="true" tabindex="-1"></a>                        compressive_symptoms,</span>
<span id="cb85-15"><a href="#cb85-15" aria-hidden="true" tabindex="-1"></a>                        hashimotos_thyroiditis,</span>
<span id="cb85-16"><a href="#cb85-16" aria-hidden="true" tabindex="-1"></a>                        family_history_thyroid_cancer,</span>
<span id="cb85-17"><a href="#cb85-17" aria-hidden="true" tabindex="-1"></a>                        exposure_radiation,</span>
<span id="cb85-18"><a href="#cb85-18" aria-hidden="true" tabindex="-1"></a>                        solitary_nodule,</span>
<span id="cb85-19"><a href="#cb85-19" aria-hidden="true" tabindex="-1"></a>                        cervical_lymphadenopathy))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred</code></pre>
</div>
<div class="cell-output-display">
<div id="lsambifgaq" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#lsambifgaq table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#lsambifgaq thead, #lsambifgaq tbody, #lsambifgaq tfoot, #lsambifgaq tr, #lsambifgaq td, #lsambifgaq th {
  border-style: none;
}

#lsambifgaq p {
  margin: 0;
  padding: 0;
}

#lsambifgaq .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#lsambifgaq .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#lsambifgaq .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#lsambifgaq .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#lsambifgaq .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#lsambifgaq .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#lsambifgaq .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#lsambifgaq .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#lsambifgaq .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#lsambifgaq .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#lsambifgaq .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#lsambifgaq .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#lsambifgaq .gt_spanner_row {
  border-bottom-style: hidden;
}

#lsambifgaq .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#lsambifgaq .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#lsambifgaq .gt_from_md > :first-child {
  margin-top: 0;
}

#lsambifgaq .gt_from_md > :last-child {
  margin-bottom: 0;
}

#lsambifgaq .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#lsambifgaq .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#lsambifgaq .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#lsambifgaq .gt_row_group_first td {
  border-top-width: 2px;
}

#lsambifgaq .gt_row_group_first th {
  border-top-width: 2px;
}

#lsambifgaq .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#lsambifgaq .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#lsambifgaq .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#lsambifgaq .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#lsambifgaq .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#lsambifgaq .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#lsambifgaq .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#lsambifgaq .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#lsambifgaq .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#lsambifgaq .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#lsambifgaq .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#lsambifgaq .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#lsambifgaq .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#lsambifgaq .gt_left {
  text-align: left;
}

#lsambifgaq .gt_center {
  text-align: center;
}

#lsambifgaq .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#lsambifgaq .gt_font_normal {
  font-weight: normal;
}

#lsambifgaq .gt_font_bold {
  font-weight: bold;
}

#lsambifgaq .gt_font_italic {
  font-style: italic;
}

#lsambifgaq .gt_super {
  font-size: 65%;
}

#lsambifgaq .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#lsambifgaq .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#lsambifgaq .gt_indent_1 {
  text-indent: 5px;
}

#lsambifgaq .gt_indent_2 {
  text-indent: 10px;
}

#lsambifgaq .gt_indent_3 {
  text-indent: 15px;
}

#lsambifgaq .gt_indent_4 {
  text-indent: 20px;
}

#lsambifgaq .gt_indent_5 {
  text-indent: 25px;
}

#lsambifgaq .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#lsambifgaq div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPUioq&quot;><div class='gt_from_md'><p><strong>OR</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>OR</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio5NSUgQ0kqKg==&quot;><div class='gt_from_md'><p><strong>95% CI</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>95% CI</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KipwLXZhbHVlKio=&quot;><div class='gt_from_md'><p><strong>p-value</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>p-value</strong></p></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="estimate">0.98</td>
<td class="gt_row gt_center" headers="ci">0.96, 0.99</td>
<td class="gt_row gt_center" headers="p.value">0.005</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="estimate">1.97</td>
<td class="gt_row gt_center" headers="ci">1.11, 3.43</td>
<td class="gt_row gt_center" headers="p.value">0.019</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">incidental_nodule</td>
<td class="gt_row gt_center" headers="estimate">0.99</td>
<td class="gt_row gt_center" headers="ci">0.48, 2.02</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">palpable_nodule</td>
<td class="gt_row gt_center" headers="estimate">2.76</td>
<td class="gt_row gt_center" headers="ci">1.29, 6.08</td>
<td class="gt_row gt_center" headers="p.value">0.010</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">rapid_enlargement</td>
<td class="gt_row gt_center" headers="estimate">1.66</td>
<td class="gt_row gt_center" headers="ci">0.46, 5.37</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">compressive_symptoms</td>
<td class="gt_row gt_center" headers="estimate">0.98</td>
<td class="gt_row gt_center" headers="ci">0.43, 2.09</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">hashimotos_thyroiditis</td>
<td class="gt_row gt_center" headers="estimate">1.19</td>
<td class="gt_row gt_center" headers="ci">0.02, 13.0</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">family_history_thyroid_cancer</td>
<td class="gt_row gt_center" headers="estimate">3.80</td>
<td class="gt_row gt_center" headers="ci">0.48, 21.7</td>
<td class="gt_row gt_center" headers="p.value">0.15</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">exposure_radiation</td>
<td class="gt_row gt_center" headers="estimate">0.00</td>
<td class="gt_row gt_center" headers="ci"><br>
</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">TSH value</td>
<td class="gt_row gt_center" headers="estimate">1.00</td>
<td class="gt_row gt_center" headers="ci">0.94, 1.05</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Nodule size (mm)</td>
<td class="gt_row gt_center" headers="estimate">1.03</td>
<td class="gt_row gt_center" headers="ci">1.01, 1.05</td>
<td class="gt_row gt_center" headers="p.value">&lt;0.001</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">solitary_nodule</td>
<td class="gt_row gt_center" headers="estimate">1.48</td>
<td class="gt_row gt_center" headers="ci">0.85, 2.51</td>
<td class="gt_row gt_center" headers="p.value">0.2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">cervical_lymphadenopathy</td>
<td class="gt_row gt_center" headers="estimate">6.74</td>
<td class="gt_row gt_center" headers="ci">2.37, 18.8</td>
<td class="gt_row gt_center" headers="p.value">&lt;0.001</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="even">
<td colspan="4" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>OR = Odds Ratio, CI = Confidence Interval</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</section>
<section id="biomarkers-2" class="level5" data-number="3.2.1.2">
<h5 data-number="3.2.1.2" class="anchored" data-anchor-id="biomarkers-2"><span class="header-section-number">3.2.1.2</span> Biomarkers</h5>
<div class="cell-container"><div class="cell-decorator"><pre>In [42]:</pre></div><div id="tbl-logistic-regression-biochem" class="cell anchored" data-tbl-caption="Logistic regression of Biochemical characteristics between benign and malignant thyrioid nodules.">
<div class="sourceCode cell-code" id="cb87"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb87-1"><a href="#cb87-1" aria-hidden="true" tabindex="-1"></a>glm_biochem <span class="ot">&lt;-</span> <span class="fu">glm</span>(final_pathology <span class="sc">~</span> age_at_scan <span class="sc">+</span> gender <span class="sc">+</span> tsh_value <span class="sc">+</span> albumin <span class="sc">+</span> lymphocytes <span class="sc">+</span> monocyte,</span>
<span id="cb87-2"><a href="#cb87-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">data =</span> train,</span>
<span id="cb87-3"><a href="#cb87-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">family =</span> <span class="fu">binomial</span>(<span class="at">link =</span> <span class="st">"logit"</span>))</span>
<span id="cb87-4"><a href="#cb87-4" aria-hidden="true" tabindex="-1"></a>gtsummary<span class="sc">::</span><span class="fu">tbl_regression</span>(glm_biochem,</span>
<span id="cb87-5"><a href="#cb87-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">exponentiate =</span> <span class="cn">TRUE</span>,</span>
<span id="cb87-6"><a href="#cb87-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">show_single_row =</span> <span class="fu">c</span>(gender))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="kitpurtzvv" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#kitpurtzvv table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#kitpurtzvv thead, #kitpurtzvv tbody, #kitpurtzvv tfoot, #kitpurtzvv tr, #kitpurtzvv td, #kitpurtzvv th {
  border-style: none;
}

#kitpurtzvv p {
  margin: 0;
  padding: 0;
}

#kitpurtzvv .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#kitpurtzvv .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#kitpurtzvv .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#kitpurtzvv .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#kitpurtzvv .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#kitpurtzvv .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#kitpurtzvv .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#kitpurtzvv .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#kitpurtzvv .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#kitpurtzvv .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#kitpurtzvv .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#kitpurtzvv .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#kitpurtzvv .gt_spanner_row {
  border-bottom-style: hidden;
}

#kitpurtzvv .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#kitpurtzvv .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#kitpurtzvv .gt_from_md > :first-child {
  margin-top: 0;
}

#kitpurtzvv .gt_from_md > :last-child {
  margin-bottom: 0;
}

#kitpurtzvv .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#kitpurtzvv .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#kitpurtzvv .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#kitpurtzvv .gt_row_group_first td {
  border-top-width: 2px;
}

#kitpurtzvv .gt_row_group_first th {
  border-top-width: 2px;
}

#kitpurtzvv .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#kitpurtzvv .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#kitpurtzvv .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#kitpurtzvv .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#kitpurtzvv .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#kitpurtzvv .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#kitpurtzvv .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#kitpurtzvv .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#kitpurtzvv .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#kitpurtzvv .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#kitpurtzvv .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#kitpurtzvv .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#kitpurtzvv .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#kitpurtzvv .gt_left {
  text-align: left;
}

#kitpurtzvv .gt_center {
  text-align: center;
}

#kitpurtzvv .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#kitpurtzvv .gt_font_normal {
  font-weight: normal;
}

#kitpurtzvv .gt_font_bold {
  font-weight: bold;
}

#kitpurtzvv .gt_font_italic {
  font-style: italic;
}

#kitpurtzvv .gt_super {
  font-size: 65%;
}

#kitpurtzvv .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#kitpurtzvv .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#kitpurtzvv .gt_indent_1 {
  text-indent: 5px;
}

#kitpurtzvv .gt_indent_2 {
  text-indent: 10px;
}

#kitpurtzvv .gt_indent_3 {
  text-indent: 15px;
}

#kitpurtzvv .gt_indent_4 {
  text-indent: 20px;
}

#kitpurtzvv .gt_indent_5 {
  text-indent: 25px;
}

#kitpurtzvv .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#kitpurtzvv div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPUioq&quot;><div class='gt_from_md'><p><strong>OR</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>OR</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio5NSUgQ0kqKg==&quot;><div class='gt_from_md'><p><strong>95% CI</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>95% CI</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KipwLXZhbHVlKio=&quot;><div class='gt_from_md'><p><strong>p-value</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>p-value</strong></p></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="estimate">0.98</td>
<td class="gt_row gt_center" headers="ci">0.97, 1.00</td>
<td class="gt_row gt_center" headers="p.value">0.010</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="estimate">1.95</td>
<td class="gt_row gt_center" headers="ci">1.15, 3.26</td>
<td class="gt_row gt_center" headers="p.value">0.012</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">TSH value</td>
<td class="gt_row gt_center" headers="estimate">1.01</td>
<td class="gt_row gt_center" headers="ci">0.96, 1.06</td>
<td class="gt_row gt_center" headers="p.value">0.5</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Albumin</td>
<td class="gt_row gt_center" headers="estimate">1.03</td>
<td class="gt_row gt_center" headers="ci">0.96, 1.12</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Lymphocytes</td>
<td class="gt_row gt_center" headers="estimate">1.04</td>
<td class="gt_row gt_center" headers="ci">0.74, 1.43</td>
<td class="gt_row gt_center" headers="p.value">0.8</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Monocytes</td>
<td class="gt_row gt_center" headers="estimate">0.18</td>
<td class="gt_row gt_center" headers="ci">0.04, 0.72</td>
<td class="gt_row gt_center" headers="p.value">0.019</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="odd">
<td colspan="4" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>OR = Odds Ratio, CI = Confidence Interval</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</section>
<section id="ultrasound-1-1" class="level5" data-number="3.2.1.3">
<h5 data-number="3.2.1.3" class="anchored" data-anchor-id="ultrasound-1-1"><span class="header-section-number">3.2.1.3</span> Ultrasound 1</h5>
<div class="cell-container"><div class="cell-decorator"><pre>In [43]:</pre></div><div id="tbl-logistic-regression-ultrasound1" class="cell anchored" data-tbl-caption="Logistic regression of Ultrasound characteristics between benign and malignant thyrioid nodules.">
<div class="sourceCode cell-code" id="cb88"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb88-1"><a href="#cb88-1" aria-hidden="true" tabindex="-1"></a>glm_ultrasound <span class="ot">&lt;-</span> <span class="fu">glm</span>(final_pathology <span class="sc">~</span> age_at_scan <span class="sc">+</span> gender <span class="sc">+</span> bta_u_classification <span class="sc">+</span> thy_classification,</span>
<span id="cb88-2"><a href="#cb88-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">data =</span> train,</span>
<span id="cb88-3"><a href="#cb88-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">family =</span> <span class="fu">binomial</span>(<span class="at">link =</span> <span class="st">"logit"</span>))</span>
<span id="cb88-4"><a href="#cb88-4" aria-hidden="true" tabindex="-1"></a>gtsummary<span class="sc">::</span><span class="fu">tbl_regression</span>(glm_ultrasound,</span>
<span id="cb88-5"><a href="#cb88-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">exponentiate =</span> <span class="cn">TRUE</span>,</span>
<span id="cb88-6"><a href="#cb88-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">show_single_row =</span> <span class="fu">c</span>(gender))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: glm.fit: algorithm did not converge</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm):
collapsing to unique 'x' values
Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm):
collapsing to unique 'x' values</code></pre>
</div>
<div class="cell-output-display">
<div id="vwzfzlbeqh" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#vwzfzlbeqh table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#vwzfzlbeqh thead, #vwzfzlbeqh tbody, #vwzfzlbeqh tfoot, #vwzfzlbeqh tr, #vwzfzlbeqh td, #vwzfzlbeqh th {
  border-style: none;
}

#vwzfzlbeqh p {
  margin: 0;
  padding: 0;
}

#vwzfzlbeqh .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#vwzfzlbeqh .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#vwzfzlbeqh .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#vwzfzlbeqh .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#vwzfzlbeqh .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#vwzfzlbeqh .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vwzfzlbeqh .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#vwzfzlbeqh .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#vwzfzlbeqh .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#vwzfzlbeqh .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#vwzfzlbeqh .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#vwzfzlbeqh .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#vwzfzlbeqh .gt_spanner_row {
  border-bottom-style: hidden;
}

#vwzfzlbeqh .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#vwzfzlbeqh .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#vwzfzlbeqh .gt_from_md > :first-child {
  margin-top: 0;
}

#vwzfzlbeqh .gt_from_md > :last-child {
  margin-bottom: 0;
}

#vwzfzlbeqh .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#vwzfzlbeqh .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#vwzfzlbeqh .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#vwzfzlbeqh .gt_row_group_first td {
  border-top-width: 2px;
}

#vwzfzlbeqh .gt_row_group_first th {
  border-top-width: 2px;
}

#vwzfzlbeqh .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#vwzfzlbeqh .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#vwzfzlbeqh .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#vwzfzlbeqh .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vwzfzlbeqh .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#vwzfzlbeqh .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#vwzfzlbeqh .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#vwzfzlbeqh .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#vwzfzlbeqh .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vwzfzlbeqh .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#vwzfzlbeqh .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#vwzfzlbeqh .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#vwzfzlbeqh .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#vwzfzlbeqh .gt_left {
  text-align: left;
}

#vwzfzlbeqh .gt_center {
  text-align: center;
}

#vwzfzlbeqh .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#vwzfzlbeqh .gt_font_normal {
  font-weight: normal;
}

#vwzfzlbeqh .gt_font_bold {
  font-weight: bold;
}

#vwzfzlbeqh .gt_font_italic {
  font-style: italic;
}

#vwzfzlbeqh .gt_super {
  font-size: 65%;
}

#vwzfzlbeqh .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#vwzfzlbeqh .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#vwzfzlbeqh .gt_indent_1 {
  text-indent: 5px;
}

#vwzfzlbeqh .gt_indent_2 {
  text-indent: 10px;
}

#vwzfzlbeqh .gt_indent_3 {
  text-indent: 15px;
}

#vwzfzlbeqh .gt_indent_4 {
  text-indent: 20px;
}

#vwzfzlbeqh .gt_indent_5 {
  text-indent: 25px;
}

#vwzfzlbeqh .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#vwzfzlbeqh div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPUioq&quot;><div class='gt_from_md'><p><strong>OR</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>OR</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio5NSUgQ0kqKg==&quot;><div class='gt_from_md'><p><strong>95% CI</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>95% CI</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KipwLXZhbHVlKio=&quot;><div class='gt_from_md'><p><strong>p-value</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>p-value</strong></p></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="estimate">0.99</td>
<td class="gt_row gt_center" headers="ci">0.97, 1.01</td>
<td class="gt_row gt_center" headers="p.value">0.2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="estimate">1.23</td>
<td class="gt_row gt_center" headers="ci">0.63, 2.32</td>
<td class="gt_row gt_center" headers="p.value">0.5</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">bta_u_classification</td>
<td class="gt_row gt_center" headers="estimate"><br>
</td>
<td class="gt_row gt_center" headers="ci"><br>
</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U1</td>
<td class="gt_row gt_center" headers="estimate">—</td>
<td class="gt_row gt_center" headers="ci">—</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U2</td>
<td class="gt_row gt_center" headers="estimate">461,584</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U3</td>
<td class="gt_row gt_center" headers="estimate">7,642,048</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U4</td>
<td class="gt_row gt_center" headers="estimate">36,642,711</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U5</td>
<td class="gt_row gt_center" headers="estimate">126,534,639</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">thy_classification</td>
<td class="gt_row gt_center" headers="estimate"><br>
</td>
<td class="gt_row gt_center" headers="ci"><br>
</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1</td>
<td class="gt_row gt_center" headers="estimate">—</td>
<td class="gt_row gt_center" headers="ci">—</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1c</td>
<td class="gt_row gt_center" headers="estimate">1.63</td>
<td class="gt_row gt_center" headers="ci">0.19, 9.45</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2</td>
<td class="gt_row gt_center" headers="estimate">0.36</td>
<td class="gt_row gt_center" headers="ci">0.09, 1.25</td>
<td class="gt_row gt_center" headers="p.value">0.11</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2c</td>
<td class="gt_row gt_center" headers="estimate">0.00</td>
<td class="gt_row gt_center" headers="ci">0.00, 0.00</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3a</td>
<td class="gt_row gt_center" headers="estimate">2.24</td>
<td class="gt_row gt_center" headers="ci">0.66, 7.80</td>
<td class="gt_row gt_center" headers="p.value">0.2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3f</td>
<td class="gt_row gt_center" headers="estimate">3.01</td>
<td class="gt_row gt_center" headers="ci">1.21, 8.47</td>
<td class="gt_row gt_center" headers="p.value">0.025</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy4</td>
<td class="gt_row gt_center" headers="estimate">7.06</td>
<td class="gt_row gt_center" headers="ci">1.93, 26.6</td>
<td class="gt_row gt_center" headers="p.value">0.003</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy5</td>
<td class="gt_row gt_center" headers="estimate">9.52</td>
<td class="gt_row gt_center" headers="ci">3.19, 31.1</td>
<td class="gt_row gt_center" headers="p.value">&lt;0.001</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="even">
<td colspan="4" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>OR = Odds Ratio, CI = Confidence Interval</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</section>
<section id="ultrasound-2" class="level5" data-number="3.2.1.4">
<h5 data-number="3.2.1.4" class="anchored" data-anchor-id="ultrasound-2"><span class="header-section-number">3.2.1.4</span> Ultrasound 2</h5>
<div class="cell-container"><div class="cell-decorator"><pre>In [44]:</pre></div><div class="cell" data-tbl-caption="Expanded logistic regression of Ultrasound characteristics between benign and malignant thyroid noduules.">
<div class="sourceCode cell-code" id="cb93"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb93-1"><a href="#cb93-1" aria-hidden="true" tabindex="-1"></a>glm_ultrasound <span class="ot">&lt;-</span> <span class="fu">glm</span>(final_pathology <span class="sc">~</span> age_at_scan <span class="sc">+</span> gender <span class="sc">+</span> incidental_nodule <span class="sc">+</span> tsh_value <span class="sc">+</span></span>
<span id="cb93-2"><a href="#cb93-2" aria-hidden="true" tabindex="-1"></a>size_nodule_mm <span class="sc">+</span> solitary_nodule <span class="sc">+</span> cervical_lymphadenopathy <span class="sc">+</span> bta_u_classification <span class="sc">+</span> thy_classification,</span>
<span id="cb93-3"><a href="#cb93-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">data =</span> train,</span>
<span id="cb93-4"><a href="#cb93-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">family =</span> <span class="fu">binomial</span>(<span class="at">link =</span> <span class="st">"logit"</span>))</span>
<span id="cb93-5"><a href="#cb93-5" aria-hidden="true" tabindex="-1"></a>gtsummary<span class="sc">::</span><span class="fu">tbl_regression</span>(glm_ultrasound,</span>
<span id="cb93-6"><a href="#cb93-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">exponentiate =</span> <span class="cn">TRUE</span>,</span>
<span id="cb93-7"><a href="#cb93-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">show_single_row =</span> <span class="fu">c</span>(</span>
<span id="cb93-8"><a href="#cb93-8" aria-hidden="true" tabindex="-1"></a>        gender,</span>
<span id="cb93-9"><a href="#cb93-9" aria-hidden="true" tabindex="-1"></a>        incidental_nodule,</span>
<span id="cb93-10"><a href="#cb93-10" aria-hidden="true" tabindex="-1"></a>        solitary_nodule,</span>
<span id="cb93-11"><a href="#cb93-11" aria-hidden="true" tabindex="-1"></a>        cervical_lymphadenopathy</span>
<span id="cb93-12"><a href="#cb93-12" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb93-13"><a href="#cb93-13" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm):
collapsing to unique 'x' values</code></pre>
</div>
<div class="cell-output-display">
<div id="zfgqzyepqs" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#zfgqzyepqs table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#zfgqzyepqs thead, #zfgqzyepqs tbody, #zfgqzyepqs tfoot, #zfgqzyepqs tr, #zfgqzyepqs td, #zfgqzyepqs th {
  border-style: none;
}

#zfgqzyepqs p {
  margin: 0;
  padding: 0;
}

#zfgqzyepqs .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#zfgqzyepqs .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#zfgqzyepqs .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#zfgqzyepqs .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#zfgqzyepqs .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#zfgqzyepqs .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zfgqzyepqs .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#zfgqzyepqs .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#zfgqzyepqs .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#zfgqzyepqs .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#zfgqzyepqs .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#zfgqzyepqs .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#zfgqzyepqs .gt_spanner_row {
  border-bottom-style: hidden;
}

#zfgqzyepqs .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#zfgqzyepqs .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#zfgqzyepqs .gt_from_md > :first-child {
  margin-top: 0;
}

#zfgqzyepqs .gt_from_md > :last-child {
  margin-bottom: 0;
}

#zfgqzyepqs .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#zfgqzyepqs .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#zfgqzyepqs .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#zfgqzyepqs .gt_row_group_first td {
  border-top-width: 2px;
}

#zfgqzyepqs .gt_row_group_first th {
  border-top-width: 2px;
}

#zfgqzyepqs .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#zfgqzyepqs .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#zfgqzyepqs .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#zfgqzyepqs .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zfgqzyepqs .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#zfgqzyepqs .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#zfgqzyepqs .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#zfgqzyepqs .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#zfgqzyepqs .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zfgqzyepqs .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#zfgqzyepqs .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#zfgqzyepqs .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#zfgqzyepqs .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#zfgqzyepqs .gt_left {
  text-align: left;
}

#zfgqzyepqs .gt_center {
  text-align: center;
}

#zfgqzyepqs .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#zfgqzyepqs .gt_font_normal {
  font-weight: normal;
}

#zfgqzyepqs .gt_font_bold {
  font-weight: bold;
}

#zfgqzyepqs .gt_font_italic {
  font-style: italic;
}

#zfgqzyepqs .gt_super {
  font-size: 65%;
}

#zfgqzyepqs .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#zfgqzyepqs .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#zfgqzyepqs .gt_indent_1 {
  text-indent: 5px;
}

#zfgqzyepqs .gt_indent_2 {
  text-indent: 10px;
}

#zfgqzyepqs .gt_indent_3 {
  text-indent: 15px;
}

#zfgqzyepqs .gt_indent_4 {
  text-indent: 20px;
}

#zfgqzyepqs .gt_indent_5 {
  text-indent: 25px;
}

#zfgqzyepqs .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#zfgqzyepqs div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th id="<div data-qmd-base64=&quot;KipDaGFyYWN0ZXJpc3RpYyoq&quot;><div class='gt_from_md'><p><strong>Characteristic</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><p><strong>Characteristic</strong></p></th>
<th id="<div data-qmd-base64=&quot;KipPUioq&quot;><div class='gt_from_md'><p><strong>OR</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>OR</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;Kio5NSUgQ0kqKg==&quot;><div class='gt_from_md'><p><strong>95% CI</strong></p>
</div></div><span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;><sup>1</sup></span>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>95% CI</strong></p>
<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
<th id="<div data-qmd-base64=&quot;KipwLXZhbHVlKio=&quot;><div class='gt_from_md'><p><strong>p-value</strong></p>
</div></div>" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><p><strong>p-value</strong></p></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="estimate">0.99</td>
<td class="gt_row gt_center" headers="ci">0.97, 1.01</td>
<td class="gt_row gt_center" headers="p.value">0.2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">gender</td>
<td class="gt_row gt_center" headers="estimate">1.19</td>
<td class="gt_row gt_center" headers="ci">0.58, 2.34</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">incidental_nodule</td>
<td class="gt_row gt_center" headers="estimate">0.94</td>
<td class="gt_row gt_center" headers="ci">0.48, 1.84</td>
<td class="gt_row gt_center" headers="p.value">0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">TSH value</td>
<td class="gt_row gt_center" headers="estimate">0.95</td>
<td class="gt_row gt_center" headers="ci">0.89, 1.00</td>
<td class="gt_row gt_center" headers="p.value">0.091</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Nodule size (mm)</td>
<td class="gt_row gt_center" headers="estimate">1.04</td>
<td class="gt_row gt_center" headers="ci">1.02, 1.06</td>
<td class="gt_row gt_center" headers="p.value">&lt;0.001</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">solitary_nodule</td>
<td class="gt_row gt_center" headers="estimate">1.49</td>
<td class="gt_row gt_center" headers="ci">0.77, 2.82</td>
<td class="gt_row gt_center" headers="p.value">0.2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">cervical_lymphadenopathy</td>
<td class="gt_row gt_center" headers="estimate">6.61</td>
<td class="gt_row gt_center" headers="ci">1.45, 29.3</td>
<td class="gt_row gt_center" headers="p.value">0.013</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">bta_u_classification</td>
<td class="gt_row gt_center" headers="estimate"><br>
</td>
<td class="gt_row gt_center" headers="ci"><br>
</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U1</td>
<td class="gt_row gt_center" headers="estimate">—</td>
<td class="gt_row gt_center" headers="ci">—</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U2</td>
<td class="gt_row gt_center" headers="estimate">239,169</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U3</td>
<td class="gt_row gt_center" headers="estimate">3,105,400</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U4</td>
<td class="gt_row gt_center" headers="estimate">14,770,696</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;U5</td>
<td class="gt_row gt_center" headers="estimate">39,781,959</td>
<td class="gt_row gt_center" headers="ci">0.00, NA</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">thy_classification</td>
<td class="gt_row gt_center" headers="estimate"><br>
</td>
<td class="gt_row gt_center" headers="ci"><br>
</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1</td>
<td class="gt_row gt_center" headers="estimate">—</td>
<td class="gt_row gt_center" headers="ci">—</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy1c</td>
<td class="gt_row gt_center" headers="estimate">0.97</td>
<td class="gt_row gt_center" headers="ci">0.06, 7.66</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2</td>
<td class="gt_row gt_center" headers="estimate">0.25</td>
<td class="gt_row gt_center" headers="ci">0.06, 0.97</td>
<td class="gt_row gt_center" headers="p.value">0.049</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy2c</td>
<td class="gt_row gt_center" headers="estimate">0.00</td>
<td class="gt_row gt_center" headers="ci">0.00, 1,045,453</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3a</td>
<td class="gt_row gt_center" headers="estimate">2.69</td>
<td class="gt_row gt_center" headers="ci">0.76, 9.91</td>
<td class="gt_row gt_center" headers="p.value">0.13</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy3f</td>
<td class="gt_row gt_center" headers="estimate">2.45</td>
<td class="gt_row gt_center" headers="ci">0.92, 7.39</td>
<td class="gt_row gt_center" headers="p.value">0.089</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy4</td>
<td class="gt_row gt_center" headers="estimate">8.34</td>
<td class="gt_row gt_center" headers="ci">2.18, 33.3</td>
<td class="gt_row gt_center" headers="p.value">0.002</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Thy5</td>
<td class="gt_row gt_center" headers="estimate">11.1</td>
<td class="gt_row gt_center" headers="ci">3.47, 39.6</td>
<td class="gt_row gt_center" headers="p.value">&lt;0.001</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="odd">
<td colspan="4" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span>
<p>OR = Odds Ratio, CI = Confidence Interval</p></td>
</tr>
</tfoot>

</table>

</div>
</div>
</div></div>
</section>
</section>
<section id="lasso" class="level4" data-number="3.2.2">
<h4 data-number="3.2.2" class="anchored" data-anchor-id="lasso"><span class="header-section-number">3.2.2</span> LASSO</h4>
<div class="cell-container"><div class="cell-decorator"><pre>In [45]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb96"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb96-1"><a href="#cb96-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Specify the LASSO model using parsnip, the key here is the use of the glmnet engine which is the R package for</span></span>
<span id="cb96-2"><a href="#cb96-2" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting LASSO regression. Technically the package fits Elastic Net but with a mixture value of 1 it is equivalent to</span></span>
<span id="cb96-3"><a href="#cb96-3" aria-hidden="true" tabindex="-1"></a><span class="do">## a plain LASSO (mixture value of 0 is equivalent to Ridge Regression in an Elastic Net)</span></span>
<span id="cb96-4"><a href="#cb96-4" aria-hidden="true" tabindex="-1"></a>tune_spec_lasso <span class="ot">&lt;-</span> parsnip<span class="sc">::</span><span class="fu">logistic_reg</span>(<span class="at">penalty =</span> hardhat<span class="sc">::</span><span class="fu">tune</span>(), <span class="at">mixture =</span> <span class="dv">1</span>) <span class="sc">|&gt;</span></span>
<span id="cb96-5"><a href="#cb96-5" aria-hidden="true" tabindex="-1"></a>  parsnip<span class="sc">::</span><span class="fu">set_engine</span>(<span class="st">"glmnet"</span>)</span>
<span id="cb96-6"><a href="#cb96-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb96-7"><a href="#cb96-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Tune the LASSO parameters via cross-validation</span></span>
<span id="cb96-8"><a href="#cb96-8" aria-hidden="true" tabindex="-1"></a>lasso_grid <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">tune_grid</span>(</span>
<span id="cb96-9"><a href="#cb96-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">object =</span> workflows<span class="sc">::</span><span class="fu">add_model</span>(thyroid_workflow, tune_spec_lasso),</span>
<span id="cb96-10"><a href="#cb96-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">resamples =</span> cv_folds,</span>
<span id="cb96-11"><a href="#cb96-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">grid =</span> dials<span class="sc">::</span><span class="fu">grid_regular</span>(<span class="fu">penalty</span>(), <span class="at">levels =</span> <span class="dv">50</span>)</span>
<span id="cb96-12"><a href="#cb96-12" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [46]:</pre></div><div id="cell-fig-lasso-tune-autoplot" class="cell">
<div class="sourceCode cell-code" id="cb97"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb97-1"><a href="#cb97-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Plot the tuning search results, see https://tune.tidymodels.org/reference/autoplot.tune_results.html</span></span>
<span id="cb97-2"><a href="#cb97-2" aria-hidden="true" tabindex="-1"></a><span class="do">##</span></span>
<span id="cb97-3"><a href="#cb97-3" aria-hidden="true" tabindex="-1"></a><span class="do">## This shows how the evaluation metrics change over time with each iteration of the Lasso (we know we are running a</span></span>
<span id="cb97-4"><a href="#cb97-4" aria-hidden="true" tabindex="-1"></a><span class="do">## LASSO because when we setup tune_spec_lasso the mixture = 1 when we define parsnip::logistic_reg()</span></span>
<span id="cb97-5"><a href="#cb97-5" aria-hidden="true" tabindex="-1"></a>tune<span class="sc">::</span><span class="fu">autoplot</span>(lasso_grid)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-lasso-tune-autoplot" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-lasso-tune-autoplot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-lasso-tune-autoplot-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-24" title="Figure&nbsp;24: Autoplot of LASSO grid search"><img src="index_files/figure-html/fig-lasso-tune-autoplot-1.png" class="img-fluid figure-img" width="672"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-lasso-tune-autoplot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;24: Autoplot of LASSO grid search
</figcaption>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [47]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb98"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb98-1"><a href="#cb98-1" aria-hidden="true" tabindex="-1"></a>lasso_kfold_roc_auc <span class="ot">&lt;-</span> lasso_grid <span class="sc">|&gt;</span></span>
<span id="cb98-2"><a href="#cb98-2" aria-hidden="true" tabindex="-1"></a>  tune<span class="sc">::</span><span class="fu">select_best</span>(<span class="at">metric =</span> <span class="st">"roc_auc"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [48]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb99"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb99-1"><a href="#cb99-1" aria-hidden="true" tabindex="-1"></a>final_lasso_kfold <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">finalize_workflow</span>(</span>
<span id="cb99-2"><a href="#cb99-2" aria-hidden="true" tabindex="-1"></a>  workflows<span class="sc">::</span><span class="fu">add_model</span>(thyroid_workflow, tune_spec_lasso),</span>
<span id="cb99-3"><a href="#cb99-3" aria-hidden="true" tabindex="-1"></a>  lasso_kfold_roc_auc)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [49]:</pre></div><div id="cell-fig-lasso-kfold-eval-importance" class="cell">
<div class="sourceCode cell-code" id="cb100"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb100-1"><a href="#cb100-1" aria-hidden="true" tabindex="-1"></a>final_lasso_kfold <span class="sc">|&gt;</span></span>
<span id="cb100-2"><a href="#cb100-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fit</span>(train) <span class="sc">|&gt;</span></span>
<span id="cb100-3"><a href="#cb100-3" aria-hidden="true" tabindex="-1"></a>  hardhat<span class="sc">::</span><span class="fu">extract_fit_parsnip</span>() <span class="sc">|&gt;</span></span>
<span id="cb100-4"><a href="#cb100-4" aria-hidden="true" tabindex="-1"></a>  vip<span class="sc">::</span><span class="fu">vi</span>(<span class="at">lambda =</span> lasso_kfold_roc_auc<span class="sc">$</span>penalty) <span class="sc">|&gt;</span></span>
<span id="cb100-5"><a href="#cb100-5" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(</span>
<span id="cb100-6"><a href="#cb100-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">Importance =</span> <span class="fu">abs</span>(Importance),</span>
<span id="cb100-7"><a href="#cb100-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">Variable =</span> <span class="fu">fct_reorder</span>(Variable, Importance)</span>
<span id="cb100-8"><a href="#cb100-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">|&gt;</span></span>
<span id="cb100-9"><a href="#cb100-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> Importance, <span class="at">y =</span> Variable, <span class="at">fill =</span> Sign)) <span class="sc">+</span></span>
<span id="cb100-10"><a href="#cb100-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-lasso-kfold-eval-importance" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-lasso-kfold-eval-importance-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-lasso-kfold-eval-importance-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-25" title="Figure&nbsp;25: Importance of variables fitted using LASSO"><img src="index_files/figure-html/fig-lasso-kfold-eval-importance-1.png" class="img-fluid figure-img" width="672"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-lasso-kfold-eval-importance-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;25: Importance of variables fitted using LASSO
</figcaption>
</figure>
</div>
</div>
</div></div>
<p><strong>NB</strong> - We may wish to inspect the coefficients at each step of tuning. A related example of how to do this can be found in the <a href="https://www.tidymodels.org/learn/">Tidymodels documentation</a> under the <a href="https://www.tidymodels.org/learn/models/coefficients/#tuning-a-glmnet-model">Tuning a <code>glmnet</code> model</a>. This would be desirable as it looks like only two features are selected as being important by this method and so rather than just accepting this I would want to investigate and see how the coefficients changed over iterations. Another useful resource is the <a href="https://glmnet.stanford.edu/articles/glmnet.html">glmnet</a> documentation, although note that since we are using the Tidymodels framework the model <code>fit</code> is wrapped up inside (hence the above article on how to extract this information).</p>
<div class="cell-container"><div class="cell-decorator"><pre>In [50]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb101"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb101-1"><a href="#cb101-1" aria-hidden="true" tabindex="-1"></a><span class="fu">save</span>(tune_spec_lasso, lasso_grid, final_lasso_kfold, lasso_kfold_roc_auc,</span>
<span id="cb101-2"><a href="#cb101-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">file =</span> <span class="st">"data/r/lasso.RData"</span></span>
<span id="cb101-3"><a href="#cb101-3" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
</section>
<section id="elastic-net" class="level4" data-number="3.2.3">
<h4 data-number="3.2.3" class="anchored" data-anchor-id="elastic-net"><span class="header-section-number">3.2.3</span> Elastic Net</h4>
<div class="cell-container"><div class="cell-decorator"><pre>In [51]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb102"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb102-1"><a href="#cb102-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Specify the Elastic Net model using parsnip, the key here is the use of the glmnet engine which is the R package for</span></span>
<span id="cb102-2"><a href="#cb102-2" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting Elastic Net regression. Technically the package fits Elastic Net but with a mixture value of 1 it is equivalent to</span></span>
<span id="cb102-3"><a href="#cb102-3" aria-hidden="true" tabindex="-1"></a><span class="do">## a plain Elastic Net (mixture value of 0 is equivalent to Ridge Regression in an Elastic Net)</span></span>
<span id="cb102-4"><a href="#cb102-4" aria-hidden="true" tabindex="-1"></a>tune_spec_elastic <span class="ot">&lt;-</span> parsnip<span class="sc">::</span><span class="fu">logistic_reg</span>(<span class="at">penalty =</span> hardhat<span class="sc">::</span><span class="fu">tune</span>(), <span class="at">mixture =</span> <span class="fl">0.5</span>) <span class="sc">|&gt;</span></span>
<span id="cb102-5"><a href="#cb102-5" aria-hidden="true" tabindex="-1"></a>  parsnip<span class="sc">::</span><span class="fu">set_engine</span>(<span class="st">"glmnet"</span>)</span>
<span id="cb102-6"><a href="#cb102-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-7"><a href="#cb102-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Tune the Elastic Net parameters via cross-validation</span></span>
<span id="cb102-8"><a href="#cb102-8" aria-hidden="true" tabindex="-1"></a>lasso_grid <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">tune_grid</span>(</span>
<span id="cb102-9"><a href="#cb102-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">object =</span> workflows<span class="sc">::</span><span class="fu">add_model</span>(thyroid_workflow, tune_spec_elastic),</span>
<span id="cb102-10"><a href="#cb102-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">resamples =</span> cv_folds,</span>
<span id="cb102-11"><a href="#cb102-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">grid =</span> dials<span class="sc">::</span><span class="fu">grid_regular</span>(<span class="fu">penalty</span>(), <span class="at">levels =</span> <span class="dv">50</span>)</span>
<span id="cb102-12"><a href="#cb102-12" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [52]:</pre></div><div id="cell-fig-elastic-tune-autoplot" class="cell">
<div class="sourceCode cell-code" id="cb103"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb103-1"><a href="#cb103-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Plot the tuning search results, see https://tune.tidymodels.org/reference/autoplot.tune_results.html</span></span>
<span id="cb103-2"><a href="#cb103-2" aria-hidden="true" tabindex="-1"></a><span class="do">##</span></span>
<span id="cb103-3"><a href="#cb103-3" aria-hidden="true" tabindex="-1"></a><span class="do">## This shows how the evaluation metrics change over time with each iteration of the Lasso (we know we are running a</span></span>
<span id="cb103-4"><a href="#cb103-4" aria-hidden="true" tabindex="-1"></a><span class="do">## Elastic Net because when we setup tune_spec_elastic the mixture = 1 when we define parsnip::logistic_reg()</span></span>
<span id="cb103-5"><a href="#cb103-5" aria-hidden="true" tabindex="-1"></a>tune<span class="sc">::</span><span class="fu">autoplot</span>(lasso_grid)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-elastic-tune-autoplot" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-elastic-tune-autoplot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-elastic-tune-autoplot-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-26" title="Figure&nbsp;26: Autoplot of Elastic Net grid search"><img src="index_files/figure-html/fig-elastic-tune-autoplot-1.png" class="img-fluid figure-img" width="672"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-elastic-tune-autoplot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;26: Autoplot of Elastic Net grid search
</figcaption>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [53]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb104"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb104-1"><a href="#cb104-1" aria-hidden="true" tabindex="-1"></a><span class="do">## K-fold best fit for Elastic Net</span></span>
<span id="cb104-2"><a href="#cb104-2" aria-hidden="true" tabindex="-1"></a>lasso_kfold_roc_auc <span class="ot">&lt;-</span> lasso_grid <span class="sc">|&gt;</span></span>
<span id="cb104-3"><a href="#cb104-3" aria-hidden="true" tabindex="-1"></a>  tune<span class="sc">::</span><span class="fu">select_best</span>(<span class="at">metric =</span> <span class="st">"roc_auc"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [54]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb105"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb105-1"><a href="#cb105-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Fit the final Elastic Net model</span></span>
<span id="cb105-2"><a href="#cb105-2" aria-hidden="true" tabindex="-1"></a>final_elastic_kfold <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">finalize_workflow</span>(</span>
<span id="cb105-3"><a href="#cb105-3" aria-hidden="true" tabindex="-1"></a>  workflows<span class="sc">::</span><span class="fu">add_model</span>(thyroid_workflow, tune_spec_elastic),</span>
<span id="cb105-4"><a href="#cb105-4" aria-hidden="true" tabindex="-1"></a>  lasso_kfold_roc_auc</span>
<span id="cb105-5"><a href="#cb105-5" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [55]:</pre></div><div id="cell-fig-elastic-kfold-eval-importance" class="cell">
<div class="sourceCode cell-code" id="cb106"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb106-1"><a href="#cb106-1" aria-hidden="true" tabindex="-1"></a>final_elastic_kfold <span class="sc">|&gt;</span></span>
<span id="cb106-2"><a href="#cb106-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fit</span>(train) <span class="sc">|&gt;</span></span>
<span id="cb106-3"><a href="#cb106-3" aria-hidden="true" tabindex="-1"></a>  hardhat<span class="sc">::</span><span class="fu">extract_fit_parsnip</span>() <span class="sc">|&gt;</span></span>
<span id="cb106-4"><a href="#cb106-4" aria-hidden="true" tabindex="-1"></a>  vip<span class="sc">::</span><span class="fu">vi</span>(<span class="at">lambda =</span> lasso_kfold_roc_auc<span class="sc">$</span>penalty) <span class="sc">|&gt;</span></span>
<span id="cb106-5"><a href="#cb106-5" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(</span>
<span id="cb106-6"><a href="#cb106-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">Importance =</span> <span class="fu">abs</span>(Importance),</span>
<span id="cb106-7"><a href="#cb106-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">Variable =</span> <span class="fu">fct_reorder</span>(Variable, Importance)</span>
<span id="cb106-8"><a href="#cb106-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">|&gt;</span></span>
<span id="cb106-9"><a href="#cb106-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> Importance, <span class="at">y =</span> Variable, <span class="at">fill =</span> Sign)) <span class="sc">+</span></span>
<span id="cb106-10"><a href="#cb106-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>() <span class="sc">+</span></span>
<span id="cb106-11"><a href="#cb106-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">dark_theme_minimal</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Inverted geom defaults of fill and color/colour.
To change them back, use invert_geom_defaults().</code></pre>
</div>
<div class="cell-output-display">
<div id="fig-elastic-kfold-eval-importance" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-elastic-kfold-eval-importance-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca" class="">
<a href="index_files/figure-html/fig-elastic-kfold-eval-importance-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-27" title="Figure&nbsp;27: Importance of variables fitted using Elastic Net"><img src="index_files/figure-html/fig-elastic-kfold-eval-importance-1.png" class="img-fluid figure-img" width="672"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-elastic-kfold-eval-importance-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;27: Importance of variables fitted using Elastic Net
</figcaption>
</figure>
</div>
</div>
</div></div>
<p><strong>NB</strong> - We may wish to inspect the coefficients at each step of tuning. A related example of how to do this can be found in the <a href="https://www.tidymodels.org/learn/">Tidymodels documentation</a> under the <a href="https://www.tidymodels.org/learn/models/coefficients/#tuning-a-glmnet-model">Tuning a <code>glmnet</code> model</a>. This would be desirable as it looks like only two features are selected as being important by this method and so rather than just accepting this I would want to investigate and see how the coefficients changed over iterations. Another useful resource is the <a href="https://glmnet.stanford.edu/articles/glmnet.html">glmnet</a> documentation, although note that since we are using the Tidymodels framework the model <code>fit</code> is wrapped up inside (hence the above article on how to extract this information).</p>
<div class="cell-container"><div class="cell-decorator"><pre>In [56]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb108"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb108-1"><a href="#cb108-1" aria-hidden="true" tabindex="-1"></a><span class="fu">save</span>(tune_spec_elastic, lasso_grid, final_elastic_kfold, lasso_kfold_roc_auc,</span>
<span id="cb108-2"><a href="#cb108-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">file =</span> <span class="st">"data/r/lasso.RData"</span></span>
<span id="cb108-3"><a href="#cb108-3" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
</section>
<section id="random-forest-1" class="level4" data-number="3.2.4">
<h4 data-number="3.2.4" class="anchored" data-anchor-id="random-forest-1"><span class="header-section-number">3.2.4</span> Random Forest</h4>
<div class="cell-container"><div class="cell-decorator"><pre>In [57]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb109"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb109-1"><a href="#cb109-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Specify the Random Forest model</span></span>
<span id="cb109-2"><a href="#cb109-2" aria-hidden="true" tabindex="-1"></a>rf_tune <span class="ot">&lt;-</span> parsnip<span class="sc">::</span><span class="fu">rand_forest</span>(</span>
<span id="cb109-3"><a href="#cb109-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">mtry =</span> <span class="fu">tune</span>(),</span>
<span id="cb109-4"><a href="#cb109-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">trees =</span> <span class="dv">100</span>,</span>
<span id="cb109-5"><a href="#cb109-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">min_n =</span> <span class="fu">tune</span>()</span>
<span id="cb109-6"><a href="#cb109-6" aria-hidden="true" tabindex="-1"></a>) <span class="sc">|&gt;</span></span>
<span id="cb109-7"><a href="#cb109-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_mode</span>(<span class="st">"classification"</span>) <span class="sc">|&gt;</span></span>
<span id="cb109-8"><a href="#cb109-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_engine</span>(<span class="st">"ranger"</span>, <span class="at">importance =</span> <span class="st">"impurity"</span>)</span>
<span id="cb109-9"><a href="#cb109-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb109-10"><a href="#cb109-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb109-11"><a href="#cb109-11" aria-hidden="true" tabindex="-1"></a><span class="do">## Tune the parameters via Cross-validation</span></span>
<span id="cb109-12"><a href="#cb109-12" aria-hidden="true" tabindex="-1"></a>rf_grid <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">tune_grid</span>(</span>
<span id="cb109-13"><a href="#cb109-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_model</span>(thyroid_workflow, rf_tune),</span>
<span id="cb109-14"><a href="#cb109-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">resamples =</span> cv_folds, <span class="do">## cv_loo,</span></span>
<span id="cb109-15"><a href="#cb109-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">grid =</span> <span class="fu">grid_regular</span>(<span class="fu">mtry</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="dv">5</span>, <span class="dv">10</span>)), <span class="co"># smaller ranges will run quicker</span></span>
<span id="cb109-16"><a href="#cb109-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">min_n</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">25</span>)),</span>
<span id="cb109-17"><a href="#cb109-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">levels =</span> <span class="dv">3</span></span>
<span id="cb109-18"><a href="#cb109-18" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb109-19"><a href="#cb109-19" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb109-20"><a href="#cb109-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb109-21"><a href="#cb109-21" aria-hidden="true" tabindex="-1"></a><span class="do">## Get the best fitting model with the highest ROC AUC</span></span>
<span id="cb109-22"><a href="#cb109-22" aria-hidden="true" tabindex="-1"></a>rf_highest_roc_auc <span class="ot">&lt;-</span> rf_grid <span class="sc">|&gt;</span></span>
<span id="cb109-23"><a href="#cb109-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select_best</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in select_best(rf_grid): No value of `metric` was given; "roc_auc" will
be used.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb111"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb111-1"><a href="#cb111-1" aria-hidden="true" tabindex="-1"></a>final_rf <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">finalize_workflow</span>(</span>
<span id="cb111-2"><a href="#cb111-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_model</span>(thyroid_workflow, rf_tune),</span>
<span id="cb111-3"><a href="#cb111-3" aria-hidden="true" tabindex="-1"></a>  rf_highest_roc_auc</span>
<span id="cb111-4"><a href="#cb111-4" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [58]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb112"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb112-1"><a href="#cb112-1" aria-hidden="true" tabindex="-1"></a>final_rf <span class="sc">|&gt;</span></span>
<span id="cb112-2"><a href="#cb112-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fit</span>(train) <span class="sc">|&gt;</span></span>
<span id="cb112-3"><a href="#cb112-3" aria-hidden="true" tabindex="-1"></a>  hardhat<span class="sc">::</span><span class="fu">extract_fit_parsnip</span>() <span class="sc">|&gt;</span></span>
<span id="cb112-4"><a href="#cb112-4" aria-hidden="true" tabindex="-1"></a>  vip<span class="sc">::</span><span class="fu">vi</span>(<span class="at">lambda =</span> final_rf<span class="sc">$</span>penalty) <span class="sc">|&gt;</span></span>
<span id="cb112-5"><a href="#cb112-5" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(</span>
<span id="cb112-6"><a href="#cb112-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">Importance =</span> <span class="fu">abs</span>(Importance),</span>
<span id="cb112-7"><a href="#cb112-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">Variable =</span> <span class="fu">fct_reorder</span>(Variable, Importance)</span>
<span id="cb112-8"><a href="#cb112-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">|&gt;</span></span>
<span id="cb112-9"><a href="#cb112-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> Importance, <span class="at">y =</span> Variable, <span class="at">fill =</span> Importance)) <span class="sc">+</span></span>
<span id="cb112-10"><a href="#cb112-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>() <span class="sc">+</span></span>
<span id="cb112-11"><a href="#cb112-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">dark_theme_minimal</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="">
<figure class="figure">
<p class=""><a href="index_files/figure-html/rf-kfold-eval-importance-1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-28"><img src="index_files/figure-html/rf-kfold-eval-importance-1.png" class="img-fluid figure-img" width="672"></a></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [59]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb113"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb113-1"><a href="#cb113-1" aria-hidden="true" tabindex="-1"></a><span class="fu">save</span>(rf_tune, rf_grid, final_rf, rf_highest_roc_auc,</span>
<span id="cb113-2"><a href="#cb113-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">file =</span> <span class="st">"data/r/random_forest.RData"</span></span>
<span id="cb113-3"><a href="#cb113-3" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
</section>
<section id="gradient-boosting-1" class="level4" data-number="3.2.5">
<h4 data-number="3.2.5" class="anchored" data-anchor-id="gradient-boosting-1"><span class="header-section-number">3.2.5</span> Gradient Boosting</h4>
<div class="cell-container"><div class="cell-decorator"><pre>In [60]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb114"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb114-1"><a href="#cb114-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Specify the Gradient boosting model</span></span>
<span id="cb114-2"><a href="#cb114-2" aria-hidden="true" tabindex="-1"></a>model_xgboost <span class="ot">&lt;-</span> parsnip<span class="sc">::</span><span class="fu">boost_tree</span>(</span>
<span id="cb114-3"><a href="#cb114-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">mode =</span> <span class="st">"classification"</span>,</span>
<span id="cb114-4"><a href="#cb114-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">trees =</span> <span class="dv">100</span>,</span>
<span id="cb114-5"><a href="#cb114-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">min_n =</span> <span class="fu">tune</span>(),</span>
<span id="cb114-6"><a href="#cb114-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">tree_depth =</span> <span class="fu">tune</span>(),</span>
<span id="cb114-7"><a href="#cb114-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">learn_rate =</span> <span class="fu">tune</span>(),</span>
<span id="cb114-8"><a href="#cb114-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">loss_reduction =</span> <span class="fu">tune</span>()</span>
<span id="cb114-9"><a href="#cb114-9" aria-hidden="true" tabindex="-1"></a>) <span class="sc">|&gt;</span></span>
<span id="cb114-10"><a href="#cb114-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_engine</span>(<span class="st">"xgboost"</span>, <span class="at">objective =</span> <span class="st">"binary:logistic"</span>)</span>
<span id="cb114-11"><a href="#cb114-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb114-12"><a href="#cb114-12" aria-hidden="true" tabindex="-1"></a><span class="do">## Specify the models tuning parameters using the `dials` package along (https://dials.tidymodels.org/) with the grid</span></span>
<span id="cb114-13"><a href="#cb114-13" aria-hidden="true" tabindex="-1"></a><span class="do">## space. This helps identify the hyperparameters with the lowest prediction error.</span></span>
<span id="cb114-14"><a href="#cb114-14" aria-hidden="true" tabindex="-1"></a>xgboost_params <span class="ot">&lt;-</span> dials<span class="sc">::</span><span class="fu">parameters</span>(</span>
<span id="cb114-15"><a href="#cb114-15" aria-hidden="true" tabindex="-1"></a>    <span class="fu">min_n</span>(),</span>
<span id="cb114-16"><a href="#cb114-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">tree_depth</span>(),</span>
<span id="cb114-17"><a href="#cb114-17" aria-hidden="true" tabindex="-1"></a>    <span class="fu">learn_rate</span>(),</span>
<span id="cb114-18"><a href="#cb114-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">loss_reduction</span>()</span>
<span id="cb114-19"><a href="#cb114-19" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb114-20"><a href="#cb114-20" aria-hidden="true" tabindex="-1"></a>xgboost_grid <span class="ot">&lt;-</span> dials<span class="sc">::</span><span class="fu">grid_max_entropy</span>(</span>
<span id="cb114-21"><a href="#cb114-21" aria-hidden="true" tabindex="-1"></a>    xgboost_params,</span>
<span id="cb114-22"><a href="#cb114-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="dv">10</span></span>
<span id="cb114-23"><a href="#cb114-23" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb114-24"><a href="#cb114-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb114-25"><a href="#cb114-25" aria-hidden="true" tabindex="-1"></a><span class="do">## Tune the model via cross-validation</span></span>
<span id="cb114-26"><a href="#cb114-26" aria-hidden="true" tabindex="-1"></a>xgboost_tuned <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">tune_grid</span>(workflows<span class="sc">::</span><span class="fu">add_model</span>(thyroid_workflow, <span class="at">spec =</span> model_xgboost),</span>
<span id="cb114-27"><a href="#cb114-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">resamples =</span> cv_folds,</span>
<span id="cb114-28"><a href="#cb114-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">grid =</span> xgboost_grid,</span>
<span id="cb114-29"><a href="#cb114-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">metrics =</span> yardstick<span class="sc">::</span><span class="fu">metric_set</span>(roc_auc, accuracy, ppv),</span>
<span id="cb114-30"><a href="#cb114-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">control =</span> tune<span class="sc">::</span><span class="fu">control_grid</span>(<span class="at">verbose =</span> <span class="cn">FALSE</span>)</span>
<span id="cb114-31"><a href="#cb114-31" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [61]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb115"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb115-1"><a href="#cb115-1" aria-hidden="true" tabindex="-1"></a><span class="do">## We get the best final fit from the Gradient Boosting model.</span></span>
<span id="cb115-2"><a href="#cb115-2" aria-hidden="true" tabindex="-1"></a>xgboost_highest_roc_auc <span class="ot">&lt;-</span> xgboost_tuned <span class="sc">|&gt;</span></span>
<span id="cb115-3"><a href="#cb115-3" aria-hidden="true" tabindex="-1"></a>    tune<span class="sc">::</span><span class="fu">select_best</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in tune::select_best(xgboost_tuned): No value of `metric` was given;
"roc_auc" will be used.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb117"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb117-1"><a href="#cb117-1" aria-hidden="true" tabindex="-1"></a>final_xgboost <span class="ot">&lt;-</span> tune<span class="sc">::</span><span class="fu">finalize_workflow</span>(</span>
<span id="cb117-2"><a href="#cb117-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">add_model</span>(thyroid_workflow, model_xgboost),</span>
<span id="cb117-3"><a href="#cb117-3" aria-hidden="true" tabindex="-1"></a>    xgboost_highest_roc_auc</span>
<span id="cb117-4"><a href="#cb117-4" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb117-5"><a href="#cb117-5" aria-hidden="true" tabindex="-1"></a>final_fit <span class="ot">&lt;-</span> final_xgboost <span class="sc">|&gt;</span> <span class="fu">fit</span> (<span class="at">data =</span> train)</span>
<span id="cb117-6"><a href="#cb117-6" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(final_fit)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>        Length Class      Mode   
pre     3      stage_pre  list   
fit     2      stage_fit  list   
post    1      stage_post list   
trained 1      -none-     logical</code></pre>
</div>
<div class="sourceCode cell-code" id="cb119"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb119-1"><a href="#cb119-1" aria-hidden="true" tabindex="-1"></a><span class="do">## it is unclear from the output what predictors were significant, so i need to examin the model to see what predictors were used</span></span>
<span id="cb119-2"><a href="#cb119-2" aria-hidden="true" tabindex="-1"></a>preprocessor <span class="ot">&lt;-</span> final_fit <span class="sc">|&gt;</span> <span class="fu">extract_preprocessor</span>()</span>
<span id="cb119-3"><a href="#cb119-3" aria-hidden="true" tabindex="-1"></a><span class="do">## view the predictors evaluated</span></span>
<span id="cb119-4"><a href="#cb119-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(preprocessor)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 23 × 4
   variable             type      role      source  
   &lt;chr&gt;                &lt;list&gt;    &lt;chr&gt;     &lt;chr&gt;   
 1 age_at_scan          &lt;chr [2]&gt; predictor original
 2 gender               &lt;chr [3]&gt; predictor original
 3 ethnicity            &lt;chr [3]&gt; predictor original
 4 incidental_nodule    &lt;chr [3]&gt; predictor original
 5 palpable_nodule      &lt;chr [3]&gt; predictor original
 6 rapid_enlargement    &lt;chr [3]&gt; predictor original
 7 compressive_symptoms &lt;chr [3]&gt; predictor original
 8 hypertension         &lt;chr [3]&gt; predictor original
 9 vocal_cord_paresis   &lt;chr [3]&gt; predictor original
10 graves_disease       &lt;chr [3]&gt; predictor original
# ℹ 13 more rows</code></pre>
</div>
<div class="sourceCode cell-code" id="cb121"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb121-1"><a href="#cb121-1" aria-hidden="true" tabindex="-1"></a><span class="do">## above showed the variables used</span></span>
<span id="cb121-2"><a href="#cb121-2" aria-hidden="true" tabindex="-1"></a><span class="do">## the next line shows the extracts the fitted model from the work flow</span></span>
<span id="cb121-3"><a href="#cb121-3" aria-hidden="true" tabindex="-1"></a>model_xgboost <span class="ot">&lt;-</span> <span class="fu">extract_fit_parsnip</span>(final_fit)<span class="sc">$</span>fit</span>
<span id="cb121-4"><a href="#cb121-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(model_xgboost)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>               Length Class              Mode       
handle             1  xgb.Booster.handle externalptr
raw            95013  -none-             raw        
niter              1  -none-             numeric    
evaluation_log     2  data.table         list       
call               8  -none-             call       
params            10  -none-             list       
callbacks          1  -none-             list       
feature_names     45  -none-             character  
nfeatures          1  -none-             numeric    </code></pre>
</div>
<div class="sourceCode cell-code" id="cb123"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb123-1"><a href="#cb123-1" aria-hidden="true" tabindex="-1"></a><span class="do">## the next code evaluate the important variables</span></span>
<span id="cb123-2"><a href="#cb123-2" aria-hidden="true" tabindex="-1"></a>importance_xgboost <span class="ot">&lt;-</span> xgboost<span class="sc">::</span><span class="fu">xgb.importance</span>(<span class="at">model =</span> model_xgboost)</span>
<span id="cb123-3"><a href="#cb123-3" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(importance_xgboost)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>                   Feature   Gain  Cover Frequency
                    &lt;char&gt;  &lt;num&gt;  &lt;num&gt;     &lt;num&gt;
1: bta_u_classification_U2 0.8201 0.5114    0.3106
2:               tsh_value 0.0900 0.0693    0.1677
3:          size_nodule_mm 0.0332 0.3619    0.3634
4:             age_at_scan 0.0314 0.0344    0.1025
5:                 albumin 0.0253 0.0230    0.0559</code></pre>
</div>
<div class="sourceCode cell-code" id="cb125"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb125-1"><a href="#cb125-1" aria-hidden="true" tabindex="-1"></a><span class="do">## the print shows age and gender as important</span></span>
<span id="cb125-2"><a href="#cb125-2" aria-hidden="true" tabindex="-1"></a><span class="do">## to make a prediction on the train set</span></span>
<span id="cb125-3"><a href="#cb125-3" aria-hidden="true" tabindex="-1"></a>xgboost_predictions <span class="ot">&lt;-</span> <span class="fu">predict</span>(final_fit, test, <span class="at">type =</span> <span class="st">"prob"</span>) <span class="sc">|&gt;</span></span>
<span id="cb125-4"><a href="#cb125-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">bind_cols</span>(<span class="fu">predict</span>(final_fit, test)) <span class="sc">|&gt;</span></span>
<span id="cb125-5"><a href="#cb125-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">bind_cols</span>(test)</span>
<span id="cb125-6"><a href="#cb125-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(xgboost_predictions)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] ".pred_Benign"                  ".pred_Cancer"                 
 [3] ".pred_class"                   "age_at_scan"                  
 [5] "gender"                        "ethnicity"                    
 [7] "incidental_nodule"             "palpable_nodule"              
 [9] "rapid_enlargement"             "compressive_symptoms"         
[11] "hypertension"                  "vocal_cord_paresis"           
[13] "graves_disease"                "hashimotos_thyroiditis"       
[15] "family_history_thyroid_cancer" "exposure_radiation"           
[17] "albumin"                       "tsh_value"                    
[19] "lymphocytes"                   "monocyte"                     
[21] "bta_u_classification"          "solitary_nodule"              
[23] "size_nodule_mm"                "cervical_lymphadenopathy"     
[25] "thy_classification"            "final_pathology"              </code></pre>
</div>
<div class="sourceCode cell-code" id="cb127"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb127-1"><a href="#cb127-1" aria-hidden="true" tabindex="-1"></a><span class="do">## analyse model performance using roc curves, need to figure out what "truth" and estimate refers to here, as final_pathology and .pred_Cancer do not run</span></span>
<span id="cb127-2"><a href="#cb127-2" aria-hidden="true" tabindex="-1"></a><span class="do">## roc_auc &lt;- xgboost_predictions |&gt;</span></span>
<span id="cb127-3"><a href="#cb127-3" aria-hidden="true" tabindex="-1"></a><span class="do">##    roc_auc(truth = , estimate = )</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [62]:</pre></div><div class="cell">
<div class="sourceCode cell-code" id="cb128"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb128-1"><a href="#cb128-1" aria-hidden="true" tabindex="-1"></a><span class="fu">save</span>(xgboost_tuned, xgboost_grid, final_xgboost, importance_xgboost, model_xgboost, <span class="do">## xgboost_highest_roc_auc,</span></span>
<span id="cb128-2"><a href="#cb128-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">file =</span> <span class="st">"data/r/xgboost.RData"</span></span>
<span id="cb128-3"><a href="#cb128-3" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb128-4"><a href="#cb128-4" aria-hidden="true" tabindex="-1"></a><span class="st">``</span></span>
<span id="cb128-5"><a href="#cb128-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-6"><a href="#cb128-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-7"><a href="#cb128-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-8"><a href="#cb128-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-9"><a href="#cb128-9" aria-hidden="true" tabindex="-1"></a><span class="do">#### SVM</span></span>
<span id="cb128-10"><a href="#cb128-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-11"><a href="#cb128-11" aria-hidden="true" tabindex="-1"></a><span class="sc">&lt;!--</span> {{<span class="sc">&lt;</span> include sections<span class="sc">/</span>_svm.qmd <span class="sc">&gt;</span>}} <span class="sc">-</span><span class="ot">-&gt;</span></span>
<span id="cb128-12"><a href="#cb128-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-13"><a href="#cb128-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-14"><a href="#cb128-14" aria-hidden="true" tabindex="-1"></a><span class="do">#### Explainability</span></span>
<span id="cb128-15"><a href="#cb128-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-16"><a href="#cb128-16" aria-hidden="true" tabindex="-1"></a>Which factors are important to classification can be assessed not just by the <span class="st">"importance"</span> but by methods know as</span>
<span id="cb128-17"><a href="#cb128-17" aria-hidden="true" tabindex="-1"></a>[LIME](https<span class="sc">:</span><span class="er">//</span>search.r<span class="sc">-</span>project.org<span class="sc">/</span>CRAN<span class="sc">/</span>refmans<span class="sc">/</span>lime<span class="sc">/</span>html<span class="sc">/</span>lime<span class="sc">-</span>package.html) (Local Interpretable Model<span class="sc">-</span>Agnostic</span>
<span id="cb128-18"><a href="#cb128-18" aria-hidden="true" tabindex="-1"></a>Explanations)  <span class="sc">@</span>ribeiro2016 and [Shapley</span>
<span id="cb128-19"><a href="#cb128-19" aria-hidden="true" tabindex="-1"></a>values](https<span class="sc">:</span><span class="er">//</span>shap.readthedocs.io<span class="sc">/</span>en<span class="sc">/</span>latest<span class="sc">/</span>example_notebooks<span class="sc">/</span>overviews<span class="sc">/</span>An<span class="sc">%20introduction%</span><span class="dv">20</span>to<span class="sc">%20explainable%</span><span class="dv">20</span>AI<span class="sc">%20with%</span><span class="dv">20</span>Shapley%<span class="dv">20</span>values.html)</span>
<span id="cb128-20"><a href="#cb128-20" aria-hidden="true" tabindex="-1"></a><span class="sc">@</span>lundberg2017</span>
<span id="cb128-21"><a href="#cb128-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-22"><a href="#cb128-22" aria-hidden="true" tabindex="-1"></a><span class="do">#### Comparision</span></span>
<span id="cb128-23"><a href="#cb128-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-24"><a href="#cb128-24" aria-hidden="true" tabindex="-1"></a>Comparing the sensitivity of the different models goes here.</span>
<span id="cb128-25"><a href="#cb128-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-26"><a href="#cb128-26" aria-hidden="true" tabindex="-1"></a><span class="sc">+</span> Table of sensitivity<span class="sc">/</span>specificity<span class="sc">/</span>other metrics.</span>
<span id="cb128-27"><a href="#cb128-27" aria-hidden="true" tabindex="-1"></a><span class="sc">+</span> ROC curves</span>
<span id="cb128-28"><a href="#cb128-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-29"><a href="#cb128-29" aria-hidden="true" tabindex="-1"></a><span class="do">## Conclusion</span></span>
<span id="cb128-30"><a href="#cb128-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-31"><a href="#cb128-31" aria-hidden="true" tabindex="-1"></a>The take<span class="sc">-</span>away message is....these things are hard<span class="sc">!</span></span>
<span id="cb128-32"><a href="#cb128-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-33"><a href="#cb128-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-34"><a href="#cb128-34" aria-hidden="true" tabindex="-1"></a><span class="do">## Appendix</span></span>
<span id="cb128-35"><a href="#cb128-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb128-36"><a href="#cb128-36" aria-hidden="true" tabindex="-1"></a><span class="do">### Data Dictionary</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [63]:</pre></div><div class="cell" data-tbl-caption="Description of variables in the Sheffield Thyroid dataset.">

<div class="cell-container"><div class="cell-decorator"><pre>In [64]:</pre></div><div><div class="sourceCode cell-code" id="cb129"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb129-1"><a href="#cb129-1" aria-hidden="true" tabindex="-1"></a>var_labels <span class="sc">|&gt;</span></span>
<span id="cb129-2"><a href="#cb129-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.data.frame</span>() <span class="sc">|&gt;</span></span>
<span id="cb129-3"><a href="#cb129-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable</span>(<span class="at">col.names =</span> <span class="fu">c</span>(<span class="st">"Description"</span>),</span>
<span id="cb129-4"><a href="#cb129-4" aria-hidden="true" tabindex="-1"></a>        <span class="at">caption=</span><span class="st">"Description of variables in the Sheffield Thyroid dataset."</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div><div id="tbl-variables" class="cell quarto-float quarto-figure quarto-figure-center anchored" data-tbl-caption="Description of variables in the Sheffield Thyroid dataset.">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-variables-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;3: Description of variables in the Sheffield Thyroid dataset.
</figcaption>
<div aria-describedby="tbl-variables-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="do-not-create-environment cell caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">age_at_scan</td>
<td style="text-align: left;">Age</td>
</tr>
<tr class="even">
<td style="text-align: left;">albumin</td>
<td style="text-align: left;">Albumin</td>
</tr>
<tr class="odd">
<td style="text-align: left;">bta_u_classification</td>
<td style="text-align: left;">BTA U</td>
</tr>
<tr class="even">
<td style="text-align: left;">cervical_lymphadenopathy</td>
<td style="text-align: left;">Cervical Lymphadenopathy</td>
</tr>
<tr class="odd">
<td style="text-align: left;">compressive_symptoms</td>
<td style="text-align: left;">Compressive symptoms</td>
</tr>
<tr class="even">
<td style="text-align: left;">consistency_nodule</td>
<td style="text-align: left;">Nodule consistency</td>
</tr>
<tr class="odd">
<td style="text-align: left;">eligibility</td>
<td style="text-align: left;">Eligibility</td>
</tr>
<tr class="even">
<td style="text-align: left;">ethinicity</td>
<td style="text-align: left;">Ethinicity</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ethnicity</td>
<td style="text-align: left;">Ethnicity</td>
</tr>
<tr class="even">
<td style="text-align: left;">exposure_radiation</td>
<td style="text-align: left;">Exposure to radiation</td>
</tr>
<tr class="odd">
<td style="text-align: left;">family_history_thyroid_cancer</td>
<td style="text-align: left;">Family history of thyroid cancer</td>
</tr>
<tr class="even">
<td style="text-align: left;">final_pathology</td>
<td style="text-align: left;">Final diagnosis</td>
</tr>
<tr class="odd">
<td style="text-align: left;">fna_done</td>
<td style="text-align: left;">FNA done</td>
</tr>
<tr class="even">
<td style="text-align: left;">gender</td>
<td style="text-align: left;">Gender</td>
</tr>
<tr class="odd">
<td style="text-align: left;">graves_disease</td>
<td style="text-align: left;">Graves’ disease</td>
</tr>
<tr class="even">
<td style="text-align: left;">hashimotos_thyroiditis</td>
<td style="text-align: left;">Hashimoto’s disease</td>
</tr>
<tr class="odd">
<td style="text-align: left;">hypertension</td>
<td style="text-align: left;">Hypertension</td>
</tr>
<tr class="even">
<td style="text-align: left;">incidental_nodule</td>
<td style="text-align: left;">Incidental nodule</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lymphocytes</td>
<td style="text-align: left;">Lymphocytes</td>
</tr>
<tr class="even">
<td style="text-align: left;">monocyte</td>
<td style="text-align: left;">Monocytes</td>
</tr>
<tr class="odd">
<td style="text-align: left;">palpable_nodule</td>
<td style="text-align: left;">Palpable nodule</td>
</tr>
<tr class="even">
<td style="text-align: left;">rapid_enlargement</td>
<td style="text-align: left;">Rapid enlargement</td>
</tr>
<tr class="odd">
<td style="text-align: left;">repeat_bta_u_classification</td>
<td style="text-align: left;">Repeat BTA U</td>
</tr>
<tr class="even">
<td style="text-align: left;">repeat_fna_done</td>
<td style="text-align: left;">Repeat FNA</td>
</tr>
<tr class="odd">
<td style="text-align: left;">repeat_thy_classification</td>
<td style="text-align: left;">Repeat Thy class</td>
</tr>
<tr class="even">
<td style="text-align: left;">repeat_ultrasound</td>
<td style="text-align: left;">Repeat ultrasound</td>
</tr>
<tr class="odd">
<td style="text-align: left;">size_nodule_mm</td>
<td style="text-align: left;">Nodule size (mm)</td>
</tr>
<tr class="even">
<td style="text-align: left;">solitary_nodule</td>
<td style="text-align: left;">Solitary nodule</td>
</tr>
<tr class="odd">
<td style="text-align: left;">study_id</td>
<td style="text-align: left;">Study ID</td>
</tr>
<tr class="even">
<td style="text-align: left;">thy_classification</td>
<td style="text-align: left;">Thy classification</td>
</tr>
<tr class="odd">
<td style="text-align: left;">thyroid_histology_diagnosis</td>
<td style="text-align: left;">Histology</td>
</tr>
<tr class="even">
<td style="text-align: left;">thyroid_surgery</td>
<td style="text-align: left;">Thyroid surgery</td>
</tr>
<tr class="odd">
<td style="text-align: left;">tsh_value</td>
<td style="text-align: left;">TSH value</td>
</tr>
<tr class="even">
<td style="text-align: left;">vocal_cord_paresis</td>
<td style="text-align: left;">Vocal cord paresis</td>
</tr>
</tbody>
</table>
</div>
</div>
</figure>
</div></div></div>
</div></div>
<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-vanBuuren2011Dec" class="csl-entry" role="listitem">
Buuren, Stef van, and Karin Groothuis-Oudshoorn. 2011. <span>“<span class="nocase">mice: Multivariate Imputation by Chained Equations in R</span>.”</span> <em>J. Stat. Soft.</em> 45 (December): 1–67. <a href="https://doi.org/10.18637/jss.v045.i03">https://doi.org/10.18637/jss.v045.i03</a>.
</div>
<div id="ref-Efron2016Jul" class="csl-entry" role="listitem">
Efron, Bradley, and Trevor Hastie. 2016. <em><span class="nocase">Computer Age Statistical Inference: Algorithms, Evidence, and Data Science</span></em>. <em>Cambridge Core</em>. Cambridge, England, UK: Cambridge University Press. <a href="https://doi.org/10.1017/CBO9781316576533">https://doi.org/10.1017/CBO9781316576533</a>.
</div>
<div id="ref-tidymodels" class="csl-entry" role="listitem">
Kuhn, Max, and Hadley Wickham. 2020. <em>Tidymodels: A Collection of Packages for Modeling and Machine Learning Using Tidyverse Principles.</em> <a href="https://www.tidymodels.org">https://www.tidymodels.org</a>.
</div>
<div id="ref-r_citation" class="csl-entry" role="listitem">
R Core Team. 2023. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-tibshirani1996" class="csl-entry" role="listitem">
<span>“<span class="nocase">Regression Shrinkage and Selection via the Lasso</span>.”</span> 1996. <em>Journal of the Royal Statistical Society. Series B (Methodological)</em>. <a href="https://www.jstor.org/stable/2346178">https://www.jstor.org/stable/2346178</a>.
</div>
<div id="ref-smith2018" class="csl-entry" role="listitem">
Smith, Gary. 2018. <span>“<span class="nocase">Step away from stepwise</span>.”</span> <em>J. Big Data</em> 5 (1): 1–12. <a href="https://doi.org/10.1186/s40537-018-0143-6">https://doi.org/10.1186/s40537-018-0143-6</a>.
</div>
<div id="ref-steyerberg2001" class="csl-entry" role="listitem">
Steyerberg, Ewout W., Marinus J. C. Eijkemans, Frank E. Harrell, and Dik. 2001. <span>“Prognostic Modeling with Logistic Regression Analysis.”</span> <em>Medical Decision Making</em> 21 (1): 45–56. <a href="https://doi.org/10.1177/0272989x0102100106">https://doi.org/10.1177/0272989x0102100106</a>.
</div>
<div id="ref-thompson1995" class="csl-entry" role="listitem">
Thompson, Bruce. 1995. <span>“Stepwise Regression and Stepwise Discriminant Analysis Need Not Apply Here: A Guidelines Editorial.”</span> <em>Educational and Psychological Measurement</em> 55 (4): 525–34. <a href="https://doi.org/10.1177/0013164495055004001">https://doi.org/10.1177/0013164495055004001</a>.
</div>
<div id="ref-tidyverse" class="csl-entry" role="listitem">
Wickham, Hadley, Mara Averick, Jennifer Bryan, Winston Chang, Lucy D’Agostino McGowan, Romain François, Garrett Grolemund, et al. 2019. <span>“Welcome to the <span class="nocase">tidyverse</span>.”</span> <em>Journal of Open Source Software</em> 4 (43): 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.
</div>
<div id="ref-zou2005" class="csl-entry" role="listitem">
Zou, Hui, and Trevor Hastie. 2005. <span>“<span class="nocase">Regularization and Variable Selection Via the Elastic Net</span>.”</span> <em>J. R. Stat. Soc. Ser. B. Stat. Methodol.</em> 67 (2): 301–20. <a href="https://doi.org/10.1111/j.1467-9868.2005.00503.x">https://doi.org/10.1111/j.1467-9868.2005.00503.x</a>.
</div>
</div>
</section>
</section>
</section>
     </main>
<!-- /main column -->  <script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>  </div> <!-- /content -->  <script>var lightboxQuarto = GLightbox({"descPosition":"bottom","closeEffect":"zoom","selector":".lightbox","loop":false,"openEffect":"zoom"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script> 
  
</body></html>